---
title: "Journées annuelles du GT Bioss (2018, 4e édition)"
date: 2018-07-02
url: /manif/jnbioss_201807/jnbioss201807.html
summary: "Les **2 et 3 juillet 2018** à Marseille"
categories: "Manifestations"
---


### Informations générales

Date : Lundi 2 (toute la journée) et mardi 3 juillet (matin) 2018

Lieu : site Saint-Charles de l\'université d\'Aix-Marseille à Marseille.
 

Organisateurs : Elisabeth Remy, Grégory Batt, Cédric Lhoussaine et Anne
Siegel 

La quatrième édition des journées annuelles du GT Bioss va se dérouler
juste avant les Journées Ouvertes Biologie, Informatique et
Mathématiques ([JOBIM](https://jobim2018.sciencesconf.org/)).


### Inscription

L\'inscription, gratuite mais obligatoire, se fait en remplissant le
formulaire accessible
[ici.](https://docs.google.com/forms/d/1adQlZhN32sjrXwBPoM8ISrscn0CegW89TOKhnvVf9AY/viewform?edit_requested=true)

### Orateurs invités

[Olivier Bernard](http://www-sop.inria.fr/members/Olivier.Bernard/),
équipe projet BioCore, Inria.\
[François Fages](http://lifeware.inria.fr/wiki/Fages/HomePage), équipe
projet Lifeware, Inria.\
[Bertie
Gottgens](https://www.cimr.cam.ac.uk/research/affiliated/gottgens),
Cambridge Institute for Medical Research.\
[Heike Siebert](http://page.mi.fu-berlin.de/hsiebert/), DFG-Research
Center Matheon, Berlin.

### Programme

#### Lundi 2 Juillet

**10h00-10h45** - **Conférencière invitée** - Heike Siebert (Université
de Berlin)- *A Boolean Look at Synthetic Biology - Finding Cell
Classifiers Using Answer Set Programming*\
**10h45-11h05** - Adrien Richard (I3S, Nice) - *Fixing monotone boolean
networks asynchronously*\
**Pause**\
**11h30-11h50** - Stephanie Chevalier (LRI) - *A logical approach to
identify Boolean Networks that model cell differentiation*\
**11h55-12h15** - Maxime Folschette (Irset/Irisa) - *GULA:
Semantics-Free Learning of a Biological Regulatory Networks from a
Synchronous, Asynchronous or Generalized State Graph*\
**12h20-12h40** - Aurelien Naldi (ENS Paris) - *Similarities and
complementarity of positive feedback circuits and stable motifs in
logical regulatory networks*\
\
**12h40- 14h00 - Pause déjeuner**\
\
**14h00-14h45** - **Conférencier invité** -Bertie Göttgens (Cambridge
institute for Medical Research) - *Reconstructing Cell States, Lineage
trajectories and Regulatory Networks from Single cells Molecular
profiles.*\
**14h50-15h10** - Alberto Valdeolivas (I2M) - *A Multiplex Network
approach to Premature Aging Diseases*\
**Pause**\
**15h30-15h50** - Céline Hernandez (ENS Paris) - *Dynamical modelling of
T cell co-inhibitory pathways to predict anti-tumour responses to
checkpoint inhibitors*\
**15h50-16h10** - Eugenia Oshurko (LIP) - *Representation and
aggregation of cellular signalling knowledge in KAMI*\
**16h15-16h35** - Sébastien Légaré (LIP) - *Biocuration and rule-based
modelling of protein interaction networks in KAMI*\
**Pause**\
**17h00-17h45** - **Conférencier invité** - François Fages (Inria
Saclay) - *Computer-aided biochemical programming of synthetic micro
reactors as diagnostic devices*\
**17h50-18h10** - Loic Pauleve (LRI) - *The CoLoMoTo Interactive
Notebook: Accessible and Reproducible Computational Analyses for
Qualitative Biological Networks* **18h15-18h35** - Discussion Bioss.\

#### Mardi 3 Juillet

**09h00-09h45** - **Conférencier invité** - Olivier Bernard (Inria Nice
Sophia Antipolis) - *Dynamical Reduction of Metabolic Networks.
Application to Microalgae*\
**09h50-10h10** - Nils Giordano (LS2N) - *Using co-activity networks to
reveal the structure of planktonic symbioses in the global ocean*\
**Pause**\
**10h30-10h50** - Ghuvan Grimaud (Biomathematica) - *EvoDRUM: an
evolutionary systems biology framework to investigate the origin of
early metabolisms*\
**10h55-11h15** - Anne Siegel (IRISA) - *Learning boolean rules for the
regulatory control of metabolism: a case study*\
**11h20-11h40** Thibault Etienne (Ibis, Inria) - *Coordination of mRNA
stability and cell physiology in bacteria: a modelling study*

### Résumés

**Stephanie Chevalier (LRI)** - *A logical approach to identify Boolean
Networks that model cell differentiation.*\
[Résumé](./Resume_ChevalierStephanie.pdf)\
\
**Thibault Etienne (Ibis, Inria)** - *Coordination of mRNA stability and
cell physiology in bacteria: a modelling study*\
Thibault Etienne, Laurence Girbal, Muriel Cocaign-Bousquet, Delphine
Ropers\
The adaptation of bacterial physiology to environmental fluctuations
involves system-wide changes of metabolism and gene expression. This
reprogramming of the cell takes place at two different levels: on a
global scale through the adjustment of the level and activity of
components of the gene expression machinery (ribosomes\...), and
locally, through the adjustment of the concentration of regulators
specifically coordinating the cell response to the new environmental
conditions.\
The different regulatory levels are interlaced and form large
biochemical networks, whose dynamic functioning is not intuitive. Among
these regulatory levels, recent studies have shown that
post-transcriptional regulations are more important than usually
assumed. Contrary to the often-made assumption in bacteria, protein and
mRNA levels are not proportional and mRNA stability (typically a few
minutes) varies with the translational activity, the cell growth rate
and the concentration of regulators (small RNAs, HFQ,\...). How these
interlocked control mechanisms adjust mRNA half-life to cell physiology
remains largely unknown. In our study, we tackle this question by means
of mathematical modelling using available times-series -omics data in
Escherichia coli (transcriptomics and stabilomics). Our objective is to
provide a mechanistic explanation of mRNA degradation profiles obtained
at various growth rates.\
We develop a structural model of mRNA degradation in E.coli based on
Michaelis-Menten kinetics. In this model, the individual parameters vary
with the nature of the mRNA and the cell growth rate. A mixed-effect
modelling framework is used to take into account the variability of
these parameters: using the genome-wide - omics data, we estimate the
mean parameters describing the population of mRNAs and the variance
parameters, which allow to reproduce the degradation profile of each
mRNA in each condition. The analysis of mean parameter values informs us
on the global regulatory effects, while the parameter variances reflect
the specific regulatory mechanisms.\
\
**Maxime Folschette (Irset/Irisa)** - *GULA: Semantics-Free Learning of
a Biological Regulatory Networks from a Synchronous, Asynchronous or
Generalized State Graph*\
The automatic learning of an interaction graph from the sole observation
of its dynamics is an ongoing challenge. An example is the existing LFIT
algorithm which learns and refines logic rules representing a model,
from a set of state transitions representing its dynamics. Starting from
the learning of purely deterministic synchronous Boolean systems,
several versions have been developed in order to tackle dynamics with
memory, inconsistencies or with multi-valued variables. However, all of
them rely on the knowledge of the underlying semantics, that is, the
update scheme of the variables. This work intents to free the learning
process from this knowledge.\
With GULA (General Usage LFIT Algorithm), we focus on three different
semantics: synchronous (all variables must update their value between
two discrete time steps), asynchronous (exactly one variable must do so)
or general (any subset of variables may do so). The learning presented
here is based on the refinement of logic rules (a set of conditional
atoms and a conclusion atom) that represent the possibility for a
variable to change its value under some conditions on the current state.
Such rules only represent the potentiality of a change, which makes them
independent of the semantics. Nevertheless, we also exhibit some
properties that characterize the dynamics of the three given semantics,
allowing to correctly interpret the rules of the final regulatory
graph.\
This work opens many outcomes. The most pressing is finding a broader
characterization of what a "learnable" semantics is, allowing to
generalize the scope of this approach. Furthermore, the semantics itself
could be learned along with the rules, allowing to entirely learn a
system. Finally, getting rid of the arbitrary but mandatory
discretization step would allow to directly learn from the gene
expression measurements, as already proposed with ACEDIA.\
\
**Ghuvan Grimaud (Biomathematica)** - *EvoDRUM: an evolutionary systems
biology framework to investigate the origin of early metabolisms*\
Ghjuvan Grimaud, Elena Litchman, Christopher Klausmeier\
The origin of the fundamental metabolic pathways and the subsequent rise
of the great metabolic diversity of microbes are two major steps in
life's evolution on Earth and potentially other habitable planets.
Understanding how different metabolisms may arise, what conditions
select for different types of metabolic networks, and how they assemble
to form ecological communities are key questions for the origin of life.
The evolutionary emergence of diverse metabolisms depends not only on
environmental conditions but also on microbial interactions such as
competition and mutualism. Ecological interactions are being
increasingly recognized as a driving force of evolutionary
diversification in different groups of organisms, including microbes
\[1\]. So far, the role of microbial interactions in the origin of
metabolic pathways under dynamic conditions has not been investigated in
detail. Here we propose to combine two novel modeling approaches from
two disparate disciplines (systems biology and evolutionary ecology) to
explore how microbial metabolic networks arise and evolve in dynamic
community contexts. We embed a recently developed metabolic modeling
approach for the elementary flux mode analysis under nonequilibrium
conditions (the Dynamic Reduction of Unbalanced Metabolism, DRUM\[2\])
in an eco-evolutionary modeling framework of trait evolution (Adaptive
Dynamics \[3,4\]) to investigate how different metabolic networks arise
and compete in different environments. The resulting new Evolutionary
Systems Biology mathematical framework (evoDRUM) is a powerful tool that
allows extensive explorations of how early metabolisms appeared and were
maintained by natural selection and, thus, is useful for the field of
early microbial evolution. EvoDRUM extends and modifies the idea of
gathering the evolutionarily possible reactions by defining a large ---
ideally universal --- mutation space in which evolution can
proceed\[5\]. In line with the Adaptive Dynamics framework, evolution
occurs by a step-by-step mutant/resident invasion dynamics, with a
defined mutation rate. The novelty of the proposed approach is that it
investigates the metabolically explicit trait changes and evolution as a
result of selection through competitive interactions of different
phenotypes, and allows the incorporation of metabolite accumulation and
evolutionary innovations. First applied to simple metabolic networks
with several resources and temporally fluctuating conditions, we then
use it for genome-scale metabolic networks.\
*References*\
1. Brodie, J., Ball, S.G., Bouget, F.-Y., Chan, C.X., De Clerck, O.,
Cock, J.M., Gachon, C., Grossman, A.R., Mock, T., Raven, J.A., Saha, M.,
Smith, A.G., Vardi, A., Yoon, H.S., and Bhattacharya, D. (2017). Biotic
interactions as drivers of algal origin and evolution. New Phytologist
216, 670-681.\
2. Baroukh C., Munoz-Tamayo R., Steyer J.P. and Bernard O. (2014). DRUM:
A new framework for metabolic modeling under non- balanced growth.
Application to the carbon metabolism of unicellular microalgae. PloS
one, 9 (8), e104499.\
3. Dieckmann U. and Law R. (1996). The dynamical theory of coevolution:
A derivation from stochastic ecological processes. Journal of
Mathematical Biology, 34, 579-612.\
5. Geritz S., Kisdi E., Meszéna G. and Metz J. (1998). Evolutionarily
singular strategies and the adaptive growth and branching of the
evolutionary tree. Evolutionary Ecology, 12, 35-57.\
5. Szappanos B., Fritzemeier J., Csörgo B., Lazar V., Lu X., Fekete G.,
Balint B., Herczeg R., Nagy I., Notebaart R.A. et al. (2016). Adaptive
evolution of complex innovations through stepwise metabolic niche
expansion. Nature communications, 7.\
\
**Céline Hernandez (ENS Paris)** - *Dynamical modelling of T cell
co-inhibitory pathways to predict anti-tumour responses to checkpoint
inhibitors*\
Céline Hernandez(1), Aurélien Naldi(1), Wassim Abou-Jaoudé(1), Guillaume
Voisinne(2), Romain Roncagalli(2), Bernard Malissen(2), Morgane
Thomas-Chollier(1), Denis Thieffry(1)\
(1) *Computational Systems Biology team, Institut de Biologie de l'Ecole
Normale Supérieure (IBENS), CNRS UMR8197, INSERM U1024, Ecole Normale
Supérieure, PSL Université, 75005 Paris, France*\
(2) *Centre d\'Immunologie de Marseille-Luminy, Aix Marseille
Université, INSERM U1104, CNRS UMR7280, 13288 Marseille, France*\
In recent years, it has been recognised that T cells often display a
reduced ability to eliminate cancer cells and that expression of
co-inhibitors at their surface accounts for their compromised function.
Antibodies blocking the functions of these co-inhibitors (checkpoint
inhibitors) have become standard treatment for metastatic melanoma
\[1\], leading to a revival in the study of T cell co-inhibitors.
However, our understanding of the immunobiology of T cell co-inhibitors
and of their harmful role during anti-tumour responses remains
fragmentary. Despite some biochemical studies, a mechanistic
understanding at the system-level of the modulation of T cell function
by co-inhibitors has remained elusive.\
To overcome these limitations, we aim at delineating the mechanisms
through which co-inhibitory molecules, such as PD-1 and CTLA-4, impede T
cell functions at the system-level. To reach this goal, we use
computational methods to map and model TCR co-signalling pathways, and
ultimately predict cell responses to perturbations.\
First, we focused on the development of comprehensive annotated
molecular maps (using the software CellDesigner \[2\]) based on the
curation of scientific literature, in parallel with automated queries to
public databases and protein-protein graph reconstruction. Next, using
the software GINsim \[3\], these maps and protein networks are
translated into a regulatory graph integrating current knowledge. The
challenge is then to properly model concurrent intracellular processes,
along with feedback control mechanisms. To cope with this complexity, we
explored some network modules using a Rule-based formalism \[4\], in
order to evaluate concurrent biological hypotheses and help specify
logical rules recapitulating observed component behaviour back into the
logical model. This model will be used to predict cell response to
single or multiple perturbations, and thereby pave the way to the
delineation of novel experiments, which will in turn be used to refine
the maps and model.\
This integrated system-level view of the mechanisms of action of key T
cell co-inhibitors in cancer will further provide a rationale for
designing and evaluating drugs targeting T cell co-inhibitory pathways
in anti-cancer immunotherapy.\
*References*\
1. Simpson TR, Li F, Montalvo-Ortiz W, Sepulveda MA, Bergerhoff K, Arce
F, Roddie C, Henry JY, Yagita H, Wolchok JD, Peggs KS, Ravetch JV,
Allison JP, Quezada SA (2013). Fc-dependent depletion of
tumor-infiltrating regulatory T cells co-defines the efficacy of
anti-CTLA-4 therapy against melanoma. The Journal of experimental
medicine 210(9): 1695--710.\
2. http://www.celldesigner.org/\
3. http://www.ginsim.org\
4. Feret J, Danos V, Krivine J, Harmer R, Fontana W (2009). Internal
coarse-graining of molecular systems. Proceedings of the National
Academy of Sciences of the USA 106(16): 6453-8\
\
**Sébastien Légaré (LIP)** - *Biocuration and rule-based modelling of
protein interaction networks in KAMI.*\
KAMI, the Knowledge Aggregator and Model Instantiator, is a software for
biocuration and modelling of molecular interaction networks. It provides
a knowledge representation to unambiguously express the details of
biomolecular interactions. This representation can be built either
programmatically or graphically via the KamiStudio interface. To assist
users in curating their biological knowledge, KAMI is organised in two
distinct layers: a network and a set of individual interactions called
nuggets. Once a new nugget is built, it can be automatically aggregated
to the network. The software then performs a series of tests to ensure
consistency including duplicate search, biological database grounding
and semantic checking. This greatly facilitates biocuration as users do
not need to have the complete network in mind to add new data.
Furthermore, interaction networks represented in KAMI can be directly
converted to rule-based models in the Kappa language for simulation and
analysis. In this talk, we will present the use of KAMI through a model
of tyrosine phosphorylation involved in cell signaling. This example is
well suited to showcase the advantages of the rule-based strategy. In
particular, we will demonstrate the use of causality analysis to
discover pathways in the model that were not explicitly input by the
user.\
\
**Aurélien Naldi​, ENS Paris** - *Similarities and complementarity of
positive feedback circuits and stable motifs in logical regulatory
networks*\
Aurélien Naldi​, Denis Thieffry​\
Discrete qualitative models have been widely used to study complex
biological regulatory networks. The increasing complexity of the systems
of interest calls for efficient analysis methods, and in particular
approaches directly relating the structure of the network to its
dynamical properties.\
The study of feedback circuits, based on the seminal work of R. Thomas,
is a prominent example of such approaches: positive circuits are
associated to the co-existence of multiple attractors, while negative
circuits are associated to sustained oscillations \[1\]. The properties
of isolated circuits and of some simple combinations of circuits have
been formally characterised \[2\], however their precise roles once
embedded in complex networks remain unknown. An embedded circuit is
called "functional" when the values of its regulators allow it to behave
as an isolated circuit.\
Stable motifs (also called symbolic steady states) have recently been
proposed to efficiently identify attractors of such models \[3,4\]. Each
stable motif represent a partial assignment of model components such
that all successors of the matching states also belong to the motif.\
The identification of stable motifs was recently added to the GINsim
software \[5\], which already supported the identification of functional
circuits. Based on the availability of these two analysis methods in the
same software tool, we further explore the connection between the
classical feedback circuits and stable motifs. The core of each stable
motif is formed by a (group of) positive circuits settled in one of
their two stable configurations. The resulting stability is often
associated to functional positive circuits which can sustain their own
functionality contexts. Stable motifs can further arise from
non-functional positive circuits, which can be locked in only one of
their two stable configurations.\
*References*\
\[1\] Comet ​et al. ​(2013). On circuit functionality in boolean
networks.​ Bulletin of Mathematical Biology​ ​75​: 906-19.\
\[2\] Remy ​et al. (2016). Boolean Dynamics of Compound Regulatory
circuits. In : Rogato A, Zazzu V, Guarracino MR (Eds.). ​Dynamics of
Mathematical Models in Biology​. Springer International Publishing, pp.
43-53.\
\[3\] Zañudo & Albert (2013). An effective network reduction approach to
find the dynamical repertoire of discrete dynamic networks. ​Chaos​
​23​: 025111.\
\[4\] Klarner ​et al. ​(2014). Computing Symbolic Steady States of
Boolean Networks. ​Lecture Notes in Computer Sciences​ ​8751​: 561-70.
\[5\] http://ginsim.org\
\
**Eugenia Oshurko (LIP)** - *Representation and aggregation of cellular
signalling knowledge in KAMI*\
Rule-based modelling has proven to be a successful approach for study-
ing complex systems of cellular signalling. A rule-based language Kappa
has been actively developed and used in recent years. However, building
and curating big explanatory models using Kappa rules is challenging and
cumbersome. To tackle exactly this problem we propose a bio-curation
tool called KAMI (Knowledge Aggregator and Model Instantiator), which
allows gradual semi-automatic aggregation of PPIs of different
provenance, their annotation, visualisation and further instantiation to
concrete rule-based models (including automatic generation of Kappa
rules).\
Models in KAMI are accommodated using a specially designed graph- based
knowledge representation system which provides robust mechanisms for
incremental aggregation of partial knowledge, its audit, update, and
transfer to various representations. In this talk we will present this
knowl- edge representation system, its properties and the mechanisms for
knowledge update based on graph rewriting. We will also focus on its
instance used in KAMI to represent models of cellular signalling
systems. Then we will speak about the strategy of automatic knowledge
aggregation that exploits the properties of this system. And finally, we
will show how KAMI uses domain-specific background knowledge (e.g.
semantics of conserved pro- tein domains, definitions of protein
families, splice variants and mutants) to sharpen aggregated models.\
\
**Loic Pauleve (LRI)** - *The CoLoMoTo Interactive Notebook: Accessible
and Reproducible Computational Analyses for Qualitative Biological
Networks*\
Joint work with A Naldi, C Hernandez, N Levy, G Stoll, P Monteiro, C
Chaouiya, T Helikar, A Zinovyev, L Calzone, S Cohen-Boulakia, D
Thieffry\
The CoLoMoTo Interactive Notebook relies on Docker and Jupyter
technologies to provide a unified environment to edit, execute, share,
and reproduce analyses of qualitative models of biological networks. To
date, the framework provides access to software tools including Cell
Collective, GINsim, BioLQM, Pint, and MaBoSS. A Python interface has
been developed for each of these tools to offer a seamless integration
in the Jupyter web interface and ease the chaining of complementary
analyses.\
Website: http://colomoto.org/notebook\
Paper: http://doi.org/10.3389/fphys.2018.00680\
\
**Alberto Valdeolivas (I2M)** - *A Multiplex Network approach to
Premature Aging Diseases.*\
Premature aging (PA) syndromes are a group of heterogeneous rare
disorders that recapitulate some of the aspects associated to
physiological aging. They are caused by mutations in several genes
involved in different biological processes. Genes and proteins do not
act isolated in cells but rather interact in complex networks of
molecular interactions. In this context, we undertook a network approach
to better understand the etiology and pathophysiolgy of these diseases.\
First, we extracted the network modules surrounding genes mutated in PA
diseases, to define the landscape of biological processes that might be
perturbed. To this goal, we applied a strategy based on our recently
developed random walk (RW) with restart on multiplex networks \[1\].
This allows us to navigate and extract information from different layers
of physical and functional interactions (e.g., protein-protein,
co-expression, molecular complexes) outperforming single-network
approaches \[1\]. We captured modules representing the hallmarks of
physiological aging, and compared the processes commonly perturbed in PA
diseases, as well as those specific to a subset of diseases.\
In a second part, we are developing a strategy to analyse the impact on
networks of PA disease-causing mutations. To this goal, we are
performing targeted attacks, removing from the multiplex network either
genes (to simulate loss-of-function) or some of their interactions (to
simulate \"edgetic\" mutations). A modified version of our RW algorithm
allows us to study the topological modifications of the network after
the attack, pinpointing to the most affected genes, modules and
processes.\
1. Valdeolivas,A. et al. Random Walk With Restart On Multiplex And
Heterogeneous Biological Networks. 2017. bioRxiv.
