<!DOCTYPE html>
<html><head>
  <title>Bioss</title> 

  <meta name="description" content="Page du groupe de travail sur la
				    biologie systémique symbolique">
				    
  <meta name="keywords" content="Modélisation symbolique de systèmes
				 biologiques, Systèmes dynamiques
				 discrets, Systèmes formels,
				 Langages de modélisation,
				 Sémantique, Vérification de
				 modèles, Réduction, Prédiction
				 sous incertitude, Inférences
				 d'interactions et de règles,
				 Systèmes hybrides">

  <meta http-equiv="content-type" content="text/html; charset=UTF-8">

  <link rel="stylesheet" type="text/css" href="../../style/style.css">
</head>

<body>,&nbsp;
  <div id="main">
    <div id="header">
      <div id="logo">
        <div id="logo_text">
          <!-- class="logo_colour", allows you to change the colour of
          the text -->
          <h1><a href="index.html">
	      <span class="logo_colour">Bioss</span>
	  </a></h1>
          <h2>Groupe de travail sur la biologie systémique symbolique</h2>
        </div>
      </div>
      <div id="menubar">
        <ul id="menu">
          <!-- put class="selected" in the li tag for the selected
          page - to highlight which page you're on -->
          <li><a href="../../index.html">Accueil</a></li>
          <li><a href="../../membres.html">Membres</a></li>
          <li><a href="../../manif.html">Manifestations</a></li>
          <li><a href="../../projets.html">Projets</a></li>
          <li><a href="../../contact.html">Contacts</a></li>
        </ul>
      </div>
    </div>

    <div id="content_header"></div>

    <div id="site_content">

      <!-- <div id="banner"></div> -->

      <div id="sidebar_container">
        <div class="sidebar">
          <div class="sidebar_top"></div>
          <div class="sidebar_item"></div>
          <div class="sidebar_base"></div>
        </div>

        <div class="sidebar">
          <div class="sidebar_top"></div>
          <div class="sidebar_item">
            <h3>Liens utiles</h3>
            <ul>
              <li><a href="http://www.cnrs.fr/">
		  <img src="../../img/logoCNRS.jpg" alt="CNRS"></a></li>
              <li><a href="http://www.gdr-bim.cnrs.fr/">
		  <img src="../../img/logoBIM.jpg" alt="GDR Bioinformatique 
						  moléculaire"></a></li>
              <li><a href="https://www.gdr-im.fr/">
		  <img src="../../img/logoIM.jpg" alt="GDR Informatique 
						  mathématique"></a></li>
            </ul>
          </div>
          <div class="sidebar_base"></div>
        </div>
      </div>

      <div id="content">
        <!-- insert the page content here -->
	<p align="justify">
        <h1>Séminaire mensuel virtuel (les vendredi à 13h)</h1>	
	<h3>Utiliser ce <a href="https://calendar.google.com/calendar/u/0?cid=aDRvOW11NnBhcnR2OHFpNmtiMDhzN3Z1bGtAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ">lien</a> pour intégrer l'agenda partagé des sémaires Bioss dans votre agenda.</h3><br/><br/>
	<p/>
<iframe src="https://calendar.google.com/calendar/embed?height=400&amp;wkst=2&amp;bgcolor=%23ffffff&amp;ctz=Europe%2FParis&amp;src=aDRvOW11NnBhcnR2OHFpNmtiMDhzN3Z1bGtAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ&amp;src=ZnIuZnJlbmNoI2hvbGlkYXlAZ3JvdXAudi5jYWxlbmRhci5nb29nbGUuY29t&amp;color=%234285F4&amp;color=%237986CB&amp;showTitle=0&amp;showTz=0&amp;mode=MONTH&amp;showCalendars=0&amp;showTabs=0&amp;showPrint=0&amp;showNav=1" style="border:solid 1px #777" width="600" height="400" frameborder="0" scrolling="no"></iframe>
	<p/>
	<b>Lieu</b> : <a href="https://u-bordeaux-fr.zoom.us/j/86764096440?pwd=b01qOG04RTMvRWNOVHBYR1ZIbkVaUT09">URL Zoom</a><br/>
Meeting ID: 867 6409 6440<br/>
Passcode: 149120
  <br/>
	<p/>
	<b><h4>1er octobre 2021: <a href="https://sites.google.com/view/laura-cantini/home">Laura Cantini</a> (IBENS / Paris) et
      <a href="https://www.linkedin.com/in/sergio-corridore-678ab675/">Sergio Corridore</a> (Institut Curie / Paris)</h4></b>
	<b>13h00 - 13h30</b> -- <b>Laura Cantini</b> (IBENS / ENS Paris) -- <i>Multi-omics integration: towards a comprehensive view of cancer.</i> <!-- a href="slides/"></a--><br/>
Due to the advent of high-throughput technologies, high-dimensional “omics” data are produced at an increasing pace. In cancer biology, national and international consortia have profiled thousands of tumors at multiple molecular levels (“multi-omics”) allowing to gather a comprehensive molecular picture of this disease. Moreover, multi-omics profiling approaches are currently being transposed at single-cell resolution, further increasing the information accessible from cancer samples.  The current main challenge is to design appropriate methods to integrate this wealth of information and translate it into actionable biological knowledge.<br/>

In this talk, I will discuss two maincomputational directions for multi-omics integration: (i) multilayer networks to integrate a large range of interactions and (ii) joint dimensionality reduction to extract biological knowledge simultaneously from multiple omics. First, I will present their application on bulk data and then I will discuss our ongoing research in single-cell.<br/>

<i>Selected associated publications & preprints</i><br/>

Cantini L, Medico E, Fortunato S, Caselle M. Detection of gene communities in multi-networks reveals cancer drivers. Scientific reports. 2015 Dec 7;5(1):1-0.<br/>

Cantini, L., Zakeri, P., Hernandez, C., Naldi, A., Thieffry, D., Remy, E., Baudot, A., 2021. Benchmarking joint multi-omics dimensionality reduction approaches for the study of cancer. Nature Communications 12.<br/>

Kang Y, Thieffry D, Cantini L. Evaluating the reproducibility of single-cell gene regulatory network inference algorithms. Frontiers in genetics. 2021 Mar 22;12:362.<br/>

Huizing GJ, Peyré G, Cantini L. Optimal Transport improves cell-cell similarity inference in single-cell omics data. bioRxiv. 2021 Jan 1.
  <br/><br/>

	<b>13h30 - 14h00</b> -- <b>Sergio Corridore</b> (Institut Curie / Paris)  -- <i>Temozolomide cellular Pharmacokinetic/ Pharmacodynamic Model in the context of Brain Tumour.</i><br/>

   Glioblastoma multiforme (GBM) is the most frequent and aggressive type of primary
brain tumours in adults. Despite very intensive treatments including maximal safe neurosurgery,
radiation therapy and chemotherapy, the prognosis of GBM patients remains poor with a median
overall survival below 18 months. Temozolomide (TMZ)-based chemotherapy is the most com-
mon pharmacological treatment in patients with diagnosed GBM. Even if TMZ administration
improves patient overall survival, prognosis remains poor [8] and no major therapeutic advance
has been accomplished within the past 10 years. This can be related to a lack of knowledge
in how the tumour evolves and ultimately escape drug activity, especially in a context of large
inter-patient variability.<br/>
New systems pharmacology  approaches combining experimental and mathematical expertise
provide interesting perspectives towards the design of safe and ecient TMZ-based therapies
against GBM [2, 4]. The present study aims to do so through the conception of a model of
TMZ pharmacokinetics-pharmacodynamics (PK-PD) and of key regulatory networks, capable
of reproducing the intracellular events from TMZ exposure to cell rescue or apoptosis. TMZ
is a methylating agent that creates lesions on the DNA after a two-step activation process [7].
Four types of DNA adducts are formed upon drug exposure, which are handled either by O6-
methylguanine-DNA methyltransferase (MGMT) or by the base excision repair (BER) system
[9, 10]. If DNA repair is unsuccessful, DNA single- or double-strand breaks are created, which
triggers Homologous Recombination (HR), ATR/Chk1 and p53 activation, cell cycle arrest and
possibly apoptosis [3].<br/>
We designed a model, based on ordinary dierential equations, that recapitulates these intra-
cellular events. Then, model calibration consisted in a modied least square approach ensuring
data best-t satised biologically-sound constraints, the numerical minimization problem being
performed by the Covariance Matrix Evolutionary Strategy (CMAES) algorithm. The model
was able to reproduce multi-type datasets of several independent studies mostly performed in
either the U87 or LN229 glioblastoma cell lines [1, 6, 5]. These datasets included longitudinal
and dose-dependent studies of TMZ cellular PK, DNA adduct formation, ATR, Chk1 and p53
phosphorylation, and cell death. This calibrated PK-PD model is currently being used as a
powerful tool to investigate new therapeutic targets. Drug combinations involving TMZ and one
to three targeted therapies are explored, among which clinically available inhibitors of ATR (e.g.
Berzosertibe), PARP (e.g. olaparib) or Cyclin Dependent Kinase4/6 (e.g. palbociclib). The next
step will imply a partial re-calibration of the model with multi-omics datasets available for GBM
patient-derived cell lines or GBM patient samples, towards a mechanism-based personalization
of GMB treatment.<br/>

<i>References</i><br/>
[1] Dorthe Aasland et al. \Temozolomide induces senescence and repression of DNA repair
pathways in glioblastoma cells via activation of ATR{CHK1, p21, and NF-B". In: Cancer
research 79.1 (2019), pp. 99-113.<br/>
[2] A Ballesta et al. \Multiscale Design of Cell-Type{Specic Pharmacokinetic/Pharmacodynamic
Models for Personalized Medicine: Application to Temozolomide in Brain Tumors". In:
CPT: pharmacometrics & systems pharmacology 3.4 (2014), pp. 1-11.<br/>
[3] Simona Caporali et al. \DNA damage induced by temozolomide signals to both ATM
and ATR: role of the mismatch repair system". In: Molecular pharmacology 66.3 (2004),
pp. 478-491.<br/>
[4] Jeremy ZR Han et al. \Personalized Medicine for Neuroblastoma: Moving from Static Geno-
types to Dynamic Simulations of Drug Response". In: Journal of Personalized Medicine
11.5 (2021), p. 395.<br/>
[5] Yang He and Bernd Kaina. \Are there thresholds in glioblastoma cell death responses
triggered by temozolomide?" In: International journal of molecular sciences 20.7 (2019),
p. 1562.<br/>
[6] Christopher B Jackson et al. \Temozolomide sensitizes MGMT-decient tumor cells to
ATR inhibitors". In: Cancer research 79.17 (2019), pp. 4331-4338.<br/>
[7] Steve Quiros, Wynand P Roos, and Bernd Kaina. \Processing of O6-methylguanine into
DNA double-strand breaks requires two rounds of replication whereas apoptosis is also
induced in subsequent cell cycles". In: Cell cycle 9.1 (2010), pp. 168-178.<br/>
[8] Sarah Smalley, Anthony J Chalmers, and Simon J Morley. \mTOR inhibition and levels
of the DNA repair protein MGMT in T98G glioblastoma cells". In: Molecular cancer 13.1
(2014), pp. 1-11.<br/>
[9] Anish Thomas et al. \Temozolomide in the era of precision medicine". In: Cancer research
77.4 (2017), pp. 823-826.<br/>
[10] J Lee Villano, Tara E Seery, and Linda R Bressler. \Temozolomide in malignant gliomas:
current use and future targets". In: Cancer chemotherapy and pharmacology 64.4 (2009),
pp. 647-655.
<br/><br/>

  	<b><h4>4 juin 2021: <a href="https://www.lri.fr/~speres">Sabine Peres</a> (LISN / Paris Saclay) et
<a href="https://www.linkedin.com/in/leonard-herault-8a62bb107/?originalSubdomain=fr">Léonard Hérault</a> (Aix Marseille Université)</h4></b>

	<b>13h00 - 13h30</b> -- <b>Sabine Peres</b> (LISN / Paris Saclay) -- <i>Computing Constraints-Based Elementary Flux Modes: Application to Escherichia coli Core Metabolism.</i> <!-- a href="slides/"></a--><br/>
	
	Elementary Flux Modes (EFMs) provide a rigorous basis to
	systematically characterize the steady state, cellular
	phenotypes, as well as metabolic network robustness and
	fragility. However, the number of EFMs typically grows
	exponentially with the size of the metabolic network, leading
	to excessive computational demands, and unfortunately, a large
	fraction of these EFMs are not biologically feasible due to
	system constraints. However, only a few constraints can be
	integrated in the traditional computation; most of them must
	be treated in post-processing and thus do not save
	computational time.

	In this talk, we will present the biological constraints that
	we integrate into the EFMs calculation. We rely on a hybrid
	computational tool based on Answer Set Programming (ASP) and
	Linear Programming (LP) that permits the computation of EFMs
	while implementing many different types of constraints. We
	will illustrate this approach to the Escherichia coli core
	model in considering transcriptional and environmental
	regulations, thermodynamic constraints, and resource usage
	considerations.  <br/><br/>

	<b>13h30 - 14h00</b> -- <b>Léonard Hérault</b>  (Aix Marseille Université)  -- <i>Single cell RNA seq assisted synthesis of a  boolean transcription factors network to model early hematopoiesis and its alteration with aging.</i><br/>

	We previously characterized early hematopoieisis in young and
aged mice through hematopoietic stem cell (HSC) sc-RNA-seq
analysis. Thanks to HSC clustering and pseudotime ordering as well as
regulon analysis we showed differentiation paths of HSC toward three
primed states and an accumulation of quiescent HSCs upon
aging. Starting from these results coupled to current knowledge of
transcriptionnal regulation of early hematopoiesis we built a boolean
network using answer set programming in order to model HSC priming and
its alteration with aging.<br/><br/>


	<b><h4>7 mai 2021: <a href="http://www.lsv.fr/~mfuegger/">Matthias Függer</a> (CNRS / ENS Paris-Saclay) et 
<a href="https://www.cristal.univ-lille.fr/profil/ddursoni?lang=fr">Danilo Dursoniah</a> (CRIStAL / Lille)</h4></b>
	<b>13h00 - 13h30</b> -- <b>Matthias Függer</b> (CNRS / ENS Paris-Saclay) -- <i>Distributed Computation with Continual Population Growth.</i> <!-- a href="slides/"></a--><br/>
	The talk is on recent work towards distributed crcuits within growing bacterial systems.
	We will focus on majority consensus that plays a key role in tolerating noise within such circuits
	and discuss the performance and correctness of an algorithm.
	The talk is based on work with Da-Jung Cho, Corbin Hopper,  Manish Kushwaha, Thomas Nowak,
	and Quentin Soubeyran.
	<br/><br/>

	<b>13h30 - 14h00</b> -- <b>Danilo Dursoniah</b>  (CRIStAL / Lille)  -- <i>Limits of a Glucose-Insulin Model to Investigate Intestinal Absorption in Type 2 Diabetes.</i><br/>
	Abnormal regulation of glucose absorption in the small
	intestine is an important cause of Type 2 Diabetes (T2D). Even if this
	hypothesis is clinically well-known, it has not been fundamentally
	validated yet, mainly due to a lack of reliable metabolic knowledge on
	the glucose regulation.  We aim to test this hypothesis on a highly
	referenced model composed of ordinary differential equations,
	published by Dalla Man & al. in 2007.  This model is tested by trying
	to infer reliable parameters from differents original datasets: one
	featuring the observations of obese diabetic patients, the other from
	minipigs undergoing several experimental conditions such as intestinal
	surgery or pancreatectomy. This latter dataset is more reliable than
	humans due to the relatively low variability between the
	individuals. In both cases, our work shows the model's limits to
	predict our post-prandial glycemia and insulinemia time series
	especially with regard to the crucial complexity of gastro-intestinal
	regulation.<br/><br/>

	<b><h4>2 avril 2021: <a href="https://annuaire.ifremer.fr/cv/23648">Francis Mairet</a> (Ifremer / Nantes) et <a href="https://www.linkedin.com/in/antrea-pavlou-218733147">Antrea Pavlou</a> (IBIS / Grenoble)</h4></b>
	<b>13h00 - 13h30</b> -- <b>Francis Mairet</b> (Ifremer, Nantes) -- <i>Optimal proteome allocation determines temperature dependence of microbial growth laws.</i> -- <a href="slides/"></a>. <br/>
	Although the effect of temperature on microbial growth has
	been widely studied, the role of proteome allocation in bringing
	about temperature-induced changes remains elusive. In this talk, I
	will present a coarse-grained model of microbial growth - including
	the processes of temperature-sensitive protein unfolding and
	chaperone-assisted (re)folding - that we develop to tackle this
	problem. We determine the proteome sector allocation that maximizes
	balanced growth rate as a function of nutrient limitation and
	temperature. Calibrated with quantitative proteomic data for
	Escherichia coli, the model allows us to clarify general principles
	of temperature-dependent proteome allocation and formulate growth
	laws.  The same activation energy for metabolic enzymes and ribosomes
	leads to an Arrhenius increase in growth rate at constant proteome
	composition over a large range of temperatures, whereas at extreme
	temperatures resources are diverted away from growth to
	chaperone-mediated stress responses. Our approach points at risks and
	possible remedies for the use of ribosome content to characterize
	complex ecosystems with temperature variation.<br/><br/>

	<b>13h30 - 14h00</b> -- <b>Antrea Pavlou</b>  (IBIS / Grenoble)  -- <i>Insights into bacterial resource allocation in dynamically changing environments using a combination of experimental and mathematical approaches..</i><br/>
	with E. Cinquemani, H. Geiselmann, H. de Jong<br/>

	The relationship between bacterial growth and the environment
	has been well characterized over the last 50 years. In most studies,
	however, bacteria are maintained at steady-state growth even though in
	reality they are rarely in a constant environment. To investigate
	bacterial adaptation in changing environments, we track growth and
	gene expression of single- cell bacteria growing in a microfluidic
	device in changing environments. We examine the behavior of specific
	ribosomal and metabolic genes in this context using fluorescent
	protein tags. The experimental results provide a detailed view of
	resource allocation strategies of bacteria in dynamically changing
	environments and are helpful in testing the predictions made by
	resource allocation models of bacterial growth.<br/><br/>

	<b><h4>12 mars 2021: <a href="http://pagesperso.ls2n.fr/~chaffron-s">Samuel Chaffron</a> (Combi team, LS2N, Nantes) et <a href="https://www.lri.fr/~schevalier">Stéphanie Chevalier</a> (LRI, Paris-Saclay)</h4></b>
	<b>13h00 - 13h30</b> -- <b>Samuel Chaffron</b> (Combi team, LS2N, Nantes) -- <i>Environmental vulnerability of the global ocean plankton community interactome.</i> -- <a href="slides/chaffron_samuel.pdf">slides</a>. <br/>
	Marine plankton form complex communities of interacting
	organisms at the base of the food web, which sustain oceanic
	biogeochemical cycles, and help regulate climate. Though
	global surveys are starting to reveal ecological drivers
	underlying planktonic community structure, and predicted
	climate change responses, it is unclear how community-scale
	species interactions will be affected by climate change. Here
	we leveraged Tara Oceans sampling to infer a global ocean
	cross-domain plankton co-occurrence network – the community
	interactome – and used niche modeling to assess its
	vulnerabilities to environmental change. Globally, this
	revealed a plankton interactome self-organized latitudinally
	into marine biomes (Trades, Westerlies, Polar), and more
	connected poleward. Integrated niche modeling revealed
	biome-specific community interactome responses to
	environmental change, and forecasted most affected lineages
	for each community. These results provide baseline approaches
	to assess community structure and organismal interactions
	under climate scenarios, while identifying plausible plankton
	bioindicators for ocean monitoring of climate change.<br/><br/>

	<b>13h30 - 14h00</b> -- <b>Stéphanie Chevalier</b> (Lifeware / Inria Saclay)  -- <i>Synthesis of Boolean networks from single-cell differentiation data.</i><br/>
	Processes like cell differentiation and cancerisation have
	dynamical properties around the notion of trajectory
	(succession of changes in gene state), non-reachability
	(bifurcating event) and stability (differentiated
	cell). Single-cell data on such behaviors are now quite widely
	available but dynamical modelling with them remains too
	complex to be commonly leveraged. I will present the approach
	we develop to automatically infer dynamical models from such
	data and prior knowledge on gene interactions. The inference
	method consists in formulating the inference as a Boolean
	satisfiability problem, described as a logic program
	containing both the modelling formalism (Most Permissive
	Boolean network - MPBN) and the data on the biological process
	(prior knowledge, experimental measurements, dynamics,
	hypotheses). Several constraints have been implemented in
	Answer-Set Programming to ensure the desired dynamical
	properties, and thanks to this logic modeling it is now
	possible to exhaustively enumerate the MPBN compatible with
	the constraints of cell differentiation behaviors. In order to
	leverage single-cell data, I firstly run classification and
	trajectory reconstruction methods, then data are translated
	into logical form to describe the cells dynamics. I will
	present preliminary results obtained for a large-scale
	modeling of hematopoiesis from cell-scale transcriptomic
	sequencing data (single-cell RNA-seq data). Potential
	influences between genes and proteins are extracted from the
	SIGNOR database, which brings more than 5500 components
	(genes, proteins and complexes).
	<br/><br/>

	<h4><b>5 février 2021: <a href="http://www.ens-lyon.fr/LBMC/laboratoire/annuaire/1-gandrillon-olivier">Olivier Gandrillon</a> (LBMC / ENS Lyon) et <a href="http://aurelien.naldi.info/">Aurélien Naldi</a> (Lifeware / Inria Saclay)</h4></b>
	<b>13h00 - 13h30</b> -- <b>Olivier Gandrillon</b> (LBMC / ENS Lyon) -- <i> A probabilistic dynamical framework for Gene Regulatory Network inference and simulation.</i>
         Joined work with Matteo Bouvier, Alexey Koshkin, Fabien Crauste, Arnaud Bonnaffoux and Olivier Gandrillon.<br/>
	In this talk, I will first recall our proposal for a GRN model that is simultaneously probabilistic, dynamical, and executable (Herbach et al. 2017; Bonnaffoux et al. 2019). It is specifically designed to reproduce and to predict the time-dependent evolution of the gene expression distributions that we observe at the single-cell level, for example during a differentiation process.<br/>
I will then address two open questions: (i) How do we compare a model's output to experimental data, that is single-cell-based gene expression distributions? and (ii) How do we compare the output of two different models? We will show that the main difficulty comes from the probabilistic nature of the model: two runs of the same model with the exact same parameter values will generate two different distributions.<br/>
I will present our current proposal and argue that there is no definitive answer to those questions and that more dedicated research is needed to answer those.<br/>
Herbach, U., Bonnaffoux, A., Espinasse, T., and Gandrillon, O. (2017). Inferring gene regulatory networks from single-cell data: a mechanistic approach. BMC Systems Biology 11, 105.<br/>
Bonnaffoux, A., Herbach, U., Richard, A., Guillemin, A., Gonin-Giraud, S., Gros, P.-A., and Gandrillon, O. (2019). WASABI: a dynamic iterative framework for gene regulatory network inference. BMC Bioinformatics 20, 220.<br/><br/>
	<b>13h30 - 14h00</b> -- <b>Aurélien Naldi</b> (Lifeware / Inria Saclay) -- <i>Kinetic assumptions in Boolean networks: a case for buffering.</i> -- <a href="slides/2021-02-05_BIOSS_buffered_models_kinetic_assumptions.pdf">slides </a><br/>
Boolean networks are widely used to study complex biological systems,
especially in absence of precise kinetic information. Their asynchronous
interpretation has long been considered as more realistic than the
synchronous one, as it removes some implicit kinetic assumptions. The
"most permissive" semantics, which has been recently introduced, removes
all known remaining assumptions and offers surprisingly good
computational properties. However, this semantics also enables some
dynamical behaviors which may conflict with the expected biological
meaning of many models, in particular it can depend on hidden dual
interactions between components of the network.
We propose buffered network as a balance between the implicit kinetic
assumptions of the asynchronous interpretation and the strong
generalization of the most permissive semantics. Using this approach to
refine the results obtained with the most permissive semantics, we
identified some key analytical results which remain valid when we
preserve the signs of all interactions, while others should be treated
more carefully.
	<br/><br/>

	<b><h4>8 janvier 2021: <a href="https://research-information.bris.ac.uk/en/persons/thomas-e-gorochowski">Thomas E. Gorochowski</a> (University of Bristol, UK) et</b><b> <a href="https://research.pasteur.fr/fr/member/olivier-borkowski">Olivier Borkowski</a> (Inria and Institut Pasteur)</h4></b>
	<b>13h00 - 13h30</b> -- <b>Thomas E. Gorochowski</b> (University of Bristol, UK) -- <i>Using diverse sequencing technologies to accelerate genetic circuit design</i> <br/>
	<b>Résumé:</b> 
Synthetic genetic circuits are composed of many interconnected parts that must function together in concert to implement desired biological computations. A major challenge when developing new circuits is that genetic parts often display unexpected changes in their performance when used in new ways. Such changes can arise due to contextual effects or unintended interactions with the host cell. In this talk, I will demonstrate how we have been using a variety of sequencing technologies to tackle problem. First, I will show how RNA-sequencing can be used to measure the function of every transcriptional part making up large genetic circuits. This enables us to better understand why some designs fail and helps pinpoint the root cause. Then, I will present some recent work where we combined RNA-sequencing with ribosome profiling and RNA spike-in standards to enable the first large-scale characterization of transcriptional and translational parts in absolute units. Finally, I will discuss some new work that uses long-read nanopore sequencing to enable the characterization of thousands of genetic parts simultaneously to better understand their design constraints. Taken together, the methods presented provide a means for a more complete and quantitative view of the inner workings of genetic circuits and improves our understanding of the rules governing the effective construction of larger and more complex biological systems. <br/><br/>
	<b>13h30 - 14h00</b> -- <b>  Olivier Borkowski</b> (Inria and Institut Pasteur) -- <i>A large-scale exploration of cell-free compositions to maximize protein production using active learning </i><br/>
	<b>Résumé:</b> 
	Lysate-based cell-free systems have become a major platform to study gene expression but batch-to-batch variation makes protein production difficult to predict. We described an active learning approach to explore a combinatorial space of ~4,000,000 cell-free compositions, maximizing protein production and identifying critical parameters involved in cell-free productivity. We also provided a one-step-method to achieve high quality predictions for protein production using minimal experimental effort regardless of the lysate quality. Eventually, we challenged our method with a collection of E. coli cell-free systems using various homemade cell lysates and lysates supplemented with antibiotics to alter the efficiency of transcription and translation processes.<br/>
Joint work with Mathilde Koch, Agnès Zettor, Amir Pandi, Angelo Cardoso Batista, Paul Soudier, and Jean-Loup Faulon at Génomique Métabolique, Genoscope, and Micalis Institute, INRAE, France
	<p/>

	<b><h4>6 novembre 2020: <a href="http://www.i3s.unice.fr/~boyenval">Déborah Boyenval</a> (I3S/Sparks) et <a href="https://www.marseille-medical-genetics.org/a-baudot/">Loïc Paulevé</a>(CNRS/LaBRI/Formal Methods)</h4></b>
	<b>13h00 - 13h30</b> -- <b>Déborah Boyenval</b> (I3S/Sparks) -- <i>Étude des checkpoints du cycle cellulaire : spécification et vérification</i> -- <a href="./slides/Boyenval_gtbioss_6nov20.pdf">slides</a><br/>
	<b>Résumé:</b> 
Le cycle cellulaire est par définition une succession d'évènements conduisant à la duplication sans erreur de l'ADN (phase S) et l'équitable division d'une cellule mère en deux cellules filles (phase M). Au cours de la progression du cycle cellulaire (G1-S-G2-M), l'intégrité de l'ADN est garantie notamment par les checkpoints. Nos travaux montrent que la modélisation discrète du cycle cellulaire permet de modéliser proprement la notion fondamentale de checkpoint. Un nouveau modèle multivalué du cycle cellulaire est présenté en suivant le formalisme de René Thomas. Le modèle se focalise sur la succession des évènements de régulation qui représente le cycle cellulaire. On y montre que plusieurs permutations de ces évènements sont admissibles, tout en permettant néanmoins de dégager des évènements clefs non permutables qui caractérisent les checkpoints. Cette étude a été rendue possible grâce à l'usage de deux types de méthodes formelles dédiées aux réseaux de régulation multivalués: la logique de Hoare "génétiquement modifiée" et le model-checking pour CTL. L'outil TotemBioNet combine efficacement ces deux approches formelles pour identifier exhaustivement les paramètres dynamiques des modèles compatibles avec nos définitions du cycle cellulaire et de ses checkpoints.</p>
	<b>13h30 - 14h00</b> -- <b>Loïc Paulevé</b> (CNRS/LaBRI/Formal Methods) -- <i>Most Permissive Boolean Networks in practice</i> -- <a href="./slides/2020-11-06-Pauleve.pdf">slides</a><br/>
	<b>Résumé:</b> 
	Logical modeling, notably with Boolean Networks (BNs), is a well-established approach that enables reasoning on the qualitative dynamics of networks. However, (a)synchronous Boolean network, besides being costly to analyze, can preclude the prediction of certain behaviors observed in quantitative systems.<br/>
	Most Permissive Boolean Networks offer the formal guarantee not to miss any behavior achievable by a quantitative model following the same logic. Moreover, MPBNs significantly reduce the complexity of dynamical analysis, enabling to model genome-scale networks.<br/>

In this talk, after an overview of the motivation and properties of MPBNs, I'll focus on their practical usage for the analysis of models of biological networks.<br/>
Related material:
<ul>
<li> <a href="https://www.nature.com/articles/s41467-020-18112-5">https://www.nature.com/articles/s41467-020-18112-5</a></li>
<li> <a href="https://zenodo.org/record/3936123">https://zenodo.org/record/3936123</a></li>
</ul>
	<p/>
	<b><h4>2 octobre 2020: <a href="https://www6.toulouse.inrae.fr/lipm/Recherche/Pouvoir-pathogene-de-Ralstonia-et-adaptation-a-son-environnement/Membres">Caroline Baroukh</a> (INRA Toulouse)  et <a href="https://www.marseille-medical-genetics.org/a-baudot/">Anaïs Baudot</a> (MMG Marseille)</b></h4>
	<b>13h00 - 13h30</b> -- <b>Caroline Baroukh</b> (INRA Toulouse) -- <i>Modélisation métabolique des interactions plantes-pathogènes</i><br/>
	<b>Résumé:</b> 
	  Les outils de la biologie des systèmes, et plus particulièrement la modélisation métabolique, sont parfaitement adaptés pour étudier l’interaction métabolique hôte-pathogène.  En effet, ils permettent de formaliser les systèmes complexes de manière rigoureuse, d’avoir une vision globale et générique, de faire des bilans matières et surtout de faire un lien entre physiologie observée (croissance, excrétion de facteur de virulence, déplétion des substrats) et données génomiques (génome, transcriptome, protéome). Ces approches ont déjà fait leur preuve dans le domaine des biotechnologies et de la biologie de synthèse pour l’optimisation de la production de molécules d’intérêts industriels. Leur adaptation au domaine de la pathologie des plantes peut aider à déchiffrer les stratégies de virulence de pathogènes de plante.<br/>

Après une brève présentation des techniques de modélisation utilisées, deux exemples de l’apport de la modélisation métabolique pour la pathologie des plantes seront présentés. Le premier exemple est la reconstruction et la modélisation semi-automatique des réseaux métaboliques des souches du complexe d’espèces Ralstonia solanacearum, bactéries pathogènes provoquant le flétrissement de nombreuses plantes. L’étude in silico a montré que l’architecture des réseaux métaboliques semble liée à la phylogénie des souches, ainsi qu’au style de vie particulier de certaines souches. Le second exemple est la reconstruction du réseau métabolique de Xylella fastidiosa (souche CFBP8418), phytopathogène bactérien responsables de nombreuses maladies dont l’«Olive Scorch » en Italie. L’étude in silico du métabolisme de cette souche a permis de révéler certaines particularités métaboliques qui impactent fortement la robustesse du pathogène et qui pourrait expliquer en partie sa croissance fastidieuse.
	</p>
	<b>13h30 - 14h00</b> -- <b>Anaïs Baudot</b> (MMG Marseille) -- <i>A Multi-Objective Genetic Algorithm to Find Active Modules in Multiplex Biological Networks</i><br/>
	  <b>Résumé:</b> The identification of subnetworks of interest - or active modules - by integrating biological networks with molecular profiles is a key resource to inform on the processes perturbed in different cellular conditions. We here propose MOGAMUN, a Multi-Objective Genetic Algorithm to identify active modules in multiplex biological networks. MOGAMUN optimizes the scores of the nodes (e.g., their differential expression) and the density of interactions from multiplex networks. Multiplex networks are composed of different layers of physical and functional relationships between genes and proteins. Each layer is associated to its own meaning, topology, and biases; the multiplex framework allows exploiting this diversity of biological networks.
	</br>	
      <!-- <div id="banner"></div> -->
      
      </div>
    </div>

    <div id="content_footer"></div>

    <div id="footer">
      <p><a href="../../index.html">Accueil</a>
      | <a href="../../membres.html">Membres</a>
      | <a href="../../manif.html">Manifestations</a>
      | <a href="../../projets.html">Projets</a>
      | <a href="../../contact.html">Contact</a></p>
      <p><a href="http://validator.w3.org/check?uri=referer">HTML5</a>
      | <a href="http://jigsaw.w3.org/css-validator/check/referer">CSS</a>
      | <a href="http://www.html5webtemplates.co.uk">HTML5 Web
      Templates</a></p>
    </div>
  </div>


</body></html>
