<!doctype html><html><head>
<title>Journée annuelle du GT Bioss (2019, 5e édition)</title>
<meta charset=utf-8>
<meta name=X-UA-Compatible content="IE=edge">
<meta name=google-site-verification content>
<meta content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=0" name=viewport>
<meta content="telephone=no" name=format-detection>
<meta name=description content>
<meta name=renderer content="webkit">
<meta name=theme-color content="#ffffff">
<meta property="og:title" content="Journée annuelle du GT Bioss (2019, 5e édition)">
<meta property="og:description" content="Le 7 novembre 2019 à l&rsquo;université Paris Diderot">
<meta property="og:type" content="article">
<meta property="og:url" content="https://www.bioss-cnrs.fr/beta/manif/jnbioss_201911/jnbioss_201911.html"><meta property="article:section" content="events">
<meta property="article:published_time" content="2019-11-07T00:00:00+00:00">
<meta property="article:modified_time" content="2019-11-07T00:00:00+00:00"><meta property="og:site_name" content="GT BIOSS">
<link type=text/css rel=stylesheet href=/beta/vendor/css/bootstrap.min.css>
<link rel=stylesheet href=https://www.bioss-cnrs.fr/beta/scss/journal.min.e98aba1605f3a725fad0875a5584b620ed0fd58ea79d459f02fb560d7738fab4.css integrity="sha256-6Yq6FgXzpyX60IdaVYS2IO0P1Y6nnUWfAvtWDXc4+rQ=" media=screen>
<link rel=stylesheet href=https://www.bioss-cnrs.fr/beta/scss/dark-mode.min.92e13ee85b2c4587b9a2c423bde35865b35fa7449a8e1babb34f5c66907a72e1.css integrity="sha256-kuE+6FssRYe5osQjveNYZbNfp0Sajhurs09cZpB6cuE=" media=screen>
<script src=/beta/vendor/js/loadCSS.js></script>
<script>loadCSS("https://fonts.googleapis.com/css?family=Fira+Mono|Material+Icons")</script>
<script src=/beta/js/toc.js></script>
<script>console.log("Hello from 'layouts/partials/extended_head.html'")</script>
</head>
<body>
<div id=app><div id=sideContainer class=side-container>
<a class="a-block nav-head false" href=https://www.bioss-cnrs.fr/beta/>
<div class=nav-title>
GT Bioss
</div>
<div class=nav-subtitle>
Groupe de travail sur la biologie systémique symbolique
</div>
</a>
<div class=nav-link-list>
<a class="a-block nav-link-item false" href=/beta/about>
À propos
</a>
<a class="a-block nav-link-item false" href=/beta/seminaires/>
Séminaires mensuels
</a>
<a class="a-block nav-link-item false" href=/beta/categories/manifestations/>
Manifestations
</a>
<a class="a-block nav-link-item false" href=/beta/projets.html>
Projets
</a>
<a class="a-block nav-link-item false" href=/beta/membres.html>
Membres
</a>
<a class="a-block nav-link-item false" href=/beta/contact.html>
Contact
</a>
</div>
<div class=nav-footer>
<p><a href=https://www.bioss-cnrs.fr/beta/index.xml>Flux RSS</a></p>
Basé sur le
Theme Hugo <a href=https://github.com/amazingrise/hugo-theme-diary>Diary</a> par <a href=https://amazingrise.net>Rise</a>
<br>
Adapté de <a href=https://mak1t0.cc/ target=_blank rel="noreferrer noopener">Makito</a>'s <a href=https://github.com/SumiMakito/hexo-theme-journal/ target=_blank rel="noreferrer noopener">Journal.</a> <br>
<br>
&copy;
GT Bioss.
</div>
</div><div id=extraContainer class=extra-container>
<div class=pagination>
<a id=globalBackToTop class="pagination-action animated-visibility" href=#top :class="{ invisible: scrollY == 0 }">
<i class="material-icons pagination-action-icon">
keyboard_arrow_up
</i>
</a>
<a type=button class=pagination-action id=darkModeToggleButton>
<span class="material-icons pagination-action-icon" id=darkModeToggleIcon>
dark_mode
</span>
</a>
</div>
</div>
<div class=single-column-drawer-container id=drawer v-bind:class="{ 'single-column-drawer-container-active': isDrawerOpen }">
<div class=drawer-content>
<div class=drawer-menu>
<a class="a-block drawer-menu-item false" href=/beta/about>
À propos
</a>
<a class="a-block drawer-menu-item false" href=/beta/seminaires/>
Séminaires mensuels
</a>
<a class="a-block drawer-menu-item false" href=/beta/categories/manifestations/>
Manifestations
</a>
<a class="a-block drawer-menu-item false" href=/beta/projets.html>
Projets
</a>
<a class="a-block drawer-menu-item false" href=/beta/membres.html>
Membres
</a>
<a class="a-block drawer-menu-item false" href=/beta/contact.html>
Contact
</a>
</div>
</div>
</div>
<transition name=fade>
<div id=drawer-mask v-bind:class="{ 'single-column-drawer-mask': mounted }" v-if=isDrawerOpen v-on:click=toggleDrawer></div>
</transition>
<nav id=navBar class="navbar sticky-top navbar-light single-column-nav-container">
<div id=navBackground class=nav-background></div>
<div class="container container-narrow nav-content">
<button id=nav_dropdown_btn class=nav-dropdown-toggle type=button v-on:click=toggleDrawer>
<i class=material-icons>
menu
</i>
</button>
<a id=navTitle class=navbar-brand href=https://www.bioss-cnrs.fr/beta/>
GT Bioss
</a>
<button type=button class=nav-darkmode-toggle id=darkModeToggleButton2>
<i class=material-icons id=darkModeToggleIcon2>
dark_mode
</i>
</button>
</div>
</nav>
<div class=single-column-header-container id=pageHead v-bind:style="{ transform: 'translateZ(0px) translateY('+.3*scrollY+'px)', opacity: 1-navOpacity }">
<a href=https://www.bioss-cnrs.fr/beta/>
<div class=single-column-header-title>GT Bioss</div>
<div class=single-column-header-subtitle>Groupe de travail sur la biologie systémique symbolique</div>
</a>
</div>
<div id=content>
<div id=streamContainer class=stream-container>
<div class="post-list-container post-list-container-shadow">
<div class=post>
<div class=post-head-wrapper-text-only>
<div class=post-title>
Journée annuelle du GT Bioss (2019, 5e édition)
<div class=post-subtitle>
Le <strong>7 novembre 2019</strong> à l&rsquo;université Paris Diderot
</div>
<div class=post-meta>
</div>
</div>
</div>
<div class=post-body-wrapper>
<div class=post-body v-pre>
<h3 id=informations-générales>Informations générales</h3>
<p><strong>Date</strong> : le 7 novembre 2019 de 9h30 à 17h30.</p>
<p><strong>Lieu</strong> : Université Paris Diderot, salle 027C, au rez-de-chaussée de
la Halle aux Farines. Plans d'accès
<a href="https://campus.univ-paris-diderot.fr/file/7495/download?token=ufRWvV6A">ici</a>
et
<a href="https://campus.univ-paris-diderot.fr/file/7492/download?token=Og2VmjpP">là</a>.</p>
<p><strong>Organisateurs</strong> : Grégory Batt, Cédric Lhoussaine, Elisabeth Remy et
Anne Siegel </p>
<p>Cette cinquième édition des journées annuelles du GT Bioss va se
dérouler juste après la journée nationale du <a href=http://www.gdr-bim.cnrs.fr/>GDR
BiM</a> qui aura lieu le 6 novembre à
l'Université Denis Diderot.</p>
<h3 id=inscription>Inscription</h3>
<p>L'inscription, gratuite mais obligatoire, se fait via la <a href="http://www.gdr-bim.cnrs.fr/?page_id=160">page du GDR
BiM de l'évènement</a>. La SFBI
offre des bourses de voyage aux doctorants et postdocs (<strong>deadline le
candidature le 10 octobre!</strong>)!</p>
<h3 id=orateurs-invités>Orateurs invités</h3>
<p><a href=http://nuel.perso.math.cnrs.fr/>Gregory Nuel</a>, LPSM (Sorbonne
Université).<br>
<a href=https://www.lptmc.jussieu.fr/users/lesne>Annick Lesne</a>, LPTMC
(Sorbonne Université) et IGMM (Montpellier)\</p>
<h3 id=programme>Programme</h3>
<p>(préliminaire)</p>
<p><strong>09h00 - 09h30</strong> - <strong>Accueil</strong><br>
<strong>09h30 - 09h35</strong> - <strong>Introduction des journées</strong><br>
<strong>09h35 - 10h20</strong> - <strong>Conférence plénière</strong>. Gregory Nuel. <em>Estimating
causal effects in gene regulation networks.</em><br>
<strong>10h20 - 10h45</strong> - Jérémie Pardo. <em>Sequential reprogramming of
biological network fate</em>.<br>
<strong>10h45 - 11h10</strong> - Aurélien Desoeuvre. <em>Homeostasis by interval</em>.<br>
<strong>11h10 - 11h35</strong> - Aurélien Naldi. <em>Dynamic modeling of cell
populations with UPMaBoSS.</em><br>
<strong>11h35 - 12h00</strong> - Stephen Chapman. <em>Flux balance analysis reveals
acetate metabolism modulates cyclic electron flow and alternative
glycolytic pathways in Chlamydomonas reinhardtii.</em><br>
<strong>12h00 - 12h10</strong> - Flash talks.<br>
<strong>12h10 - 13h30</strong> - <strong>Pause déjeuner</strong><br>
<strong>13h30 - 14h15</strong> - <strong>Conférence plénière</strong>. Annick Lesne. <em>Bifurcation
analysis of biological circuits: time scales matter</em>.<br>
<strong>14h15 - 14h40</strong> - Diane Peurichard. <em>A new model for the emergence of
vascular networks.</em><br>
<strong>14h40 - 15h05</strong> - D. Regnault. <em>Non-cooperatively assembling large
structures.</em><br>
<strong>15h05 - 15h35</strong> - <strong>Pause</strong><br>
<strong>15h35 - 16h00</strong> - Émilie Allart. <em>Computing Difference Abstractions of
Metabolic Networks.</em><br>
<strong>16h00 - 16h25</strong> - Andreea Beica. <em>Tropical abstractions of Biochemical
Reaction Networks with guarantees.</em><br>
<strong>16h25 - 16h50</strong> - Zach Fox. <em>Optimal Experiment Designs of Signal
Activated Stochastic Gene Expression in S. Cerevisae.</em><br>
<strong>16h50 - 17h15</strong> - Mathilde Koch.<em>Large scale active-learning-guided
exploration to maximize cell-free production</em>.\</p>
<h3 id=résumés>Résumés</h3>
<p><strong>Émilie Allart</strong> - <em>Computing Difference Abstractions of Metabolic
Networks</em>. Algorithms based on abstract interpretation were proposed
recently for predicting changes of reaction networks with partial
kinetic information. Their prediction precision, however, depends
heavily on which heuristics are applied in order to add linear
consequences of the steady state equations of the metabolic network. In
this paper we ask the question whether such heuristics can be avoided
while obtaining the highest possible precision. This leads us to the
first algorithm for computing the difference abstractions of a linear
equation system exactly without any approximation. This algorithm relies
on the usage of elementary flux modes in a nontrivial manner,
first-order definitions of the abstractions, and finite domain
constraint solving.<br>
<strong>Andreea Beica</strong> - <em>Tropical abstractions of Biochemical Reaction
Networks with guarantees.</em><br>
Biochemical molecules interact through modification and binding
reactions, giving raise to a combinatorial number of possible
biochemical species. The time-dependent evolution of concentrations of
the species is commonly described by a system of coupled ordinary
differential equations (ODEs). However, the analysis of such
high-dimensional, non-linear system of equations is often
computationally expensive and even prohibitive in practice. The major
challenge towards reducing such models is providing the guarantees as to
how the solution of the reduced model relates to that of the original
model, while avoiding to solve the original model. In this paper, we
have designed and tested an approximation method for ODE models of
biochemical reaction systems, in which the guarantees are our major
requirement. Borrowing from tropical analysis techniques, dominance
relations among terms of each species' ODE are exploited to simplify
the original model, by neglecting the dominated terms. As the dominant
subsystems can change during the system's dynamics, depending on which
species dominate the others, several possible modes exist. Thus, simpler
models consisting of only the dominant subsystems can be assembled into
hybrid, piecewise smooth models, which approximate the behavior of the
initial system. By combining the detection of dominated terms with
symbolic bounds propagation, we show how to approximate the original
model by an assembly of simpler models, consisting in ordinary
differential equations that provide time-dependent lower and upper
bounds for the concentrations of the initial models species. Our method
provides sound interval bounds for the concentrations of the chemical
species, and hence can serve to evaluate the faithfulness of
tropicalization-based reduction heuristics for ODE models of biochemical
reduction systems. The method is tested on several case studies.<br>
<br>
<strong>Stephen Chapman</strong> - <em>Flux balance analysis reveals acetate metabolism
modulates cyclic electron flow and alternative glycolytic pathways in
Chlamydomonas reinhardtii.</em><br>
Cells of the green alga Chlamydomonas reinhardtii cultured in the
presence of acetate perform mixotrophic growth, involving both
photosynthesis and organic carbon assimilation. Under such conditions,
cells exhibit a reduced capacity for photosynthesis but a higher growth
rate, compared to phototrophic cultures. Better understanding of the
down regulation of photosynthesis would enable more efficient conversion
of carbon into valuable products like biofuels. In this study, Flux
Balance Analysis (FBA) and Flux Variability Analysis (FVA) have been
used with a genome scale model of C. reinhardtii to examine changes in
intracellular flux distribution in order to explain their changing
physiology. Additionally, a reaction essentiality analysis was performed
to identify which reaction subsets are essential for a given growth
condition. Our results suggest that exogenous acetate feeds into a
modified tricarboxylic acid (TCA) cycle, which bypasses the CO2
evolution steps, explaining increases in biomass, consistent with
experimental data. In addition, reactions of the oxidative pentose
phosphate and glycolysis pathways, inactive under phototrophic
conditions, show substantial flux under mixotrophic conditions.
Importantly, acetate addition leads to an increased flux through cyclic
electron flow (CEF), but results in a repression of CO2 fixation via
Rubisco, explaining the down regulation of photosynthesis. However,
although CEF enhances growth on acetate, it is not essential-impairment
of CEF results in alternative metabolic pathways being increased. We
have demonstrated how the reactions of photosynthesis interconnect with
carbon metabolism on a global scale, and how systems approaches play a
viable tool in understanding complex relationships at the scale of the
organism.<br>
<br>
<strong>Aurélien Desoeuvre</strong> - <em>Homeostasis by interval</em>.<br>
The presence of parametric uncertainty in biological systems and the
importance of the homeostasis concept in medicine led us to look for a
method to find homeostatic variables of a system. To do this, we use an
algorithmic method based on the Ibex library (Interval Based
EXplorer)(Constraint programming), and a definition of homeostasis on a
equilibrium in terms of intervals.<br>
<br>
<strong>Zachary R Fox</strong> - <em>Optimal Experiment Designs of Signal Activated
Stochastic Gene Expression in S. Cerevisae.</em><br>
Modern biological experiments are complex and gaining quantitative
insight from data collected by such experiments remains a challenge.
Increasingly, computational models of complex stochastic biological
systems are used as a method to understand how a particular system works
and also to make quantitative predictions about how the system will
behave under different conditions. Quantitative predictions allow one to
use models to design experiments for particular goals, such as learning
about model parameters. A classic approach to experiment design is to
use Fisher information, which quantifies the expected information a
particular experiment will reveal about model parameters. The Finite
State Projection based Fisher information was recently developed and
allows one to compute the Fisher information for these systems without
resorting to moment-based computations of the master equation dynamics.
In this work, we use a previously validated stochastic model of stress
response genes in _S. cerevisae_ to design optimal measurements of
mRNA. We validate the Fisher information for a time-varying stochastic
model in the context of the chemical master equation. We then optimize
the number of cells that should be quantified at particular times to
learn as much as possible about the model parameters. We extend the
Fisher information approach to design experiments which minimize the
uncertainty in the extracellular environment - in this case, in the
extracellular salinity. This work demonstrates the potential of
quantitative models to make sense of modern biological data sets and
close the loop between data collection and quantitative modeling.<br>
<br>
<strong>Mathilde Koch</strong>.<em>Large scale active-learning-guided exploration to
maximize cell-free production.</em><br>
Cell-free systems are an increasingly mature and useful platform for
prototyping, testing, and engineering biological parts and systems.
However, lysate-based cell-free systems currently suffer from important
batch-to-batch variability which render quantitative predictions and
mathematical modeling hard to generalise between set-ups. Here we
describe an active learning approach to explore a combinatorial space of
~4,000,000 cell-free compositions, maximizing protein production and
identifying critical parameters involved in cell-free productivity. We
also provide a one-step-method to achieve high quality predictions for
protein production using minimal experimental effort regardless of the
lysate quality.<br>
<br>
<strong>Annick Lesne</strong>. <em>Bifurcation analysis of biological circuits: time
scales matter</em>.<br>
<em>Joint work with Marc-Thorsten Hütt (Jacobs University, Bremen, Germany)
and his former students Pencho Yordanov and Stefka Tyanova.</em><br>
The core of dynamical systems theory is to focus on asymptotic states of
the system and to investigate its phase portrait and its bifurcation
diagram. However, when considering complex biological systems, this
approach could fail, for instance when investigating the effect of
external stimuli on a system displaying several characteristic time
scales. I will present a case study of a system comprising two
interlinked positive feedback loops. Depending on the time scales of
these loops, the system could (or not) be robust with respect to
external noise. When a stimulus with an intermediary time scale is
applied, incomplete bifurcation and stabilization of non-equilibrium
states are observed. I will conclude with some other examples where the
current framework of bifurcation theory is not sufficient to capture the
complexity of the dynamics.<br>
<br>
<strong>P.-E. Meunier and D. Regnault</strong> - <em>Non-cooperatively assembling large
structures.</em><br>
Algorithmic self-assembly is the study of the local, distributed,
asynchronous algorithms ran by molecules to self-organise, in particular
during crystal growth. The general cooperative model, also called
``temperature 2", uses synchronisation to simulate Turing machines,
build shapes using the smallest possible amount of tile types, and other
algorithmic tasks. However, in the non-cooperative (``temperature 1")
model, the growth process is entirely asynchronous, and mostly relies on
geometry. Even though the model looks like a generalisation of finite
automata to two dimensions, its 3D generalisation is capable of
performing arbitrary (Turing) computation, and of universal simulations,
whereby a single 3D non-cooperative tileset can simulate the dynamics of
all possible 3D non-cooperative systems, up to a constant scaling
factor. However, the original 2D non-cooperative model is not capable of
universal simulations, and the question of its computational power is
still widely open and it is conjectured to be weaker than
``temperature 2" or its 3D counterpart. Here, we show an unexpected
result, namely that this model can reliably grow assemblies of diameter
n log(n) with only n tile types, which is the first asymptotically
efficient positive construction.<br>
<br>
<strong>Aurélien Naldi</strong>. <em>Dynamic modeling of cell populations with
UPMaBoSS</em>.<br>
Joint work with: Gautier Stoll (CRC, Paris), Vincent Noel (Curie,
Paris), Eric Viara (Sysra), Emmanuel Barillot (Curie, Paris), Denis
Thieffry (IBENS Paris), Laurence Calzone (Curie, Paris)<br>
Over the last decade, various parts of the immune response have been
studied through qualitative dynamical models. These models focus on
intra-cellular mechanisms, often controlled by external events and
leading to alternative cell fates. However, they do not fully account
for the control of the size and composition of the cell population,
which ultimately determines the nature and intensity of the immune
response.<br>
In this context, UPMaBoSS is a new framework for dynamical modeling of
circulating cell populations, based on qualitative models of the
intra-cellular mechanisms. It relies on the pre-existing tool MaBoSS,
which estimates a distribution of probabilities of individual cellular
states. Here, we propose to interpret this distribution of probabilities
as a mirror of the composition of an heterogeneous cell population.
UPMaBoSS enables this interpretation by accounting for inter-cellular
communication, cell division and cell death. It provides an efficient
and natural method for extending mechanistic models toward the
population scale.<br>
Preliminary studies on models of immune cells confirm that accounting
for the population-level feedbacks can change drastically the results,
in particular in the balance between sub-populations of regulatory T
cells.<br>
<br>
Gregory Nuel. <em>Estimating causal effects in gene regulation networks.</em><br>
In this talk we present a modelization of gene regulation networks based
on causal Gaussian Bayesian networks. In the presence of any arbitrary
mixture of observation (e.g. wild type experiments) and intervention
data (e.g. knock-out experiments), we establish the maximum likelihood
estimator given the directed acyclic graph (DAG) structure as a simple
linear regressor. In a second time, we then use the Metropolis-Hasting
algorithm jointly with a model selection criterion (e.g. BIC) to obtain
the full posterior distribution of DAG structures and parameters. This
collection of Bayesian networks allows to estimate direct and total
causal effects. Finally, we present a DAG clustering algorithm that
might be useful for interpreting and representing the posterior DAG
distribution.<br>
<br>
<strong>Jérémie Pardo</strong> - <em>Sequential reprogramming of biological network
fate.</em><br>
Cell reprogramming consists in modifying gene expression to induce a
particular cell behavior naturally or artificially. A number of
potential beneficial outcomes in the field of médecine such as cancerous
targeted therapy, regenerative or precision medicine could come from
such reprogramming. The action of targeted therapies can be interpreted
as network rewiring as the effect of mutations and drugs can be
described as elementary topological actions on the network, assimilated
to a control. The main issue is to infer the control inputs (i.e.
topological actions) redirecting the biological system dynamics to an
expected fate. Two computational approaches of controls can be studied:
single control or sequential control of the interaction network. In this
talk, we will present a framework investigating the sequential control
of Boolean controlled networks. Control sequence Inference is a decision
problem of PSPACE complexity. Thus, in the aim to find a minimal
parsimonious control sequence, we propose a heuristic method focused on
the partitioning of the states dependent on observed variables and an
abductive-based inference.<br>
<br>
<strong>Diane Peurichard</strong> - <em>A new model for the emergence of vascular
networks.</em> Abstract: The generation of vascular networks is a long
standing problem which has been the subject of intense research in the
past decades, because of its wide range of applications (tissue
regeneration, wound healing, cancer treatments etc). The mechanisms
involved in the formations of vascular networks are complex and despite
the vast amount of research devoted to it there are still many
mechanisms involved which are poorly understood. Our aim is to bring
insight into the study of vascular networks by defining heuristic rules,
as simple as possible, and to simulate them numerically to test their
relevance in the vascularization process. We introduce a hybrid
agent-based/continuum model coupling blood flow, oxygen flow, capillary
network dynamics and tissues dynamics. We provide two different,
biologically relevant geometrical settings and numerically analyze the
influence of each of the capillary creation mechanism in detail. All
mechanisms seem to concur towards a harmonious network but the most
important ones are those involving oxygen gradient and sheer stress.<br>
\</p>
<hr width=100% id=EOF>
<p style=color:#777>Dernière modification le 07/11/2019</p>
</div>
</div>
<nav class=post-pagination>
<a class=newer-posts href=https://www.bioss-cnrs.fr/beta/seminaires/2020-10/>
Suivant<br>Séminaire virtuel: vendredi 2 octobre 2020
</a>
<a class=older-posts href=https://www.bioss-cnrs.fr/beta/manif/nantes_201907/nantes_201907.html>
Précédent<br>Journées Bioss-Médecine-Personalisée
</a>
</nav>
<div class=post-comment-wrapper>
</div>
</div>
</div>
</div>
</div><div id=single-column-footer><p><a href=https://www.bioss-cnrs.fr/beta/index.xml>Flux RSS</a></p>
Basé sur le
Theme Hugo <a href=https://github.com/amazingrise/hugo-theme-diary>Diary</a> par <a href=https://amazingrise.net>Rise</a>
<br>
Adapté de <a href=https://mak1t0.cc/ target=_blank rel="noreferrer noopener">Makito</a>'s <a href=https://github.com/SumiMakito/hexo-theme-journal/ target=_blank rel="noreferrer noopener">Journal.</a> <br>
<br>
&copy;
GT Bioss.
</div>
</div>
<script src=/beta/js/journal.js></script>
</body>
</html>