<!DOCTYPE html>
<html><head>
  <title>Bioss</title>

  <meta name="description" content="Page du groupe de travail sur la
				    biologie systémique symbolique">

  <meta name="keywords" content="Modélisation symbolique de systèmes
				 biologiques, Systèmes dynamiques
				 discrets, Systèmes formels,
				 Langages de modélisation,
				 Sémantique, Vérification de
				 modèles, Réduction, Prédiction
				 sous incertitude, Inférences
				 d'interactions et de règles,
				 Systèmes hybrides">

  <meta http-equiv="content-type" content="text/html; charset=UTF-8">

  <link rel="stylesheet" type="text/css" href="../../style/style.css">
</head>

<body>,&nbsp;
  <div id="main">
    <div id="header">
      <div id="logo">
        <div id="logo_text">
          <!-- class="logo_colour", allows you to change the colour of
          the text -->
          <h1><a href="index.html">
	      <span class="logo_colour">Bioss</span>
	  </a></h1>
          <h2>Groupe de travail sur la biologie systémique symbolique</h2>
        </div>
      </div>
      <div id="menubar">
        <ul id="menu">
          <!-- put class="selected" in the li tag for the selected
          page - to highlight which page you're on -->
          <li><a href="../../index.html">Accueil</a></li>
          <li><a href="../../membres.html">Membres</a></li>
          <li><a href="../../manif.html">Manifestations</a></li>
          <li><a href="../../projets.html">Projets</a></li>
          <li><a href="../../contact.html">Contacts</a></li>
        </ul>
      </div>
    </div>

    <div id="content_header"></div>

    <div id="site_content">

      <!-- <div id="banner"></div> -->

      <div id="sidebar_container">
        <div class="sidebar">
          <div class="sidebar_top"></div>
          <div class="sidebar_item"></div>
          <div class="sidebar_base"></div>
        </div>

        <div class="sidebar">
          <div class="sidebar_top"></div>
          <div class="sidebar_item">
            <h3>Liens utiles</h3>
            <ul>
              <li><a href="http://www.cnrs.fr/">
		  <img src="../../img/logoCNRS.jpg" alt="CNRS"></a></li>
              <li><a href="http://www.gdr-bim.cnrs.fr/">
		  <img src="../../img/logoBIM.jpg" alt="GDR Bioinformatique
						  moléculaire"></a></li>
              <li><a href="https://www.gdr-im.fr/">
		  <img src="../../img/logoIM.jpg" alt="GDR Informatique
						  mathématique"></a></li>
            </ul>
          </div>
          <div class="sidebar_base"></div>
        </div>
      </div>

      <div id="content">
        <!-- insert the page content here -->
        <h1>Journée annuelle du GT Bioss (5ème édition)</h1>


	<h3>Informations générales</h3>
	<p align="justify">
	  <b>Date</b> : le 7 novembre 2019 de 9h30 à 17h30.
	</p>
	<p align="justify">
	  <b>Lieu</b> : Université Paris Diderot, salle 027C, au rez-de-chaussée de la Halle aux
Farines. </br>Plans d'accès <a href="https://campus.univ-paris-diderot.fr/file/7495/download?token=ufRWvV6A">ici</a> et <a href="https://campus.univ-paris-diderot.fr/file/7492/download?token=Og2VmjpP">là</a>.
	</p>

	<p align="justify">
	  <b>Organisateurs</b> : Grégory Batt, Cédric Lhoussaine, Elisabeth Remy et Anne Siegel&nbsp;
	</p>


<p align="justify">
	  Cette cinquième édition des journées annuelles du GT Bioss va
	  se dérouler juste après la journée nationale du <a href="http://www.gdr-bim.cnrs.fr/">GDR BiM</a>
	  qui aura lieu le 6 novembre à l'Université Denis Diderot.
</p>



	<h3>Inscription</h3>
	<p align="justify">
	  L'inscription, gratuite mais obligatoire, se fait via la <a href="http://www.gdr-bim.cnrs.fr/?page_id=160">page du GDR BiM de l'évènement</a>.</br>
	  La SFBI offre des bourses de voyage aux doctorants et postdocs (<b>deadline le candidature le 10 octobre!</b>)!
	</p>

	<h3>Orateurs invités</h3>

  <p align="justify">
	  <a href="http://nuel.perso.math.cnrs.fr/">Gregory Nuel</a>, LPSM (Sorbonne Université). <br/>
    <a href="https://www.lptmc.jussieu.fr/users/lesne">Annick Lesne</a>, LPTMC (Sorbonne Université) et IGMM (Montpellier)<br/>
	</p>


	<h3>Programme</h3> (préliminaire)

	</p>
  <p align="left">
    <b>09h00 - 09h30</b> - <b>Accueil</b><br/>
    <b>09h30 - 09h35</b> - <b>Introduction des journées</b><br/>
    <b>09h35 - 10h20</b> - <b>Conférence plénière</b>. Gregory Nuel. <i>Estimating causal effects in gene regulation networks.</i><br/>
    <b>10h20 - 10h45</b> - Jérémie Pardo.  <i>Sequential reprogramming of biological network fate</i>.<br/>
    <b>10h45 - 11h10</b> - Aurélien Desoeuvre. <i>Homeostasis by interval</i>.<br/>
    <b>11h10 - 11h35</b> - Aurélien Naldi. <i>Dynamic modeling of cell populations with UPMaBoSS.</i><br/>
    <b>11h35 - 12h00</b> - Stephen Chapman. <i>Flux balance analysis reveals acetate metabolism modulates cyclic electron flow and alternative glycolytic pathways in Chlamydomonas reinhardtii.</i><br/>
    <b>12h00 - 12h10</b> - Flash talks. <br/>
    <b>12h10 - 13h30</b> - <b>Pause déjeuner</b><br/>
    <b>13h30 - 14h15</b> - <b>Conférence plénière</b>. Annick Lesne. <i>Bifurcation analysis of biological circuits: time scales matter</i>.<br/>
    <b>14h15 - 14h40</b> - Diane Peurichard. <i>A new model for the emergence of vascular networks.</i><br/>
    <b>14h40 - 15h05</b> - D. Regnault. <i>Non-cooperatively assembling large structures.</i> <br/>
    <b>15h05 - 15h35</b> - <b>Pause</b><br/>
    <b>15h35 - 16h00</b> - Émilie Allart. <i>Computing Difference Abstractions of Metabolic Networks.</i><br/>
    <b>16h00 - 16h25</b> - Andreea Beica. <i>Tropical abstractions of Biochemical Reaction Networks with guarantees.</i><br/>
    <b>16h25 - 16h50</b> - Zach Fox. <i>Optimal Experiment Designs of Signal Activated Stochastic Gene Expression in S. Cerevisae.</i><br/>
    <b>16h50 - 17h15</b> - Mathilde Koch.<i>Large scale active-learning-guided exploration to maximize cell-free production</i>.<br/>
  </p>

	<h3>Résumés</h3>
	<p align="justify">
    <b> Émilie Allart</b> - <i>Computing Difference Abstractions of Metabolic Networks</i>.</br>
    Algorithms based on abstract interpretation were proposed recently for predicting changes of reaction networks with partial kinetic information. Their prediction precision, however, depends heavily on which heuristics are applied in order to add linear consequences of the steady state equations of the metabolic network. In this paper we ask the question whether such heuristics can be avoided while obtaining the highest possible precision. This leads us to the first algorithm for computing the difference abstractions of a linear equation system exactly without any approximation. This algorithm relies on the usage of elementary flux modes in a nontrivial manner, first-order definitions of the abstractions, and finite domain constraint solving.</br><br/>

    <b>Andreea Beica</b> - <i>Tropical abstractions of Biochemical Reaction Networks with guarantees.</i><br/>
    Biochemical molecules interact through modification and binding reactions, giving raise to a combinatorial number of possible biochemical species. The time-dependent evolution of concentrations of the species is commonly described by a system of coupled ordinary differential equations (ODEs). However, the analysis of such high-dimensional, non-linear system of equations is often computationally expensive and even prohibitive in practice. The major challenge towards reducing such models is providing the guarantees as to how the solution of the reduced model relates to that of the original model, while avoiding to solve the original model. In this paper, we have designed and tested an approximation method for ODE models of biochemical reaction systems, in which the guarantees are our major requirement. Borrowing from tropical analysis techniques, dominance relations among terms of each species' ODE are exploited to simplify the original model, by neglecting the dominated terms. As the dominant subsystems can change during the system's dynamics, depending on which species dominate the others, several possible modes exist. Thus, simpler models consisting of only the dominant subsystems can be assembled into hybrid, piecewise smooth models, which approximate the behavior of the initial system. By combining the detection of dominated terms with symbolic bounds propagation, we show how to approximate the original model by an assembly of simpler models, consisting in ordinary differential equations that provide time-dependent lower and upper bounds for the concentrations of the initial models species. Our method provides sound interval bounds for the concentrations of the chemical species, and hence can serve to evaluate the faithfulness of tropicalization-based reduction heuristics for ODE models of biochemical reduction systems. The method is tested on several case studies.<br/><br/>

    <b>Stephen Chapman</b> - <i>Flux balance analysis reveals acetate metabolism modulates cyclic electron flow and alternative glycolytic pathways in Chlamydomonas reinhardtii.</i><br/>
    Cells of the green alga Chlamydomonas reinhardtii cultured in the presence of acetate perform mixotrophic growth, involving both photosynthesis and organic carbon assimilation. Under such conditions, cells exhibit a reduced capacity for photosynthesis but a higher growth rate, compared to phototrophic cultures. Better understanding of the down regulation of photosynthesis would enable more efficient conversion  of carbon into valuable products like biofuels. In this study, Flux Balance Analysis (FBA) and Flux Variability Analysis (FVA) have been used with a genome scale model of C. reinhardtii to examine changes in intracellular flux distribution in order to explain their changing physiology. Additionally, a reaction essentiality analysis was performed to identify which reaction subsets are essential for a given growth condition. Our results suggest that exogenous acetate feeds into a modified tricarboxylic acid (TCA) cycle, which bypasses the CO2 evolution steps, explaining increases in biomass, consistent with experimental data. In addition, reactions of the oxidative pentose phosphate and glycolysis pathways, inactive under phototrophic conditions, show substantial flux under mixotrophic conditions. Importantly, acetate addition leads to an increased flux through cyclic electron flow (CEF), but results in a repression of CO2 fixation via Rubisco, explaining the down regulation of photosynthesis. However, although CEF enhances growth on acetate, it is not essential-impairment of CEF results in alternative metabolic pathways being increased. We have demonstrated how the reactions of photosynthesis interconnect with carbon metabolism on a global scale, and how systems approaches play a viable tool in understanding complex relationships at the scale of the organism. <br/><br/>

    <b>Aurélien Desoeuvre</b> - <i>Homeostasis by interval</i>.<br/>
    The presence of parametric uncertainty in biological systems and the importance
    of the homeostasis concept in medicine led us to look for a method to find
    homeostatic variables of a system. To do this, we use an algorithmic method
    based on the Ibex library (Interval Based EXplorer)(Constraint programming),
    and a definition of homeostasis on a equilibrium in terms of intervals.<br/><br/>

    <b>Zachary R Fox</b> - <i>Optimal Experiment Designs of Signal Activated Stochastic Gene Expression in S. Cerevisae.</i> <br/>
    Modern biological experiments are complex and gaining quantitative insight from data collected by such experiments remains a challenge.
    Increasingly, computational models of complex stochastic biological systems are used as a method to understand how a particular system works and also to make quantitative predictions about how the system will behave under different conditions.
    Quantitative predictions allow one to use models to design experiments for particular goals, such as learning about model parameters.
    A classic approach to experiment design is to use Fisher information, which quantifies the expected information a particular experiment will reveal about model parameters.
    The Finite State Projection based Fisher information was recently developed and allows one to compute the Fisher information for these systems without resorting to moment-based computations of the master equation dynamics.
    In this work, we use a previously validated stochastic model of stress response genes in _S. cerevisae_ to design optimal measurements of mRNA.
    We validate the Fisher information for a time-varying stochastic model in the context of the chemical master equation.
    We then optimize the number of cells that should be quantified at particular times to learn as much as possible about the model parameters.
    We extend the Fisher information approach to design experiments which minimize the uncertainty in the extracellular environment - in this case, in the extracellular salinity.
    This work demonstrates the potential of quantitative models to make sense of modern biological data sets and close the loop between data collection and quantitative modeling.<br/><br/>

<b>Mathilde Koch</b>.<i>Large scale active-learning-guided exploration to maximize cell-free production.</i><br/>
Cell-free systems are an increasingly mature and useful platform for prototyping, testing, and engineering biological parts and systems. However, lysate-based cell-free systems currently suffer from important batch-to-batch variability which render quantitative predictions and mathematical modeling hard to generalise between set-ups. Here we describe an active learning approach to explore a combinatorial space of ~4,000,000 cell-free compositions, maximizing protein production and identifying critical parameters involved in cell-free productivity. We also provide a one-step-method to achieve high quality predictions for protein production using minimal experimental effort regardless of the lysate quality. <br/><br/>

    <b>Annick Lesne</b>. <i>Bifurcation analysis of biological circuits: time scales matter</i>.<br/>
    <i>Joint work with Marc-Thorsten Hütt (Jacobs University, Bremen, Germany) and his former students Pencho Yordanov and Stefka Tyanova.</i><br/>
    The core of dynamical systems theory is to focus on asymptotic states of the system and to investigate its phase portrait and its bifurcation diagram. However, when considering complex biological systems, this approach could fail, for instance when investigating the effect of external stimuli on a system displaying several characteristic time scales. I will  present a case study of a system comprising two interlinked positive feedback loops. Depending on the time scales of these loops,  the system could (or not) be robust with respect to external noise. When a stimulus  with an intermediary time scale is applied, incomplete bifurcation and stabilization of non-equilibrium states are observed. I will conclude with some other examples where the current framework of bifurcation theory is not sufficient to capture the complexity of the dynamics.<br/><br/>

    <b>P.-E. Meunier and D. Regnault</b> - <i>Non-cooperatively assembling large structures.</i><br/>
    Algorithmic self-assembly is the study of the local, distributed, asynchronous algorithms ran by molecules to self-organise, in particular during crystal growth. The general cooperative model, also called ``temperature 2", uses synchronisation to simulate Turing machines, build shapes using the smallest possible amount of tile types, and other algorithmic tasks.
    However, in the non-cooperative (``temperature 1") model, the growth process is entirely asynchronous, and mostly relies on geometry. Even though the model looks like a generalisation of finite automata to two dimensions, its 3D generalisation is capable of performing arbitrary (Turing) computation, and of universal simulations, whereby a single 3D non-cooperative tileset can simulate the dynamics of all possible 3D non-cooperative systems, up to a constant scaling factor.
    However, the original 2D non-cooperative model is not capable of universal simulations, and the question of its computational power is still widely open and it is conjectured to be weaker than ``temperature 2" or its 3D counterpart. Here, we show an unexpected result, namely that this model can reliably grow assemblies of diameter n log(n) with only n tile types, which is the first asymptotically efficient positive construction.<br/><br/>

    <b>Aurélien Naldi</b>. <i>Dynamic modeling of cell populations with UPMaBoSS</i>.<br/>
    Joint work with: Gautier Stoll (CRC, Paris), Vincent Noel (Curie, Paris), Eric Viara (Sysra), Emmanuel Barillot (Curie, Paris), Denis Thieffry (IBENS Paris), Laurence Calzone (Curie, Paris)<br/>
    Over the last decade, various parts of the immune response have been studied through
    qualitative dynamical models. These models focus on intra-cellular mechanisms, often
    controlled by external events and leading to alternative cell fates. However, they do not
    fully account for the control of the size and composition of the cell population, which
    ultimately determines the nature and intensity of the immune response.<br>
    In this context, UPMaBoSS is a new framework for dynamical modeling of circulating cell
    populations, based on qualitative models of the intra-cellular mechanisms. It relies on the
    pre-existing tool MaBoSS, which estimates a distribution of probabilities of individual cellular
    states. Here, we propose to interpret this distribution of probabilities as a mirror of the
    composition of an heterogeneous cell population. UPMaBoSS enables this interpretation by
    accounting for inter-cellular communication, cell division and cell death. It provides an efficient
    and natural method for extending mechanistic models toward the population scale.<br/>
    Preliminary studies on models of immune cells confirm that accounting for the population-level feedbacks can change drastically the results, in particular in the balance between sub-populations of regulatory T cells.<br/><br/>

    Gregory Nuel. <i>Estimating causal effects in gene regulation networks.</i><br/>
    In this talk we present a modelization of gene regulation networks based on causal Gaussian Bayesian networks. In the presence of any arbitrary mixture of observation (e.g. wild type experiments) and intervention data (e.g. knock-out experiments), we establish the maximum likelihood estimator given the directed acyclic graph (DAG) structure as a simple linear regressor. In a second time, we then use the Metropolis-Hasting algorithm jointly with a model selection criterion (e.g. BIC) to obtain the full posterior distribution of DAG structures and parameters. This collection of Bayesian networks allows to estimate direct and total causal effects. Finally, we present a DAG clustering algorithm that might be useful for interpreting and representing the posterior DAG distribution.<br/><br/>

    <b>Jérémie Pardo</b> - <i>Sequential reprogramming of biological network fate.</i><br/>
Cell reprogramming consists in modifying gene expression to induce a particular cell behavior naturally or artificially. A number of potential beneficial outcomes in the field of médecine such as cancerous targeted therapy, regenerative or precision medicine could come from such reprogramming. The action of targeted therapies can be interpreted as network rewiring as the effect of mutations and drugs can be described as elementary topological actions on the network, assimilated to a control. The main issue is to infer the  control inputs (i.e. topological actions) redirecting  the biological system dynamics to an expected fate. Two computational approaches of controls can be studied: single control or sequential control of the interaction network. In this talk, we will present a framework investigating the sequential control of Boolean controlled networks. Control sequence Inference is a decision problem of PSPACE complexity. Thus, in the aim to find a minimal parsimonious control sequence, we propose a heuristic method focused on the partitioning of the states dependent on observed variables and an abductive-based inference.<br/><br/> 

    <b>Diane Peurichard</b> - <i>A new model for the emergence of vascular networks.</i>
    Abstract: The generation of vascular networks is a long standing problem which has been the subject of intense research in the past decades, because of its wide range of applications (tissue regeneration, wound healing, cancer treatments etc). The mechanisms involved in the formations of vascular networks are complex and despite the vast amount of research devoted to it there are still many mechanisms involved which are poorly understood. Our aim is to bring insight into the study of vascular networks by defining heuristic rules, as simple as possible, and to simulate them numerically to test their relevance in the vascularization process. We introduce a hybrid agent-based/continuum model coupling blood flow, oxygen flow, capillary network dynamics and tissues dynamics.  We provide two different, biologically relevant geometrical settings and numerically analyze the influence of each of the capillary creation mechanism in detail. All mechanisms seem to concur towards a harmonious network but the most important ones are those involving oxygen gradient and sheer stress.<br/><br/>
	</p>


	<p style="text-indent:2em" align="justify"></p>
      </div>

      <!-- <div id="banner"></div> -->

    </div>

    <div id="content_footer"></div>

    <div id="footer">
      <p><a href="../../index.html">Accueil</a>
      | <a href="../../membres.html">Membres</a>
      | <a href="../../manif.html">Manifestations</a>
      | <a href="../../projets.html">Projets</a>
      | <a href="../../contact.html">Contact</a></p>
      <p><a href="http://validator.w3.org/check?uri=referer">HTML5</a>
      | <a href="http://jigsaw.w3.org/css-validator/check/referer">CSS</a>
      | <a href="http://www.html5webtemplates.co.uk">HTML5 Web
      Templates</a></p>
    </div>
  </div>


</body></html>
