<!DOCTYPE html>
<html><head>
  <title>Bioss</title> 

  <meta name="description" content="Page du groupe de travail sur la
				    biologie systémique symbolique">
				    
  <meta name="keywords" content="Modélisation symbolique de systèmes
				 biologiques, Systèmes dynamiques
				 discrets, Systèmes formels,
				 Langages de modélisation,
				 Sémantique, Vérification de
				 modèles, Réduction, Prédiction
				 sous incertitude, Inférences
				 d'interactions et de règles,
				 Systèmes hybrides">

  <meta http-equiv="content-type" content="text/html; charset=UTF-8">

  <link rel="stylesheet" type="text/css" href="../../style/style.css">
</head>

<body>,&nbsp;
  <div id="main">
    <div id="header">
      <div id="logo">
        <div id="logo_text">
          <!-- class="logo_colour", allows you to change the colour of
          the text -->
          <h1><a href="index.html">
	      <span class="logo_colour">Bioss</span>
	  </a></h1>
          <h2>Groupe de travail sur la biologie systémique symbolique</h2>
        </div>
      </div>
      <div id="menubar">
        <ul id="menu">
          <!-- put class="selected" in the li tag for the selected
          page - to highlight which page you're on -->
          <li><a href="../../index.html">Accueil</a></li>
          <li><a href="../../membres.html">Membres</a></li>
          <li><a href="../../manif.html">Manifestations</a></li>
          <li><a href="../../projets.html">Projets</a></li>
          <li><a href="../../contact.html">Contacts</a></li>
        </ul>
      </div>
    </div>

    <div id="content_header"></div>

    <div id="site_content">

      <!-- <div id="banner"></div> -->

      <div id="sidebar_container">
        <div class="sidebar">
          <div class="sidebar_top"></div>
          <div class="sidebar_item"></div>
          <div class="sidebar_base"></div>
        </div>

        <div class="sidebar">
          <div class="sidebar_top"></div>
          <div class="sidebar_item">
            <h3>Liens utiles</h3>
            <ul>
              <li><a href="http://www.cnrs.fr/">
		  <img src="../../img/logoCNRS.jpg" alt="CNRS"></a></li>
              <li><a href="http://www.gdr-bim.cnrs.fr/">
		  <img src="../../img/logoBIM.jpg" alt="GDR Bioinformatique 
						  moléculaire"></a></li>
              <li><a href="https://www.gdr-im.fr/">
		  <img src="../../img/logoIM.jpg" alt="GDR Informatique 
						  mathématique"></a></li>
            </ul>
          </div>
          <div class="sidebar_base"></div>
        </div>
      </div>

      <div id="content">
        <!-- insert the page content here -->
        <h1>Journées Bioss-Médecine-Personalisée</h1>
	
	<h3>Informations générales</h3> 
	<p align="justify">
	  <b>Date</b> : 1er et 2 juillet (matin) 2019
	</p>
	<p align="justify">
	  <b>Lieu</b> : Salles A-B-C, Bâtiment 34, LS2N - Laboratoire des Sciences du Numérique de Nantes, Université de Nantes Faculté des Sciences et Techniques, 2 Chemin de la Houssinière, 44322 Nantes
	</p>
	
	<p align="justify">
	  <b>Organisatrice</b> : Carito GUZIOLOWSKI &nbsp;
	</p>

	<p align="justify">
	  <b>Presentation</b> : 
	The objective of the "BIOSS Personalized Medicine" meeting is to present and discuss informally numerical and mathematical modeling frameworks applied to understand medically important Human states. Talks are reassembled on discussing results that combine networks and (mathematical, probabilistic, logic, machine learning, among others) models. These models, integrate experimental or clinical observations and propose a computable representation of Human cellular, tissues, and clinical states related to unhealthy behaviors and cellular differentiation.
	</p>
 
	<h4>Lundi 1er Juillet</h4>
	<p align="left">
	  <b>09h00 - 09h10</b>  - <b>Accueil des participants</b><br>
	  <b>09h10 - 09h15</b>  - <b>Ouverture</b><br>
	  <b>09h15 - 09h35</b> - Maxime Folschette (LS2N, Nantes) - <i>Search of Therapeutic Targets on the Hepatocellular Carcinoma with Database Extraction and Graph Coloring Methods.</i><br> 
	  <b>09h40 - 10h00</b> - Lokmane Chebouba (LS2N, Nantes) - <i>Proteomics measurements combined with constraint programming for predicting treatment response in Acute Myeloid Leukemia cancer case.</i><br> 
	  <b>10h00 - 10h45</b> - Benno Schwikowski (Institut Pasteur, Paris) -
	  <i>Interpretable machine learning to discover and map physiological activity using omics data.</i> <br>
	  
	  <b>10h45 - 11h15</b> - <b>Pause café</b><br>
	  <b>11h15 - 12h00</b> - Diana Mateus (LS2N, Nantes) - <i>Prognosis Prediction of Myeloma Patients with Random Survival Forests.</i>
	  <br>
	  <b>12h00 - 12h45</b> - Herve Isambert (Institut Curie, Paris) - <i>Learning clinical networks from medical records based on information estimates in mixed-type data</i> <br>
	  
	  <b>12h45 - 14h00 - Pause déjeuner</b><br><br>
	  <b>14h00 - 14h45</b> - Denis Thieffry (IBENS, Paris) - <i>Cooperation between T cell receptor and Toll-like receptor 5 signaling for CD4+ T cell activation.</i> <br> 
	  <b>14h45 - 15h30</b> - Samuel Chaffron (LS2N, Nantes) - <i>Human gut microbiome co-activity networks in heath and disease.</i> <br>
	  <b>15h30 - 15h45</b> - <b>Pause café</b><br>
 	  <b>15h45 - 16h30</b> - Laurence Calzone (Institut Curie, Paris) - <i>Une méthodologie de personalisation des modèles Booléens pour tester des inhibiteurs, simples ou doubles, avec des réponses qui varient selon les profils de patients.</i><br>
	  <b>16h30 - 17h15</b> - Celia Biane (IRISA, Rennes) - <i>Different approaches for the identification of perturbations in Boolean networks</i><br> 	  
	</p>

	<h4>Mardi 2 Juillet</h4>
	<p align="left">
	  <b>09h00 - 09h15</b>  - <b>Accueil des participants</b><br>
	  <b>09h15 - 10h00</b> - Loic Paulevé (LaBRI, Bordeaux) - <i>Most Permissive Boolean Networks: Application to Inference of Models of Cellular Differentiation</i><br> 
	  <b>10h00 - 10h45</b> - Dimitri Meistermann (CRTI, LS2N, Nantes) - <i>The limit of cell specification concept: a lesson from scRNA-Seq on early human development.</i> <br>
	  
	  <b>10h45 - 11h15</b> - <b>Pause café</b><br>
	  <b>11h15 - 12h00</b> - Vera Pancaldi (CRCT, Toulouse) - <i>Quantification of tumour-infiltrating immune cells and beyond: modelling of cellular interactions in the tumour micro-environment.</i>
	  <br>
	  <b>12h00 - 12h15</b> - <b>Clôture et annonces</b>
	</p>

	</br>	
      <!-- <div id="banner"></div> -->
	<h3>Abstracts</h3>
	<p> <b>Diana Mateus</b> - <i>Prognosis Prediction of Myeloma Patients with Random Survival Forests.</i> <br/>
	Multiple myeloma (MM) is a bone marrow cancer that accounts for 10\% of all hematological malignancies. FDG PET Quantitative imaging has great importance for its treatment protocol guidance. In this study, we aim to develop a computer-assisted method based on PET imaging features towards assisting personalized diagnosis and treatment decisions for MM patients. We consider texture-based (radiomics) features on top of conventional (e.g. SUVmax) and clinical biomarkers, resulting in a large input/feature vector. Our proposed model relies on a two-stage Random Survival Forest (RFS) for both feature selection and prediction. The targeted variable for prediction is the progression-free survival(PFS), that is, the period of time until the first progression or relapse. We demonstrate the performance of the proposed approach in terms of C-index and final prognosis separation on a database of 66 patients who were part of the prospective multi-centric french IMAJEM study. Our results confirm the predictive value of radiomics for MM patients. Indeed, quantitative/heterogeneity image-based features reduce the error of the predicted progression.
	</p>
	<p> <b>Celia Biane</b> - <i>Different approaches for the identification of perturbations in Boolean networks.</i> </br/>
	Boolean networks are discrete dynamical systems that are increasingly used in the field of systems biology to understand how complex cellular behaviors (phenotypes) emerge from the interaction of their molecular components. In this context, the interacting elements of the network represent diverse molecules whose local Boolean state is influenced by the state of other elements of the network, and the asymptotic states of the network represent the phenotype. During the last few years, different modelling and algorithmic approaches have been proposed for the identification of sets of local perturbations leading to a goal asymptotic behavior. During this presentation, I will propose different criteria of comparison of these approaches, show their application on a published model of bladder cancer and discuss their interpretation in the context of personalized/precision medicine.
	</p>
	<p>
	<b>Benno Schwikowski</b> - <i>Interpretable machine learning to discover and map physiological activity using omics data.</i><br/>
The activation or deactivation of most physiological processes in health and disease can be expected to be reflected in coordinated changes at the molecular level. The discovery and mapping of physiological processes from transcriptomic data can thus be attempted, for example, using models that are based on the quantification of single RNAs, or linear combinations thereof. The underlying biology reality is often likely to be more complex than this, but limited data availability and limited computational resources make it difficult to go beyond these simple models while preserving statistical power and interpretability of the results.<br/>
In my talk, I will discuss two instances of new and carefully calibrated data analysis approaches that allowed us to discover and validate previously unknown associations between transcriptomic data and biomedically relevant physiology. Both models are highly interpretable and generic enough to be applied to a wide range omics analysis scenarios.<br/>
References: Gwinner et al. (2017), https://doi.org/10.1093/bioinformatics/btw676, and Nikolayeva et al. (2018), https://doi.org/10.1093/infdis/jiy086
	</p>
	<p><b>Loic Paulevé</b> - <i>Most Permissive Boolean Networks: Application to Inference of Models of Cellular Differentiation</i><br/>
	Boolean networks are a commonly used framework to model dynamics of large-scale interaction networks. They aim at enabling to reason on temporal behaviours of networks without requiring precise knowledge on kinetics and interaction thresholds.<br/>
However, their usual interpretation can lead to wrong conclusions on their capability to reach certain behaviours. More precisely, refining a Boolean network model (with multivalued or ODEs for instance) can restrict some behaviours, but also create new ones, which are not predicted at the Boolean level.<br/>
This is problematic when inferring networks at the Boolean level, as it leads to reject actually valid models, hence introducing bias in the analysis of candidate models of cellular processes.<br/>
We introduce a new interpretation of Boolean networks which fixes this issue: with Most Permissive Boolean Networks, it is guaranteed that model refinements only restrict the capabilities of the model, thus allowing a correct abstract reasoning.<br/>
Moreover, Most Permissive Boolean Networks are also much more tractable to analyse and do not suffer from the state space explosion.<br/>
We illustrate their application to a scalable inference of models of cellular differentiation, which involve thorough constraints on the global dynamics of the network.
	</p>
	<p> <b>Dimitri Meistermann</b> - <i>The limit of cell specification concept: a lesson from scRNA-Seq on early human development.</i> (joint work with Sophie Loubersac, Arnaud Reignier, Valentin Francois-Campion, Thomas Fréour, Jérémie Bourdon and Laurent David)<br>
Recent technological advances such as single-cell RNAseq have allowed an unprecedented access into processes orchestrating human preimplantation development [1, 2]. However, the sequence of events which occur during human preimplantation development are still unknown. In particular, timing of the very first human lineage specification remains elusive. During this event, the morula cells are can acquire two fates: the trophectoderm that will give rise the placenta and inner cell mass that will give rise the fetus. We present a human preimplantation development model based on transcriptomic pseudotime modelling of four scRNAseq dataset, biologically validated by spatial information and precise time-lapse staging. In contrast to mouse [3], we show that trophectoderm / inner cell mass lineage specification in human is only detectable at the transcriptomic level at the blastocyst stage, just prior to expansion. By studying this delay, we show that cellular specification is a time window that begins with the establishment of cellular junctions, which polarize the embryo. These are the first factors that discriminates the two cell fates. The cell specification ends with the divergence of transcriptome profiles. For identifying the precise timings of this divergence, we have coupled the pseudotime modelling from Monocle2 [4] with several other tools. First, we performed an estimation of RNA velocity with velocyto [5]. This tool can retrieve the genes that are going to be down or upregulated in each cell, by processing the intron data that are contained into scRNAseq reads. We used WGCNA [6] for describing the waves of genes that paces human preimplantation development. By combining these tools, we found novel markers, validated by immunofluorescences. Their expression profile enables a precise staging of human preimplantation embryos, such as IFI16 which highlights establishment of epiblast and NR2F2 which appears at the transition from specified to mature trophectoderm. Strikingly, mature trophectoderm cells arise from the polar side, just after specification, supporting a model of polar trophectoderm cells driving trophectoderm maturation. Altogether, our study unravels the first lineage specification event in the human embryo and provides a browsable resource, based on d3.js, for mapping spatio-temporal events underlying human lineage specification.<br/>
References<br/>
[1] L. Yan et al., « Single-cell RNA-Seq profiling of human preimplantation embryos and embryonic stem cells », Nature Structural and Molecular Biology, vol. 20, no 9, p. 1131, sept. 2013.<br/>
[2] S. Petropoulos et al., « Single-Cell RNA-Seq Reveals Lineage and X Chromosome Dynamics in Human Preimplantation Embryos », Cell, vol. 165, no 4, p. 1012‐1026, mai 2016.<br/>
[3] E. Posfai et al., « Position- and Hippo signaling-dependent plasticity during lineage segregation in the early mouse embryo », eLife, vol. 6.<br/>
[4] C. Trapnell et al., « The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells », Nature Biotechnology, vol. 32, no 4, p. 381‐386, mars 2014.<br/>
[5] G. La Manno et al., « RNA velocity of single cells », Nature, vol. 560, no 7719, p. 494‐498, août 2018.<br/>
[6] P. Langfelder et S. Horvath, « WGCNA: an R package for weighted correlation network analysis », BMC
Bioinformatics, vol. 9, p. 559, 2008.<br/>
	</p>
<p> <b>Maxime Folschette </b> - <i> Hepatocellular carcinoma (HCC) is the most widespread and lethal type of liver cancer today. Understanding the causes of its proliferation is thus a major challenge.</i><br/>
In this work, we extract new biological regarding HCC proliferation based on a signaling network and partial observations of its components. Our network is extracted from Kegg, although Pathway Commons and other databases are also candidates. The observations come from a differential analysis of gene expression between invasive and non-invasive tumor tissues. Based on this initial data, we run a prediction algorithm called Iggy which extracts new knowledge when the observations are sufficient. The results illustrate the statistical precision of our computational predictions and exposes new knowledge concerning the activity of three protein-complexes (NFKB1::BCL3, NFKB2::RELB and JUND::NACA) which are validated through functional analyses and literature review on HCC.</p>
<p> Otoniel Rodríguez-Jorge, Linda A. Kempis-Calanis, Wassim Abou-Jaoudé, Darely Y. Gutiérrez-Reyna, Céline Hernandez, Oscar Ramirez-Pliego, Morgane Thomas-Chollier, Salvatore Spicuglia, Maria A. Santana, <b>Denis Thieffry</b> - <i>Cooperation between T cell receptor and Toll-like receptor 5 signaling for CD4+ T cell activation</i><br/>
CD4+ T cells recognize antigens through their T cell receptors. However, additional signals involving co-stimulatory receptors, for example CD28, are required for proper T cell activation. Alternative co-stimulatory receptors have been proposed, including members of the Toll-like receptor family, such as TLR5 and TLR2. However, the molecular mechanism underlying this co-stimulatory function has not yet been fully elucidated.<br/> 
Here, we report the generation of detailed molecular maps and logical models for the T cell receptor (TCR) and Toll-like receptor (TLR5) signalling pathways, along with a merged model accounting for cross-interactions. Furthermore, we validated the resulting model by analysing the responses of T cells to the activation of these pathways alone or in combination, in terms of CREB, AP-1 (c-Jun) and NF-kB (p65) activation. <br/>
Our merged model accurately reproduces the experimental results, showing that the activation of TLR5 can play a similar role to that of CD28, regarding AP-1, CREB and NF-кB activation, thereby, providing novel insights regarding cross-regulations of these pathways in CD4+ T cells.</p>
<p> <b>Samuel Chaffron</b> - <i>Human gut microbiome co-activity networks in heath and disease</i><br/>
Microbial communities inhabiting our intestinal tract impact and influence our nutrition, immunity and development. Today, High-Throughput Sequencing and functional genomics are revealing the under-explored diversity and complexity of these microbial ecosystems. Limited by the fact that most microbes can hardly be isolated and cultivated in lab-controlled environments, we are just starting to grasp the complexity and diversity of microbial interactions. Even when successful, laboratory experiments inherently lose valuable information about the richness and diversity of community functioning and interactions in situ. Today, large scale environmental surveys of microbial communities across Earth's ecosystems (e.g. Tara Oceans expeditions, integrative Human Microbiome Project) gathered large volumes of meta-omic and contextual data that are enabling the reconstruction of genomes of uncultivated microbial species or Metagenome-Assembled Genomes (MAGs). While classical co-occurrence analyses enable to predict interactions between newly identified microbes, these approaches are inherently limited since true biotic interactions can hardly be disentangle from abiotic (environmental) effects. Here, we developed a trait-based approach to enrich co-occurring information and uncover putative biotic interactions among human gut MAGs. Genomic and growth traits can directly be inferred from MAGs and meta-omics data. Here, co-growth signals across individuals are used to reveal positive and negative putative interactions between co-occurring microbes. In addition, the functional content of MAGs and the reconstruction of their metabolism will be used to predict and model potential microorganismsâ€™ dependencies. Inferring and combining (meta-)genomic traits in a global approach can help to identify consortia of microbes and pave the way towards the functional understanding and the metabolic modeling of their interactions in health and disease.</p>
<p><b>Laurence Calzone</b> - <i>Une méthodologie de personalisation des modèles Booléens pour tester des inhibiteurs, simples ou doubles, avec des réponses qui varient selon les profils de patients</i><br/>
Logical models of cancer pathways are typically built by mining the literature for relevant experimental observations or by inquiring pathway databases. They are usually generic as they apply for large cohorts of individuals. As a consequence, they generally do not capture the heterogeneity of patient tumours and their therapeutic responses. After introducing our approach for constructing logical models and simulating them stochastically, I will present the methodology for personalising logical models to data and show how these models can be used for testing the effect of drugs.
</p>
<p><b>Lokmane Chebouba</b> -<i>Proteomics measurements combined with constraint programming for predicting treatment response in Acute Myeloid Leukemia cancer case</i><br/>
The use of data from high-throughput technologies to target drugs has been widespread in recent decades. Several approaches have been applied to biomedical data to detect disease-specific proteins and genes to better target drugs. We propose a new method for discriminating the response of patients with acute myeloid leukemia (AML) to treatments. The proposed approach uses proteomic data and the prior knowledge network to predict the results of cancer treatment by discovering the different Boolean networks specific to each type of treatment response.<br/>
The results are encouraging and demonstrate the benefit of our approach to distinguish patient groups with different response to treatment. In particular each treatment response group is characterized by a predictive model in the form of a signaling Boolean network. This model describes regulatory mechanisms which are specific to each response group. This mechanistic and predictive model also allows us to classify new patients data into the two different patient response groups.
</p>
    </div>
      </div>

    <div id="content_footer"></div>

    <div id="footer">
      <p><a href="../../index.html">Accueil</a>
      | <a href="../../membres.html">Membres</a>
      | <a href="../../manif.html">Manifestations</a>
      | <a href="../../projets.html">Projets</a>
      | <a href="../../contact.html">Contact</a></p>
      <p><a href="http://validator.w3.org/check?uri=referer">HTML5</a>
      | <a href="http://jigsaw.w3.org/css-validator/check/referer">CSS</a>
      | <a href="http://www.html5webtemplates.co.uk">HTML5 Web
      Templates</a></p>
    </div>
  </div>


</body></html>
