---
date: 2021-10-01
title: "S√©minaire virtuel: vendredi 1er octobre 2021"
summary: "[Laura Cantini](https://sites.google.com/view/laura-cantini/home) (IBENS / ENS Paris) et [Sergio Corridore](https://www.linkedin.com/in/sergio-corridore-678ab675/) (Institut Curie / Paris)"
categories: "S√©minaires"
---

[Lien Zoom](https://u-bordeaux-fr.zoom.us/j/86764096440?pwd=b01qOG04RTMvRWNOVHBYR1ZIbkVaUT09)
* Meeting ID: 867 6409 6440
* passcode: 149120 

[üé¶ Enregistrement
zoom](https://u-bordeaux-fr.zoom.us/rec/share/AtGgDR0G-3Y0V13XDFDxFZd_QLPbFe8fEu26LrnFhZdqTfVXFHRz024tTVFGDKlf.zdz6ydmEfNX-Ec7s) (jusqu'au 30 octobre)
<!--- X9aw9@T= -->

### 13h00 - 13h30 -- Laura Cantini (IBENS / ENS Paris)

**Multi-omics integration: towards a comprehensive view of cancer**

Due to the advent of high-throughput technologies, high-dimensional ‚Äúomics‚Äù data are produced at an increasing pace. In cancer biology, national and international consortia have profiled thousands of tumors at multiple molecular levels (‚Äúmulti-omics‚Äù) allowing to gather a comprehensive molecular picture of this disease. Moreover, multi-omics profiling approaches are currently being transposed at single-cell resolution, further increasing the information accessible from cancer samples. The current main challenge is to design appropriate methods to integrate this wealth of information and translate it into actionable biological knowledge.

In this talk, I will discuss two maincomputational directions for multi-omics integration: (i) multilayer networks to integrate a large range of interactions and (ii) joint dimensionality reduction to extract biological knowledge simultaneously from multiple omics. First, I will present their application on bulk data and then I will discuss our ongoing research in single-cell.

Selected associated publications & preprints
- Cantini L, Medico E, Fortunato S, Caselle M. Detection of gene communities in multi-networks reveals cancer drivers. Scientific reports. 2015 Dec 7;5(1):1-0.
- Cantini, L., Zakeri, P., Hernandez, C., Naldi, A., Thieffry, D., Remy, E., Baudot, A., 2021. Benchmarking joint multi-omics dimensionality reduction approaches for the study of cancer. Nature Communications 12.
- Kang Y, Thieffry D, Cantini L. Evaluating the reproducibility of single-cell gene regulatory network inference algorithms. Frontiers in genetics. 2021 Mar 22;12:362.
- Huizing GJ, Peyr√© G, Cantini L. Optimal Transport improves cell-cell similarity inference in single-cell omics data. bioRxiv. 2021 Jan 1.

### 13h30 - 14h00 -- Sergio Corridore (Institut Curie / Paris)

**Temozolomide cellular Pharmacokinetic/ Pharmacodynamic Model in the context of Brain Tumour.**

Glioblastoma multiforme (GBM) is the most frequent and aggressive type of primary brain tumours in adults. Despite very intensive treatments including maximal safe neurosurgery, radiation therapy and chemotherapy, the prognosis of GBM patients remains poor with a median overall survival below 18 months. Temozolomide (TMZ)-based chemotherapy is the most com- mon pharmacological treatment in patients with diagnosed GBM. Even if TMZ administration improves patient overall survival, prognosis remains poor [8] and no major therapeutic advance has been accomplished within the past 10 years. This can be related to a lack of knowledge in how the tumour evolves and ultimately escape drug activity, especially in a context of large inter-patient variability.
New systems pharmacology approaches combining experimental and mathematical expertise provide interesting perspectives towards the design of safe and ecient TMZ-based therapies against GBM [2, 4]. The present study aims to do so through the conception of a model of TMZ pharmacokinetics-pharmacodynamics (PK-PD) and of key regulatory networks, capable of reproducing the intracellular events from TMZ exposure to cell rescue or apoptosis. TMZ is a methylating agent that creates lesions on the DNA after a two-step activation process [7]. Four types of DNA adducts are formed upon drug exposure, which are handled either by O6- methylguanine-DNA methyltransferase (MGMT) or by the base excision repair (BER) system [9, 10]. If DNA repair is unsuccessful, DNA single- or double-strand breaks are created, which triggers Homologous Recombination (HR), ATR/Chk1 and p53 activation, cell cycle arrest and possibly apoptosis [3].
We designed a model, based on ordinary dierential equations, that recapitulates these intra- cellular events. Then, model calibration consisted in a modied least square approach ensuring data best-t satised biologically-sound constraints, the numerical minimization problem being performed by the Covariance Matrix Evolutionary Strategy (CMAES) algorithm. The model was able to reproduce multi-type datasets of several independent studies mostly performed in either the U87 or LN229 glioblastoma cell lines [1, 6, 5]. These datasets included longitudinal and dose-dependent studies of TMZ cellular PK, DNA adduct formation, ATR, Chk1 and p53 phosphorylation, and cell death. This calibrated PK-PD model is currently being used as a powerful tool to investigate new therapeutic targets. Drug combinations involving TMZ and one to three targeted therapies are explored, among which clinically available inhibitors of ATR (e.g. Berzosertibe), PARP (e.g. olaparib) or Cyclin Dependent Kinase4/6 (e.g. palbociclib). The next step will imply a partial re-calibration of the model with multi-omics datasets available for GBM patient-derived cell lines or GBM patient samples, towards a mechanism-based personalization of GMB treatment.

References
1.  Dorthe Aasland et al. Temozolomide induces senescence and repression of DNA repair pathways in glioblastoma cells via activation of ATR{CHK1, p21, and NF-B". In: Cancer research 79.1 (2019), pp. 99-113.
2. A Ballesta et al. Multiscale Design of Cell-Type{Specic Pharmacokinetic/Pharmacodynamic Models for Personalized Medicine: Application to Temozolomide in Brain Tumors". In: CPT: pharmacometrics & systems pharmacology 3.4 (2014), pp. 1-11.
3. Simona Caporali et al. DNA damage induced by temozolomide signals to both ATM and ATR: role of the mismatch repair system". In: Molecular pharmacology 66.3 (2004), pp. 478-491.
4. Jeremy ZR Han et al. Personalized Medicine for Neuroblastoma: Moving from Static Geno- types to Dynamic Simulations of Drug Response". In: Journal of Personalized Medicine 11.5 (2021), p. 395.
5. Yang He and Bernd Kaina. Are there thresholds in glioblastoma cell death responses triggered by temozolomide?" In: International journal of molecular sciences 20.7 (2019), p. 1562.
6. Christopher B Jackson et al. Temozolomide sensitizes MGMT-decient tumor cells to ATR inhibitors". In: Cancer research 79.17 (2019), pp. 4331-4338.
7. Steve Quiros, Wynand P Roos, and Bernd Kaina. Processing of O6-methylguanine into DNA double-strand breaks requires two rounds of replication whereas apoptosis is also induced in subsequent cell cycles". In: Cell cycle 9.1 (2010), pp. 168-178.
8. Sarah Smalley, Anthony J Chalmers, and Simon J Morley. mTOR inhibition and levels of the DNA repair protein MGMT in T98G glioblastoma cells". In: Molecular cancer 13.1 (2014), pp. 1-11.
9. Anish Thomas et al. Temozolomide in the era of precision medicine". In: Cancer research 77.4 (2017), pp. 823-826.
10. J Lee Villano, Tara E Seery, and Linda R Bressler. Temozolomide in malignant gliomas: current use and future targets". In: Cancer chemotherapy and pharmacology 64.4 (2009), pp. 647-655.
