<!doctype html><html><head>
<title>Journées annuelles du GT Bioss (2017, 3e édition)</title>
<meta charset=utf-8>
<meta name=X-UA-Compatible content="IE=edge">
<meta name=google-site-verification content>
<meta content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=0" name=viewport>
<meta content="telephone=no" name=format-detection>
<meta name=description content>
<meta name=renderer content="webkit">
<meta name=theme-color content="#ffffff">
<meta property="og:title" content="Journées annuelles du GT Bioss (2017, 3e édition)">
<meta property="og:description" content="Les 13 et 14 mars 2017 à Montpellier">
<meta property="og:type" content="article">
<meta property="og:url" content="https://www.bioss-cnrs.fr/beta/manif/jnbioss_201703/jnbioss201703.html"><meta property="article:section" content="events">
<meta property="article:published_time" content="2017-03-13T00:00:00+00:00">
<meta property="article:modified_time" content="2017-03-13T00:00:00+00:00"><meta property="og:site_name" content="GT BIOSS">
<link type=text/css rel=stylesheet href=/beta/vendor/css/bootstrap.min.css>
<link rel=stylesheet href=https://www.bioss-cnrs.fr/beta/scss/journal.min.e98aba1605f3a725fad0875a5584b620ed0fd58ea79d459f02fb560d7738fab4.css integrity="sha256-6Yq6FgXzpyX60IdaVYS2IO0P1Y6nnUWfAvtWDXc4+rQ=" media=screen>
<link rel=stylesheet href=https://www.bioss-cnrs.fr/beta/scss/dark-mode.min.92e13ee85b2c4587b9a2c423bde35865b35fa7449a8e1babb34f5c66907a72e1.css integrity="sha256-kuE+6FssRYe5osQjveNYZbNfp0Sajhurs09cZpB6cuE=" media=screen>
<script src=/beta/vendor/js/loadCSS.js></script>
<script>loadCSS("https://fonts.googleapis.com/css?family=Fira+Mono|Material+Icons")</script>
<script src=/beta/js/toc.js></script>
<script>console.log("Hello from 'layouts/partials/extended_head.html'")</script>
</head>
<body>
<div id=app><div id=sideContainer class=side-container>
<a class="a-block nav-head false" href=https://www.bioss-cnrs.fr/beta/>
<div class=nav-title>
GT Bioss
</div>
<div class=nav-subtitle>
Groupe de travail sur la biologie systémique symbolique
</div>
</a>
<div class=nav-link-list>
<a class="a-block nav-link-item false" href=/beta/about>
À propos
</a>
<a class="a-block nav-link-item false" href=/beta/seminaires/>
Séminaires mensuels
</a>
<a class="a-block nav-link-item false" href=/beta/categories/manifestations/>
Manifestations
</a>
<a class="a-block nav-link-item false" href=/beta/projets.html>
Projets
</a>
<a class="a-block nav-link-item false" href=/beta/membres.html>
Membres
</a>
<a class="a-block nav-link-item false" href=/beta/contact.html>
Contact
</a>
</div>
<div class=nav-footer>
<p><a href=https://www.bioss-cnrs.fr/beta/index.xml>Flux RSS</a></p>
Basé sur le
Theme Hugo <a href=https://github.com/amazingrise/hugo-theme-diary>Diary</a> par <a href=https://amazingrise.net>Rise</a>
<br>
Adapté de <a href=https://mak1t0.cc/ target=_blank rel="noreferrer noopener">Makito</a>'s <a href=https://github.com/SumiMakito/hexo-theme-journal/ target=_blank rel="noreferrer noopener">Journal.</a> <br>
<br>
&copy;
GT Bioss.
</div>
</div><div id=extraContainer class=extra-container>
<div class=pagination>
<a id=globalBackToTop class="pagination-action animated-visibility" href=#top :class="{ invisible: scrollY == 0 }">
<i class="material-icons pagination-action-icon">
keyboard_arrow_up
</i>
</a>
<a type=button class=pagination-action id=darkModeToggleButton>
<span class="material-icons pagination-action-icon" id=darkModeToggleIcon>
dark_mode
</span>
</a>
</div>
</div>
<div class=single-column-drawer-container id=drawer v-bind:class="{ 'single-column-drawer-container-active': isDrawerOpen }">
<div class=drawer-content>
<div class=drawer-menu>
<a class="a-block drawer-menu-item false" href=/beta/about>
À propos
</a>
<a class="a-block drawer-menu-item false" href=/beta/seminaires/>
Séminaires mensuels
</a>
<a class="a-block drawer-menu-item false" href=/beta/categories/manifestations/>
Manifestations
</a>
<a class="a-block drawer-menu-item false" href=/beta/projets.html>
Projets
</a>
<a class="a-block drawer-menu-item false" href=/beta/membres.html>
Membres
</a>
<a class="a-block drawer-menu-item false" href=/beta/contact.html>
Contact
</a>
</div>
</div>
</div>
<transition name=fade>
<div id=drawer-mask v-bind:class="{ 'single-column-drawer-mask': mounted }" v-if=isDrawerOpen v-on:click=toggleDrawer></div>
</transition>
<nav id=navBar class="navbar sticky-top navbar-light single-column-nav-container">
<div id=navBackground class=nav-background></div>
<div class="container container-narrow nav-content">
<button id=nav_dropdown_btn class=nav-dropdown-toggle type=button v-on:click=toggleDrawer>
<i class=material-icons>
menu
</i>
</button>
<a id=navTitle class=navbar-brand href=https://www.bioss-cnrs.fr/beta/>
GT Bioss
</a>
<button type=button class=nav-darkmode-toggle id=darkModeToggleButton2>
<i class=material-icons id=darkModeToggleIcon2>
dark_mode
</i>
</button>
</div>
</nav>
<div class=single-column-header-container id=pageHead v-bind:style="{ transform: 'translateZ(0px) translateY('+.3*scrollY+'px)', opacity: 1-navOpacity }">
<a href=https://www.bioss-cnrs.fr/beta/>
<div class=single-column-header-title>GT Bioss</div>
<div class=single-column-header-subtitle>Groupe de travail sur la biologie systémique symbolique</div>
</a>
</div>
<div id=content>
<div id=streamContainer class=stream-container>
<div class="post-list-container post-list-container-shadow">
<div class=post>
<div class=post-head-wrapper-text-only>
<div class=post-title>
Journées annuelles du GT Bioss (2017, 3e édition)
<div class=post-subtitle>
Les <strong>13 et 14 mars 2017</strong> à Montpellier
</div>
<div class=post-meta>
</div>
</div>
</div>
<div class=post-body-wrapper>
<div class=post-body v-pre>
<h3 id=informations-générales>Informations générales</h3>
<p>Date : Lundi 13 mars (toute la journée) et mardi 14 mars (matin) 2017</p>
<p>Lieu : Amphitheatre de l'espace colloque du CNRS, 1919 route de Mende à
Montpellier.  </p>
<p>Organisateurs : Ovidiu Radulescu, Grégory Batt, Cédric Lhoussaine,
Elisabeth Remy et Anne Siegel </p>
<p>La troisième édition des journées annuelles du GT Bioss va se dérouler
juste avant les journées nationales du GDR Informatique-Mathématique.
Ainsi, les 13 et 14 mars 2017, les membres du GT auront le plaisir de se
rencontrer autour de conférences autour des thèmes suivants :</p>
<p>- la modélisation des systèmes biologiques et ses applications;</p>
<p>- les systèmes dynamiques discrets, hybrides;</p>
<p>- les langages de modélisation et leurs sémantiques (déterministes,
non-déterministes, stochastiques);</p>
<p>- la vérification de modèles;</p>
<p>- la réduction de modèles et leur pouvoir prédictif (sous incertitude);</p>
<p>- l'inférence d&rsquo;interactions et de règles à partir de données
biologiques;</p>
<p>- et plus généralement tout problème de modélisation lié à
l&rsquo;intégration de données réelles. </p>
<p>\</p>
<h3 id=inscription>Inscription</h3>
<p>L'inscription, gratuite mais obligatoire, se fait en remplissant le
formulaire accessible
<a href="https://docs.google.com/forms/d/e/1FAIpQLSe9ppo0cu9LLTPrXTaw0dpHYzUNyACkeXst15Ag5ZX9jhoQ5A/viewform?c=0&w=1">ici.</a></p>
<h3 id=orateurs-invités>Orateurs invités</h3>
<p>[]{#docs-internal-guid-34c540d9-cafa-f741-0321-10cf7fc90e08
style=&ldquo;font-size:14.666666666666666px;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;"}
Anaïs Baudot, Institut de Mathématiques de Marseille.<br>
Jakob Ruess, INRIA Saclay.<br>
Thomas Sturm, CNRS-LORIA, Nancy & Max Planck Institute, Saarbrücken,
Germany.\</p>
<h3 id=programme>Programme</h3>
<h4 id=lundi-13-mars>Lundi 13 Mars</h4>
<p><strong>09h00 - 09h25</strong> - <strong>Accueil - Café</strong><br>
<em>Chairwoman: Anne Siegel</em><br>
<strong>09h25 - 09h30</strong> - Ovidiu Radulescu - <em>Introduction</em>.<br>
<strong>09h30 - 10h15</strong> - <strong>Conférencier invité</strong> - Jakob Ruess - <em>Control of
bio-digital systems in single cells.</em><br>
<strong>10h20 - 10h35</strong> - Ovidiu Radulescu - <em>Time dependent multivariate
distributions for piecewise-deterministic models of gene networks.</em>
<a href=slides/Radulescu.pdf>slides</a><br>
<strong>10h40 - 10h55</strong> - Stefano Casagranda - <em>Principal Process Analysis and
reduction of biological models with order of magnitude.</em><br>
<a href=slides/Casagranda.pdf>slides</a> <strong>11h00 - 11h30</strong> - <strong>Pause</strong><br>
<em>Chairman: Cédric Lhoussaine</em><br>
<strong>11h30 - 11h45</strong> - Yves-Stan Le Cornec - <em>Le Projet Kami.</em><br>
<strong>11h50 - 12h05</strong> - Andreea Beica - <em>Synchronous balanced analysis.</em>
<a href=slides/Beica.pdf>slides</a>i<br>
<strong>12h10 - 12h25</strong> - François Fages - <em>Complexité algorithmique des
calculs analogiques et compilation de fonctions mathématiques en
réactions biochimiques élémentaires.</em><br>
<br>
<strong>12h30- 13h45 - Pause déjeuner</strong><br>
<br>
<em>Chairman: Ovidiu Radulescu</em><br>
<strong>13h45 - 14h30</strong> - <strong>Conférencier invité</strong> - Thomas Sturm - <em>Symbolic
Methods in Bifurcation Analysis.</em> <a href=slides/Sturm.pdf>slides</a><br>
<strong>14h35 - 14h50</strong> - Matthieu Pichene - <em>Predicting tumor growth using a
statistical layered population abstraction.</em>
<a href=slides/Pichene.pdf>slides</a><br>
<strong>14h55 - 15h10</strong> - François Boulier - <em>Identifiabilité, équations
intégro-différentielles et neurobiologie.</em> <a href=slides/Boulier.pdf>slides</a><br>
<strong>15h15 - 15h40</strong> - <strong>Pause</strong><br>
<em>Chairman: François Fages</em><br>
<strong>15h40 - 15h55</strong> - Clémence Frioux - <em>Hybrid gap-filling to reconcile
qualitative and quantitative abstractions of metabolism.</em><br>
<strong>16h00 - 16h15</strong> - Emilie Allart - <em>Elementary modes refine abstract
interpretation of reaction networks with partial kinetic information.</em><br>
<strong>16h20 - 16h35</strong>; Marie Beurton-Aimar - <em>How to display patterns inside
elementary flux modes.</em><br>
<strong>16h40 - 17h00</strong> - <strong>Pause</strong><br>
<em>Chairman: Grégory Batt</em><br>
<strong>17h00 - 17h15</strong> - Florian Bridoux - <em>On The Cost Of Simulating A
Parallel Boolean Automata Networks By A Sequential One.</em><br>
<strong>17h20 - 17h35</strong> - Aurélien Naldi - <em>Reversed logical models for the
study of basins of attraction.</em><br>
<strong>17h40 - 17h55</strong> - Pierre Siegel - <em>Des logiques non-monotones aux
systèmes dynamiques discrets (SDD).</em><br>
<strong>18h00 - 18h15</strong> - Laurent Trilling - <em>Apport de la non monotonie pour
la modélisation logique de réseuax de régulation génique.</em><br>
<br>
<strong>19h15</strong> - Repas au Restaurant <a href=http://www.trinquefougasse.com/o-nord>Trinque Fougasse
O'Nord</a> 1581 route de Mende (à
350m à pied du lieu de la conférence), formule 30€, vin et concert jazz
compris. Il est encore possible de s&rsquo;inscrire au repas!</p>
<h4 id=mardi-14-mars>Mardi 14 Mars</h4>
<p><em>Chairwoman: Élisabeth Rémy</em><br>
<strong>09h00 - 09h45</strong>: <strong>Conférencière invitée</strong> - Anais Baudot - <em>Mining
and modeling biological networks to study rare and common human
diseases.</em><br>
<strong>09h50 - 10h10</strong>: <strong>Pause</strong><br>
<strong>10h10 - 10h25</strong>: Jean Coquet - <em>Analysis of TGF-β signaling networks
to find different families of trajectories.</em><br>
<strong>10h30 - 10h45</strong>: Arnaud Poret - <em>Linking Cancer Models with
Therapeutic Effects.</em><br>
<strong>10h50 - 11h05</strong>: Amos Korman - <em>Confidence sharing: an economic
strategy for efficient information flows in animal groups.</em><br>
<strong>11h10 - 11h30</strong>: <strong>Pause</strong><br>
<em>Chairman: Laurent Trilling</em><br>
<strong>11h30 - 11h45</strong>: Ofer Feinerman - <em>Algorithmic challenges in ant
cooperative transport.</em><br>
<strong>11h50 - 12h05</strong>: Jonathan Behaegel - <em>Réseaux génétiques hybrides: de
la logique de Hoare à l'identification de paramètres.</em><br>
<strong>12h10 - 12h25</strong>: Celia Biane - <em>Inférence d'action sur les réseaux
pour la reprogrammation cellulaire.</em>\</p>
<h3 id=résumés>Résumés</h3>
<p><strong>Jakob Ruess (INRIA Saclay)</strong> - <em>Control of bio-digital systems in
single cells.</em><br>
Akin to the developments in industry, also in scientific wet labs more
and more manual tasks are being automated and many experiments are
nowadays carried out by computer controlled robots. Such experiments
are, however, still almost exclusively pre-planned by the scientist as a
series of exact instructions that are to be carried out by the robot.
Industry, on the other hand, has seen developments such as self-driving
cars where the user specifies a goal and the system decides for itself
how to best attain that goal. In this talk, I will present our recent
efforts to construct autonomous bio-digital systems that are based on
real-time computer-to-single cell communication. In particular, I will
present an experimental/technological platform in which cells, equipped
with an optogenetically inducible promoter, signal to a computer through
microscopy. The computer processes the incoming data in real time and
sends back light signals to the cells. Experiments on this platform can
therefore be specified as algorithms that map the list of received
signals from all the cells into a list of optogenetic stimulations. To
demonstrate the functionality of this setup, I will show results
obtained with two different algorithms implemented on the platform.
First, we used a model-predictive controller to drive the expression of
a gene coding for a fluorescent reporter protein and managed to ensure
that all cells follow the same gene expression target profile despite
significant heterogeneity of the cells and even in the presence of
environmental perturbations. Subsequently, we used the same controller
to make all cells follow different target profiles, demonstrating that
the platform can be used to structure populations into pre-definable
dynamic gene expression phenotypes. Finally, I will provide results
where we coupled an algorithm mimicking a negative feedback loop and a
digital cell-to-cell communication mechanism to the gene in order to
produce synchronized fluorescence oscillations in the cells.<br>
<br>
<strong>Ovidiu Radulescu (University of Montpellier, DIMNP lab)</strong> - <em>Time
dependent multivariate distributions for piecewise-deterministic models
of gene networks.</em><br>
We discuss piecewise-deterministic approximations of gene networks
dynamics. These approximations capture in a simple way the stochasticity
of gene expression and the propagation of expression noise in networks
and circuits. By using partial omega expansions, piecewise deterministic
approximations can be formally derived from the more commonly used
Markov pure jump processes. We are interested in time dependent
multivariate distributions that describe the stochastic dynamics of the
gene networks. This problem is difficult even in the simplified
framework of piecewise deterministic processes. We consider three
methods to compute these distributions: the direct Monte Carlo
simulation, the numerical integration of the Liouville-master equation
and the push-forward method. We present applications of this approach to
stochastic biological switches and logical gates that can be used as
memories and computing elements in synthetic biocomputing machines.<br>
<br>
<strong>Stefano Casagranda (INRIA Sophia-Antipolis, BICOLORE group)</strong> -
<em>Principal Process Analysis and reduction of biological models with
order of magnitude.</em><br>
We present a simple method that allows to analyze the biological
processes of a dynamical model and classify them. Along the system
trajectories, we decompose the model into biological meaningful
processes and then study their activity or inactivity during the time
evolution of the system. The structure of the model is then reduced to
the core mechanisms involving only the active processes. The initial
conditions are supposed to lie in some rectangle, that could represent
one order of magnitude for the variables. Keeping only the active
processes, we obtain the principal processes in the rectangle and then
in the adjacent rectangles where the trajectories may have a transition.
Finally we obtain a partition of the space with a reduced model within
each rectangle. We apply these techniques to a classical model of gene
expression with protein and messenger RNA.<br>
<br>
<strong>Yves-Stan Le Cornec (ENS Lyon)</strong> <em>Le Projet Kami.</em><br>
Rule-based modelling enables one to describe complex biological systems
by defining and combining together low level mechanisms. However, the
details of these mechanisms and the exact conditions for each
interaction are not always well known and often spread apart across
multiple articles which impedes the process of building a model. KAMI is
a software application as well as a graph-based formalism which aims to
aggregate biological knowledge in an incremental and flexible way as
well as ensuring its consistency. In KAMI, basic pieces of knowledge are
represented as graphs called nuggets which are ideally all "typed" by
another graph (the action graph). This has the effect of organizing and
combining the knowledge contained inside the nuggets as well as
providing a graphical overview of it. This action graph is itself typed
by a meta-model which provides syntactic restrictions and structure to
the model. This in particular facilitates the use of well defined graph
transformations when needed, for instance to merge together multiple
knowledge bases which can use different formats, to communicate with
other existing tools, or to make the meta-model itself evolve if
needed.<br>
In practice, we developed a generic graph rewriting library in python
(Regraph) and a web interface (RegraphGui) that are used to build and
manipulate the biological knowledge base. Rule-based models written in
kappa can then be generate and sent to the KaSim simulator.<br>
<br>
<strong>Andrea Beica (ENS, Paris)</strong> - <em>Synchronous balanced analysis.</em><br>
When modeling Chemical Reaction Networks, a commonly used mathematical
formalism is that of Petri Nets, with the usual inter-leaving execution
semantics. We aim to substitute to a Chemical Reaction Network,
especially a growth one (i.e., for which an exponential stationary phase
exists), a piecewise synchronous approximation of the dynamics: a
resource-allocation-centered Petri Net with maximal-step execution
semantics. In the case of unimolecular chemical reactions, we prove the
correctness of our method and show that it can be used either as an
approximation of the dynamics, or as a method of constraining the
reaction rate constants (an alternative to flux balance analysis, using
an emergent formally de ned notion of growth rate as the objective
function), or a technique of refuting models.<br>
<br>
<strong>François Fages (Inria Saclay Ile-de-France, Lifeware group)</strong> -
<em>Complexité algorithmique des calculs analogiques et compilation de
fonctions mathématiques en réactions biochimiques élémentaires.</em><br>
L&rsquo;aspect continu de beaucoup d&rsquo;interactions protéiques nous conduit à
considérer des modes de calcul mixtes analogiques-digitaux, pour
lesquels des résultats récents de Bournez, Graca et Pouly en théorie de
la calculabilité et de la complexité analogiques établissent des liens
fondamentaux avec la programmation classique. Nous dérivons de ces
résultats un compilateur de spécifications comportementales en systèmes
de réactions biochimiques que l&rsquo;on peut comparer aux circuits naturels
résultats de l&rsquo;évolution. Nous illustrons cette démarche par l&rsquo;exemple
du module de signalisation MAPK qui a une fonction de convertisseur
analogique-digital dans la cellule, et par le contrôle du cycle
cellulaire.<br>
<br>
<strong>Thomas Sturm (CNRS-LORIA, Nancy & Max Planck Institute, Saarbrücken,
Germany)</strong> - <em>Symbolic Methods in Bifurcation Analysis.</em><br>
We are going to discuss the potential of symbolic computation methods
with the analysis of biological networks for bifurcations. Those methods
include various real quantifier elimination methods in combination with
preprocessing heuristics, real triangularizations, and subtropical
approaches. Applications range from the identification of Hopf
bifurcations in comparatively small models to work-in-progress towards
the identification of saddelnode bifurcations with respect to more than
one parameter in a model of MAPK from the BioModels database. After
several years of systematic work and collaborations in the area we feel
confident that there is a potential for symbolic tools in biological
research, in particular with parametric situations, although there are
obvious obstacles: Firstly, our approaches are not fully automatic yet,
so that users have to understand their theoretical background to some
extent. Secondly, for complexity reasons the methods cannot be expected
to scale to all relevant problem sizes. It is thus important to identify
application areas that are sufficiently large to justify the effort for
both developers and potential users. We hope to stimulate critical but
open-minded constructive discussions around those topics.<br>
<br>
<strong>Matthieu Pichene (Inria Rennes, SUMO group)</strong> - <em>Predicting tumor
growth using a statistical layered population abstraction.</em><br>
Tumor growth models such as B. Waclaw et al. (2015) can be a useful tool
to design protocols for cancer treatment. A challenging problem is that
models (usually cell based) can rapidly be time and memory consuming due
to tumors reaching hundreds of thousands of cells. These cells can each
duplicate, move and die. We designed an abstraction that represent the
tumor evolution. Tumor is partitioned into layers. In each layer, few
data are kept, e.g. the density of the layer, but not the individual
cell data. From statistics drawn from agent based models such as
TumourSimulator B. Waclaw et al. (2015) for small tumors, we
automatically obtain a statistical model that can reproduce the tumor
evolution, and predict the evolution of bigger tumor under treatment.
Compared to the original model, we can reproduce very faithfully the
evolution of the tumor. Also, we have clear speed up (e.g. 30 times for
a tumor of 100.000 cells).<br>
<br>
<strong>François Boulier (University of Lille, CRIStAL lab)</strong> -
<em>Identifiabilité, équations intégro-différentielles et neurobiologie.</em><br>
L'identifiabilité est une étude théorique d'un modèle mathématique
destinée à établir si les paramètres du modèle ont une valeur
complètement déterminée par les mesures expérimentales. C'est une étude
de principe. Dans le cas d'un modèle présenté sous la forme d'un
système d'équations différentielles paramétriques non linéaires, cette
propriété peut être établie, après une étape de calcul formel, par
l'analyse du « polynôme entrée-sortie » du modèle. Le résultat de
l'étude peut ensuite être réemployé pour effectuer une estimation de
paramètres à partir de données expérimentales réelles. Des travaux
récents en calcul formel et en théorie du contrôle ouvrent de nouvelles
perspectives théoriques et pratiques visant à généraliser cette approche
en faisant apparaître des équations intégro-différentielles. Dans cet
exposé, nous exposerons ces idées et les illustrerons sur une
problématique issue des neurosciences, où la question posée consiste à
clarifier le rôle des astrocytes pathologiques dans le développement de
la dépression corticale propagée.<br>
Il s'agit de travaux en cours, menés en collaboration entre une équipe
de calcul formel (Univ. Lille/CRIStAL), une équipe de modélisation en
mathématiques appliquées (Univ. Le Havre/LMAH) et une équipe de
neurobiologie (Rouen/INSERM).<br>
<br>
<strong>Clémence Frioux (Inria Rennes, Dyliss group)</strong> - <em>Hybrid gap-filling
to reconcile qualitative and quantitative abstractions of metabolism.</em><br>
As the amount of genomics data keeps on increasing, so is the need to
develop efficient techniques to model the metabolism. An important part
of the modelling is the gap-filling step during the reconstruction of
metabolic networks. It aims to complete a draft metabolic network based
on experimental evidence by selecting adequate reactions from a database
to restore the observed metabolic behaviour. Most of the existing
techniques to achieve this procedure are quantitative and use linear
programming to model flux distribution in the networks. They need
accurate data and some linear problems are not solvable without a
relaxation of constraints due to the space of solutions. We thus
developed lately a topological method called meneco that can perform
gap-filling when few data is available. Based on Answer Set Programming
(ASP), it takes advantage of efficient solvers to easily sample the
space of solutions, but the outputs are often not flux-balanced. Thanks
to the progress in the development of ASP, it is now possible to plug a
linear programming solver to the ASP one. Here we propose a hybrid
method to take the most out of the two modellings and use the efficiency
of the topological solving to reach quantitative-compliant solutions for
gap-filling. This hybrid solving enables to take into account new
constraints and hypotheses about the dynamic system (initial state,
steady state, inputs/outputs) and can be relevant to solve new metabolic
networks modelling problems.<br>
<br>
<strong>Emilie Allart (University of Lille, CRIStAL lab)</strong> - <em>Elementary modes
refine abstract interpretation of reaction networks with partial kinetic
information.</em><br>
Genetic engineering is nowadays widely used to change the genome of
cells in order to modify their behaviour, for example to overproduce
some metabolite of interest. Gene knockout is one common genetic
engineering technique which consists in removing one or more genes from
the genome. Given the combinatorial number of possible genetic changes
and the consequent impossibility of testing all of them by wet lab
experiments, in silico prediction of the most interesting genetic
modifications is often desirable. However, the necessary information for
the in silico modelling of the organism of interest is usually lacking.
This is the case for example for the bacterium B. Subtilis: its genetic
engineering for the overproduction of surfactin (a well-known antibiotic
[1]) is of high interest in agriculture among other fields, but the
lack of information about the biochemical functioning of this organism
prevented us from modeling it with the existing methods. In order to
overcome this problem, we developed a modeling language to represent
reaction networks with partial kinetic information, and a method based
on abstract interpretation that allows us to analyse models written in
this language even in the absence of quantitative information [2, 3]:
from the steady state semantics of the reaction network, we transform
arithmetic constraints into abstract constraints over a finite domain
where unknowns have been abstracted away. The qualitative behaviour of
the system can therefore be evaluated in silico by means of a constraint
solver, which gives us the set of all solutions (corresponding to
combinations of feeding changes and gene knockouts) that lead to the
desired behaviour. However, abstract interpretation usually
over-approximates the solution space. As an example, while a finite
system of linear equations implies all the equations that can be
expressed as a linear combination, this implication no longer holds once
the equations have been translated into abstract constraints. My current
work consists in optimally generating new arithmetic constraints from
the existing in order to minimize the solution space. Here I will
discuss in particular our interest in elementary flux modes [4] and
how to adapt classical elementary mode analysis to the generation of
abstract constraints to improve the qualitative analysis of reaction
networks with partial kinetic information.<br>
<br>
<strong>Marie Beurton-Aimar (University of Bordeaux, Labri lab)</strong> - <em>How to
display patterns inside elementary flux modes.</em><br>
Elementary flux modes computing is a powerful tool to identify feasible
routes trough a metabolic network(Schuster et al. Nat. Biotechnol., 18
(3), 2000). According to their definitions, the list of all elementary
flux modes provides a way to analyze the behaviors of the network both
in its current functioning and under perturbations. But the main problem
with this tool remains the size of this list. Currently a metabolic
network with approximatively 50 reactions and metabolites can generate
more than several thousands of elementary flux modes. To be useful, new
tools to automatically analyze the results are required. In a first
attempt, we have used the computing of Minimal Cut Sets (Gagneur J.,
Klamt S. BMC Bioinformatics, 2004) of all elementary flux modes to
identify patterns, i.e. list of common reactions, in the set of
elementary flux modes . To visualize these patterns which can be see as
a tree of sub-patterns we have chosen a technique coming from the domain
of large data set visualization, the parallel coordinates displaying.
This technique presents data as flux through variables put on the x
axis. Values of the flux is on the y axis. In our case, reactions are
put on the x axis and elementary flux modes are data lines or edges
between reactions. The reaction stoichiometry values are set on the y
axis: 1 if the reaction is present in forward direction in this
elementary flux mode, 2 if it is in backward reaction and 0 if the
reaction is absent. The flux size between two reactions informs about
the quantity of elementary flux modes which use the same association
between two reactions. The figure below shows a result example of
patterns sharing by the elementary flux modes in the plant cell
metabolism (Beurton-Aimar et al. Plant Metabolic Flux Analysis, 2013).
The displaying tool CoPHI is accessible on this website
<a href=http://www.labri.fr/perso/jsansen/>http://www.labri.fr/perso/jsansen/</a>. The display is dynamique, users can
change the order of the reactions on x axis and so explore by themselves
the patterns that can be found. Information given by MCS computing is
used to sort reactions by taking into account set of reactions. Future
works will be done to develop a CoPHI version dedicated to metabolic
network analysis.<br>
<br>
<strong>Florian Bridoux (University of Marseille, LIF lab)</strong> - <em>On The Cost Of
Simulating A Parallel Boolean Automata Networks By A Sequential One.</em><br>
In this presentation, we study Boolean automata networks (BANs). A given
BAN can be associated with several dynamics, depending on the schedule
(i.e. the order) we choose to update its automata. In this presentation,
we consider all block-sequential update schedules: we group automata
into blocks, and we update all automata of a block at once, and iterate
the blocks sequentially. For the last 15 years, people have studied the
influence of the update schedules on the dynamics of a BAN. Here, we do
the opposite. We want to determine the minimum number κ of additional
automata that a BAN associated with a given block-sequential update
schedule needs to simulate a given BAN with a parallel update schedule.
To solve this problem, we introduce a graph that we call confusion graph
built from the BAN and the update schedule. We show the relation between
κ and the chromatic number of the confusion graph. Thanks to this
confusion graph, we bound κ in the worst case between n/2 and 2n/3 + 2
(n being the size of the BAN simulated) and we conjecture that this
number equals n/2. We support this conjecture with two results: the
clique number of a confusion graph is always less than or equal to n/2
and, for the subclass of bijective BANs, κ is always less than or equal
to n/2.<br>
<br>
<strong>Aurélien Naldi (University of Montpellier, DIMNP lab)</strong> - <em>Reversed
logical models for the study of basins of attraction.</em><br>
Boolean, and more generally logical, models are widely used for the
study of biological systems, especially differentiation systems. The
formal identification of their attractors has been widely studied,
however the corresponding basins of attraction received less attention.<br>
In this purpose, we propose a method to construct a "reversed" model
regarding the asynchronous dynamical behaviour: the successors of the
states of a reversed model correspond to their predecessors in the
original one. While the reversal can be generalized only to a particular
class of multivalued models, we show how to use model booleanization to
study the reversed asynchronous dynamics of any model.<br>
The study of the reversed dynamics then facilitates the identification
of the basins of attraction, assuming that we already know the
attractors. We can further compute "strong" and "weak" basins (from
which we can respectively reach a unique or multiple attractors), as
well as their "frontiers", i.e., the sets of states such that the
reachable attractors are different from the attractors reachable from
neighbouring states (predecessors or successors).<br>
This approached is illustrated on a published model of the cell fate
decision in response to death-receptor engagement.<br>
<br>
<strong>Pierre Siegel (University of Marseille, LIF lab)</strong> - <em>Des logiques
non-monotones aux systèmes dynamiques discrets (SDD).</em><br>
Du point de vue logique et représentation des connaissances, un système
biologique peut être considéré comme un ensemble de variables qui
interagissent. Dans ce cadre la cellule pose des problèmes intéressants
à l&rsquo;Intelligence Artificielle. Il faut d&rsquo;abord formaliser les
interactions mais une formalisation en logique « classique » est
difficile et donne des incohérences. Ensuite ce que l&rsquo;on sait vient en
bonne partie d&rsquo;expériences. On ne connaît donc qu&rsquo;une petite partie des
interactions et cette connaissance peut être révisable, incertaine,
contradictoire et même fausse. On veut aussi essayer de compléter
in-silico les interactions. Enfin la complexité algorithme est
importante. Ces problèmes sont étudiés depuis longtemps en IA en
utilisant des logiques non-monotones et des techniques de programmation
par contraintes. Dans le cas particulier de la cellule des résultats ont
été obtenus en utilisant la logique des défauts [1].<br>
D&rsquo;un autre côté, les systèmes biologiques peuvent être regardés dans le
contexte des réseaux d&rsquo;automates et des SDD. Des théorèmes importants
portent sur les cycles d&rsquo;interactions dont l&rsquo;étude est fondamentale.
Mais une représentation des SDD par la logique des défauts (et par la
plupart des logiques non-monotones) n&rsquo;est pas adaptée ; par exemple
l&rsquo;équivalent d&rsquo;un circuit négatif n&rsquo;a pas d&rsquo;extension (de solution, de
point fixes&mldr;) Cette absence d&rsquo;extension pour les logiques des défauts
a été étudiée [2,3] et cette étude a donné la Logique des Hypothèses.
Pour cette logique on a toujours des extensions mais certaines d&rsquo;entre
elles, les extensions fantômes, bien caractérisées sont spéciales (et
leur utilité n&rsquo;était pas claire).<br>
Nous étudions une représentation des SDD par la Logique des Hypothèses.
Un but est de permettre de discriminer les états stables, les cycles
stables et les cycles instables. Les extensions fantômes semblent
permettre de le faire. Un autre but est de donner des algorithmes
efficaces pour calculer les états et cycles. Quelques premiers résultats
seront présentés.<br>
<br>
<strong>Laurent Trilling (University of Grenoble, TIMC-IMAG lab)</strong> - <em>Apport
de la non monotonie pour la modélisation logique de réseuax de
régulation génique.</em> Nous nous intéresserons à la modélisation
déclarative des réseaux logiques de régulation génique proposés par René
Thomas, à l'aide de la technologie de programmation logique ASP (Answer
Set Programming). Cette technologie est basée sur une logique non
monotone n'autorisant que certains types de modèles logiques, dits
stables: intuitivement, ne sont vrais que les atomes qui sont prouvables
grâce aux axiomes. Un modèle stable est minimal en ce sens que la
suppression d'un atome du modèle ne peut résulter en un modèle. La
technologie ASP connait un certain engouement à l'heure actuelle,
assurément dû au pouvoir d'expression des langages proposés et aussi
aux performances des solveurs associés. Mais l'intérêt de cette
propriété de non monotonie et son impact sur la modélisation
d'applications en termes de méthodologie et d'optimisation ne sont pas
toujours soulignés précisément. Nous tâcherons de le faire dans le cadre
de la modélisation des réseaux de Thomas.<br>
Par approche déclarative, nous entendons : représentation de toutes les
données biologiques disponibles (sur la structure ou la dynamique), sous
forme d'axiomes logiques(contraintes) et obtention sous forme
intensionelle (implicite) des réseaux de Tomas cohérents avec ces
données en cas de satisfaisabilité. Dans le cas contraire, une procédure
automatique de réparation justifiée doit être applicable. L'approche
fournit, entre autres, une faculté d'apprentissage appréciable : celle
de tous les théorèmes, restreints à une formulation fixée (par exemple
les clauses d'une taille limitée sur un ensemble déterminé d'atomes),
déductibles de ces modèles.<br>
Deux avantages généraux d'ASP sont reconnus : un traitement des données
incomplètes radical (les atomes non prouvables sont réputés faux) et un
pouvoir de déduction augmenté (les modèles stables forment un
sous-ensemble des modèles). Nous aborderons plutôt trois aspects précis,
concernant la modélisation étudiée : 1) l'utilisation de défauts pour
spécifier la réparation d'inconsistance en soulignant l'intérêt
d'exprimer la minimisation sous-jacente en termes logiques (et non pas
algorithmiques), 2) l'emploi (et la méthodologie de construction)
d'une conjonction de défauts originale pour exprimer la notion de
compositions d'interactions géniques seulement généralement vraies
(sauf à être prouvée effectivement fausses), 3) la mise en œuvre de
formules CTL générales, en particulier du type AF (indispensables pour
exprimer des propriétés exhaustives sur les comportements), en prenant
appui sur la minimalité des modèles stables.<br>
<br>
<strong>Anaïs Baudot (Institut de Mathématiques de Marseille)</strong> - <em>Mining and
modeling biological networks to study rare and common human diseases.</em><br>
Networks are scaling-up the analysis of gene and protein functions,
hence offering new avenues to study the diseases in which these genes
and proteins are involved. In this context, I will talk about the
exploration of interactome networks containing thousands of physical and
functional interactions between proteins. We develop partitioning
algorithms to recover communities &ndash; or functional modules &ndash; from these
large-scale networks, and use them to study the cellular functions of
proteins of interest. Recently, we have extended the community detection
to multiplex networks, i.e., networks containing different layers of
different interaction categories, such as protein-protein interaction or
gene co-expression. I will show how these approaches can be used to
study the relationships between different common diseases, in particular
their inverse comorbidities. The last part of my talk will be dedicated
to our recent work on random walks with restart on multiplexes, in order
to study rare monogenic diseases. I will finally briefly mention logical
modeling to decipher drug synergies in cancer, and how mining
large-scale static networks and modeling smaller-scale dynamical network
could unite around expression data integration.<br>
<br>
<strong>Jean Coquet (Inria Rennes, Dyliss group)</strong> - <em>Analysis of TGF-β
signaling networks to find different families of trajectories.</em><br>
The Transforming Growth Factor TGF-β is a multifunctional cytokine that
regulates mammalian development, differentiation, and homeostasis
(Ikushima and Miyazono, 2011). As a growth inhibitor of epithelial,
endothelial, and hematopoietic cells, TGF-β is a potent anticancer agent
in normal tissue. At the opposite TGF-β acts as a promoter of tumor by
inducing the hallmarks of the cancer. The context-dependent pleiotropic
nature of TGF-β is associated with complex signaling pathways. Our group
recently developed a model for TGF-β- dependent signalling based on the
integration of the 137 signaling pathways from Pathway Interaction
Database (Andrieux et al, 2014). Based on guarded transition formalism,
we identified 15,934 chains of reactions (trajectories) linking TGF-β to
at least one of 159 target genes. The combined size and complexity of
this model place it beyond current understanding. Its analysis require
automated tools for identifying general patterns. In this study, we
focused on designing a reasoning method to characterize the composition
of these trajectories. We used an exhaustive and without prior
assumptions approach to explore them. First, we grouped the trajectories
into several families using the Relevant Set Correlation model (Houle,
2008). Then, we extracted the over-represented molecules for each
families. Finally, we identified pairs of over-represented molecules
included in the same trajectories but not described in pathway
databases. The main results of the study were the identification of two
trajectory families representing different TGF-β-dependent signaling
pathways, and the prediction of hundreds of molecule pairs occurring in
trajectories and never reported in signaling databases.<br>
<br>
<strong>Arnaud Poret (Ecole Centrale de Nantes, LS2N lab)</strong> - <em>Linking Cancer
Models with Therapeutic Effects.</em><br>
The goal is to model the signaling pathways involved in the drug
response of cancerous cells to find how to counteract resistance. We
have the gene expression profile (GEP) of several breast cancer and
multiple myeloma cells treated with various drugs. Along with the GEPs,
we also have the phenotypic drug response of the cells in term of
survival percentage. Building models of the signaling pathways involved
in drug response/ resistance could allow us to predict molecular
interventions able to counteract resistance, and then decrease the
survival of resistant cells. For reconstructing these signaling pathways
from the GEPs, we will use existing tools such as PID2SIF2Graph and
MCWalk, operating on public databases such as KEGG, Reactome,
Wikipathways and NCI-PID. For modeling the reconstructed networks, we
envision two approaches: logical programming, namely answer set
programming, and Boolean networks. The obtained models will be used to
assess several molecular interventions in their ability at decreasing
the predicted survival percentages.<br>
<br>
<strong>Amos Korman (University Paris Diderot, IRIF lab)</strong> - <em>Confidence
sharing: an economic strategy for efficient information flows in animal
groups.</em><br>
Social animals may share information to obtain a more complete and
accurate picture of their surroundings. However, physical constraints on
communication limit the flow of information between interacting
individuals in a way that can cause an accumulation of errors and
deteriorated collective behaviors. In this talk, I will theoretically
discuss a general model of information sharing within animal groups, and
take an algorithmic perspective to identify efficient communication
schemes that are, nevertheless, economic in terms of communication,
memory and individual internal computation. I will present a simple
algorithm in which each agent compresses all information it has gathered
into a single parameter that represents its confidence in its behavior.
Confidence is communicated between agents by means of active signaling.
It turns out that this algorithm competes extremely well with the best
possible algorithm that operates without any computational constraints.<br>
The proofs rely on the Cramer-Rao bound and on our definition of a
Fisher Channel Capacity. These concepts are used to quantify information
flows within the group and to obtain lower bounds on collective
performance. The results suggest confidence sharing as a central notion
in the context of animal communication.<br>
The talk is based on a paper published in PLoS Computational Biology.<br>
<br>
<strong>Oder Feinerman (Weizmann institute of science)</strong> - <em>Algorithmic
challenges in ant cooperative transport.</em><br>
Ants that encounter a large food item may join forces to cooperatively
carry it towards the nest. To do this, the ants must coordinate their
forces while simultaneously navigating towards the nest, often through
complex environments.This talk will discuss the behavioral algorithm
that allows that ants to achieve this non-trivial feat. I will show how
the ants' carrying algorithm allows the group to efficiently extract
useful information that is made available by the navigationally
competent individuals. Furthermore, I will show how the same simple
algorithmic laws lead to the emergence of alternative group-level
solutions when individual ant capabilities do not suffice.<br>
<br>
<strong>Jonathan Behaegel (University of Nice, I3S lab)</strong> - <em>Réseaux
génétiques hybrides: de la logique de Hoare à l'identification de
paramètres.</em><br>
La modélisation de la dynamique des réseaux de gènes repose sur des
paramètres qui reproduisent les comportements du système. Lorsque le
temps considéré évolue de manière continu (temps chronométrique),
l'identi cation des paramètres devient extrêmement di cile. Il devient
alors nécessaire de prendre en compte de nouvelles contraintes provenant
des données biologiques qui représentent, dans la plupart des cas, des
durées entre des événements observés. Pour prendre en compte de telles
durées, nous étendons le cadre de mod- élisation de René Thomas en
considérant que chaque état qualitatif devient un sous-domaine de
l'espace des concentrations dans lequel les trajectoires évoluent de
manière continue. Ainsi, chaque trajectoire met un temps non nul à
traverser chaque sous-domaine. Il s'agit d'une classe particulière
d'automates hybrides linéaires dont les paramètres sont appelés
"célérités".<br>
Les traces biologiques observées (succession d'états qualitatifs
combinés avec des durées) peuvent alors être interprétées comme une
exécution de programme impératif, et la logique de Hoare munie du calcul
de la plus faible précondition nous permet de construire des contraintes
sur les paramètres du modèle. Nous illustrons ce calcul de la plus
faible précondition sur un modèle simpli é du cycle circadien.<br>
<br>
<strong>Celia Biane (University of Evry, Ibisc lab)</strong> - <em>Inférence d'action
sur les réseaux pour la reprogrammation cellulaire.</em><br>
La reprogrammation consiste à modifier un système dynamique initial afin
d&rsquo;atteindre ou d&rsquo;éviter certains comportements. En biologie, la
reprogrammation cellulaire est étudiée notamment dans le cas du Cancer
et de l&rsquo;étude des cellules souches. Par exemple, au cours de la
tumorigenèse, des évènements génétiques et épi-génétiques sont
responsables d&rsquo;une transition d&rsquo;une cellule saine à une cellule
cancéreuse et les médicaments reprogramment la cellule cancéreuse vers
la mort cellulaire. Un défi majeur est l&rsquo;identification, sur
l'interactome (réseau biologique décrivant les interactions entre
molécules), des perturbations causales de la maladie, et inversement,
des cibles d&rsquo;action des médicaments. Dans ce cas, la reprogrammation se
caractérise par des actions structurelles de délétions ou de destruction
de sous-ensembles de nœuds ou d&rsquo;arcs responsables de transitions de
comportement cellulaire. L'inférence d&rsquo;actions causales est
combinatoire et constitue un problème inverse à une simulation de
perturbations structurelles responsables de modifications de la
dynamique du réseau moléculaire. Dans cette présentation, nous décrirons
la formalisation de ce problème par le gain et la perte de propriétés à
l&rsquo;équilibre dans un réseau booléen, nous présenterons des méthodes
algorithmiques d&rsquo;inférence d&rsquo;actions sur les réseaux et montrerons un
exemple d&rsquo;application de ces méthodes pour la prédiction de «drivers»
(évènements génétiques initiant un processus cancéreux) et de cibles
thérapeutiques dans le cas du Cancer du Sein. Reprogrammation
cellulaire, Problème Inverse, Inférence d&rsquo;actions structurelles, Réseaux
Booléens, Dynamique moléculaire, Cancer.<br>
\</p>
<p>[<br>
]{style=&ldquo;font-weight: bold;"}</p>
<h3 id=participants>Participants</h3>
<hr width=100% id=EOF>
<p style=color:#777>Dernière modification le 13/03/2017</p>
</div>
</div>
<nav class=post-pagination>
<a class=newer-posts href=https://www.bioss-cnrs.fr/beta/manif/jnbioss_201807/jnbioss201807.html>
Suivant<br>Journées annuelles du GT Bioss (2018, 4e édition)
</a>
<a class=older-posts href=https://www.bioss-cnrs.fr/beta/manif/lille_201611/lille201611.html>
Précédent<br>Workshop "Metabolism in Systems Biology"
</a>
</nav>
<div class=post-comment-wrapper>
</div>
</div>
</div>
</div>
</div><div id=single-column-footer><p><a href=https://www.bioss-cnrs.fr/beta/index.xml>Flux RSS</a></p>
Basé sur le
Theme Hugo <a href=https://github.com/amazingrise/hugo-theme-diary>Diary</a> par <a href=https://amazingrise.net>Rise</a>
<br>
Adapté de <a href=https://mak1t0.cc/ target=_blank rel="noreferrer noopener">Makito</a>'s <a href=https://github.com/SumiMakito/hexo-theme-journal/ target=_blank rel="noreferrer noopener">Journal.</a> <br>
<br>
&copy;
GT Bioss.
</div>
</div>
<script src=/beta/js/journal.js></script>
</body>
</html>