<!DOCTYPE HTML>
<html>

<head>
  <title>Bioss</title> 

  <meta name="description" content="Page du groupe de travail sur la
				    biologie systémique symbolique">
				    
  <meta name="keywords" content="Modélisation symbolique de systèmes
				 biologiques, Systèmes dynamiques
				 discrets, Systèmes formels,
				 Langages de modélisation,
				 Sémantique, Vérification de
				 modèles, Réduction, Prédiction
				 sous incertitude, Inférences
				 d'interactions et de règles,
				 Systèmes hybrides">

  <meta http-equiv="content-type" content="text/html;
					   charset=utf-8">

  <link rel="stylesheet" type="text/css" href="../../style/style.css">
</head>

<body>
  <div id="main">
    <div id="header">
      <div id="logo">
        <div id="logo_text">
          <!-- class="logo_colour", allows you to change the colour of
          the text -->
          <h1><a href="index.html">
	      <span class="logo_colour">Bioss</span>
	  </a></h1>
          <h2>Groupe de travail sur la biologie systémique symbolique</h2>
        </div>
      </div>
      <div id="menubar">
        <ul id="menu">
          <!-- put class="selected" in the li tag for the selected
          page - to highlight which page you're on -->
          <li><a href="../../index.html">Accueil</a></li>
          <li><a href="../../membres.html">Membres</a></li>
          <li><a href="../../manif.html">Manifestations</a></li>
          <li><a href="../../projets.html">Projets</a></li>
          <li><a href="../../contact.html">Contacts</a></li>
        </ul>
      </div>
    </div>

    <div id="content_header"></div>

    <div id="site_content">

      <!-- <div id="banner"></div> -->

      <div id="sidebar_container">
        <div class="sidebar">
          <div class="sidebar_top"></div>
          <div class="sidebar_item"></div>
          <div class="sidebar_base"></div>
        </div>

        <div class="sidebar">
          <div class="sidebar_top"></div>
          <div class="sidebar_item">
            <h3>Liens utiles</h3>
            <ul>
              <li><a href="http://www.cnrs.fr/">
		  <img src="../../img/logoCNRS.jpg" alt="CNRS"></a></li>
              <li><a href="http://www.gdr-bim.cnrs.fr/">
		  <img src="../../img/logoBIM.jpg" alt="GDR Bioinformatique 
						  moléculaire"></a></li>
              <li><a href="https://www.gdr-im.fr/">
		  <img src="../../img/logoIM.jpg" alt="GDR Informatique 
						  mathématique"></a></li>
            </ul>
          </div>
          <div class="sidebar_base"></div>
        </div>
      </div>

      <div id="content">
        <!-- insert the page content here -->
        <h1>1st Bioss Workshop: "Systemic Symbolic Biology"<br/><br/>
	<i>September 18th, 2015</i>
		</h1>
	
	<h3>Objectives</h3>
	<p align="justify">
	  Bioss is a new French working group on the symbolic modelling of 
	  biologic systems, combining discrete mathematics and fundamental 
	  computer science with molecular biology and medicine. This thematic 
	  aims at studying information transmissions in biological systems, 
	  by using formal methods from computer science (or new ones) in order 
	  to model, analyse and understand the dynamics of complex living systems.<br>
	  This first workshop aims at bringing together the members of the 
	  Bioss working group as well as anyone interested in the Bioss topics, 
	  and to discuss the current activities of our research interests. It will 
	  concentrate on the young generation, by giving invited PhD students and 
	  postdocs the opportunity to present their recent or current work. This 
	  will give a panorama, representative of the French activities on Symbolic 
	  Systems Biology.
	</p>
	
	<h3>Scope</h3>
	<p align="justify">
	  The program will consist of invited talks, mainly by PhD students and postdocs.<br>
	  Topics of Symbolic Systems Biology include, but are not limited to:<br>
	  <ul>
	   <li> modelling of biological systems, </li>
	   <li> discrete or hybrid formalisms, </li>
	   <li> semantics (included stochastic), </li>
	   <li> model verification, </li>
	   <li> model reduction, </li>
	   <li> prediction (under uncertainty), </li>
	   <li> model inference. </li>
	  </ul>
	</p>

	<h3>Registration</h3>
	<p align="justify">
	  Registration is free but mandatory through the CMSB <a href="http://dr17.azur-colloque.cnrs.fr/pre-inscription.php?colloque=85&lang=en">registration page</a>, even if you don't attend CMSB.
	</p>
	
	<h3>Programme</h3>
	<p align="left">
	<b>14:30 - 14:55</b> <br>
	&nbsp;&nbsp;&nbsp;&nbsp;Marc Bouffard - 
	<i> Extracting logic gates from a metabolic network, why and how ? </i><br>
	<b>14:55 - 15:20</b> <br> 
	&nbsp;&nbsp;&nbsp;&nbsp;Pauline Traynard -
	<i> Logical modeling and model-checking of the mammalian cell cycle </i><br>
	<b>15:20 - 15:45</b> <br>
	&nbsp;&nbsp;&nbsp;&nbsp;Julie Laniau -
	<i> Metabolic network reconstruction with combinatorial
        optimization : influence of cross-compartment metabolites </i><br>
	<b>15:45 - 16:00</b> <br>
	&nbsp;&nbsp;&nbsp;&nbsp;<i>Coffee break</i> <br>
	<b>16:00 - 16:25</b> <br> 
	&nbsp;&nbsp;&nbsp;&nbsp;Alexandre Rocca -
	<i> Application of Formal Methods to Biological Systems Modelling </i><br>
	<b>16:25 - 16:50</b> <br> 
	&nbsp;&nbsp;&nbsp;&nbsp;Alexandre Temperville -
	<i> Computation of sparse conservation laws and applications </i><br>
	<b>16:50 - 17:15</b> <br> 
	&nbsp;&nbsp;&nbsp;&nbsp;Louis Fippo-Fitime -
	<i> Integrating time-series data on large-scale discrete cell-based models </i><br>
	<b>17:15 - 17:40</b> <br> 
	&nbsp;&nbsp;&nbsp;&nbsp;Pierre Boutillier -
	<i> A new simulator for large Kappa models </i><br>
	</p>
	
	
	<h3>Abstracts</h3> 
	<p align="justify">
	  <b>Marc Bouffard (Univ. Paris Sud / LRI)</b> - 
	  <i>Extracting logic gates from a metabolic network, why and how ?</i><br> 
	  Synthetic biology can be used to design artificial
        bio-devices that can perform successively the samples intake, its
        analysis and provide an integrated response. This will be performed by a
        logic function applied to the output of the sensors that respond to the
        bio-markers of the targeted pathology. This logic function is computed
        using logic circuits made of interconnected logic gates. For this
        purpose, a library of basic logic gates that can be wired together so as
        to function correctly in the same environment, will be automatically
        extracted from the metabolic networks of living organisms.
		<br><br>
		
		<b>Pauline Traynard (Inria Paris-Rocquencourt)</b> - 
	  <i>Logical modeling and model-checking of the mammalian cell cycle</i><br> 
	  The molecular networks controlling cell cycle progression
        have been predominantly modelled using differential equations, an
        approach which demands to define complex regulatory terms with poorly
        characterised kinetic parameters. In contrast, qualitative dynamical
        models are easier to define, analyse and compose.<br>
        We revisit a boolean model for the core network behind the mammalian
        cell cycle (Fauré et al. Bioinformatics, 2006), taking into account
        recent advances in the characterisation of the underlying molecular
        networks to obtain a better qualitative consistency between simulations
        and documented mutants features. In particular, we refine the model to
        account for the role of the tumour suppressor protein Rb as a
        multifunctional protein, involved in independent proliferative control
        mechanisms. Using a multilevel rather than a boolean variable models the
        fact that differently phosphorylated forms of Rb result in different
        effects on other components of the network. We then introduce a crucial
        regulatory link between Rb and p27, another important cell cycle
        repressor.<br> 
        We evaluate the dynamical properties of the resulting model with
        synchronous and asynchronous simulations using the software GINsim
        (http://www.ginsim.org). We also have designed temporal logic queries,
        enabling an efficient and automatic verification of key dynamical
        properties such as conditions on the activation of components or the
        order of changes of their levels, with the symbolic model checker
        NuSMV. Moreover, adding transition probabilities allow a stochastic
        simulation of the model with the software MaBoSS and provides new
        insights into the dynamical behavior of the system.
		<br><br>
		
		<b>Julie Laniau (Inria Rennes)</b> - 
	  <i>Metabolic network reconstruction with combinatorial
        optimization : influence of cross-compartment metabolites</i><br> 
	  <br>
		
		<b>Alexandre Rocca (Univ. Joseph Fourier / Verimag)</b> - 
	  <i>Application of Formal Methods to Biological Systems Modelling</i><br> 
	  Modelling complex biological systems as differential
        equations mostly results in non-linear, high-dimensional, and stiff
        models. The validation of those models according to experimental results
        is often done with simulation. Moreover, to cope with the uncertainty of
        real biological systems, and to expand the experimental results to a
        more general model, uncertain parameters are used.To validate a large
        set of initial states and parameter values, a large number of simulation
        runs is needed. However, the result, being non-exhaustive, is not
        guaranteed. Formal verification techniques allow proving properties,
        with set-based reachability computation techniques, by replacing
        simulation runs with conservative sets of trajectories. Applying the
        Bernstein expansion of polynomials, we developed a reachability tool on
        polynomial systems with uncertain parameters. This tool takes advantage
        of the particular forms of the polynomials modelling of biochemical
        networks to speed up the Bernstein expansion.This allows a faster
        verification of complex biological systems. The tool is directly applied
        on a real system (Iron Homeostasis project). It is developed in
        collaboration with a biological experimentation team (LBFA, Grenoble) to
        address the needs of biologists.
		<br><br>
		
		<b>Alexandre Temperville (Univ. Lille 1 / CRIStAL)</b> - 
		<i>Computation of sparse conservation laws and applications</i><br>
	  Conservation laws are a key-tool to study systems of chemical
        reactions in biology. After discussing about what good conservation laws
        can be, we propose a greedy algorithm, computing a sparsest set of
        conservation laws from a given set of conservation laws. Then, we
        briefly present an improvement of this algorithm, some benchmarks on
        Maple and Python codes of this algorihtm over a subset of the curated
        models taken from the BioModels database. At last, we will finish to
        talk about prospects of practical applications of this algorithm.
		<br><br>
		
		<b>Louis Fippo-Fitime  (École Centrale de Nantes / IRCCYN)</b> - 
	  <i>Integrating time-series data on large-scale discrete cell-based models</i><br> 
	  In this work we propose an automatic way of generating and
        verifying formal hybrid models of signaling and transcriptional events,
        gathered in large-scale regulatory networks.This is done by integrating
        temporal and stochastic aspects of the expression of some biological
        com- ponents. The hybrid approach lies in the fact that measurements
        take into account both times of lengthening phases and discrete switches
        between them. The model proposed is based on a real case study of
        keratinocytes differentiation, in which gene time-series data was
        generated upon Calcium stimulation.<br>
        To achieve this we rely on the Process Hitting (PH) formalism that was
        designed to consider large-scale system analysis. We first propose an
        automatic way of detecting and translating biological motifs from the
        PID database to the PH formalism. Then, we propose a way of estimating
        temporal and stochastic parameters from time-series expression data of
        action on the PH. Simulations emphasize the interest of synchronizing
        concurrent events.
		<br><br>
		
		<b>Pierre Boutillier (Univ. Paris Diderot / PPS)</b> - 
	  <i>A new simulator for large Kappa models</i><br> 
	  When trying to model biological systems, one can either rely
        on so called "chemical reaction network" representations and use
        techniques based on multisets rewriting or ODEs for simulations. An
        alternative is to use representations that rely on reaction patterns
        also called rule-based. While the first method is extremely efficient
        when it applies, it requires somehow to compute the space of reachable
        species and is subject to combinatorial explosion as the number of
        simulated reactions grows rapidly. Kappa is a language based on graph
        pattern rewriting, that attemps to cope with the combinatorics of
        signalling pathways. A simulation step is based on Gillespie's algorithm
        and requires to maintain all rule's left hand sides matches at each
        state of the system. Keeping these matches up-to date is the bottleneck
        of the simulator. In this talk I will describe how we attempt to
        minimize the cost of this update operation.
		<br>
	</p>
	
	<p style="text-indent:2em" align="justify"></p>
      </div>

      <!-- <div id="banner"></div> -->

    </div>

    <div id="content_footer"></div>

    <div id="footer">
      <p><a href="../../index.html">Accueil</a>
      | <a href="../../membres.html">Membres</a>
      | <a href="../../manif.html">Manifestations</a>
      | <a href="../../projets.html">Projets</a>
      | <a href="../../contact.html">Contact</a></p>
      <p><a href="http://validator.w3.org/check?uri=referer">HTML5</a>
      | <a href="http://jigsaw.w3.org/css-validator/check/referer">CSS</a>
      | <a href="http://www.html5webtemplates.co.uk">HTML5 Web
      Templates</a></p>
    </div>
  </div>
</body>
</html>
