<!DOCTYPE HTML>
<html>

<head>
  <title>Bioss</title> 

  <meta name="description" content="Page du groupe de travail sur la
				    biologie systémique symbolique">
				    
  <meta name="keywords" content="Modélisation symbolique de systèmes
				 biologiques, Systèmes dynamiques
				 discrets, Systèmes formels,
				 Langages de modélisation,
				 Sémantique, Vérification de
				 modèles, Réduction, Prédiction
				 sous incertitude, Inférences
				 d'interactions et de règles,
				 Systèmes hybrides">

  <meta http-equiv="content-type" content="text/html;
					   charset=utf-8">

  <link rel="stylesheet" type="text/css" href="../../style/style.css">
</head>

<body>
  <div id="main">
    <div id="header">
      <div id="logo">
        <div id="logo_text">
          <!-- class="logo_colour", allows you to change the colour of
          the text -->
          <h1><a href="index.html">
	      <span class="logo_colour">Bioss</span>
	  </a></h1>
          <h2>Groupe de travail sur la biologie systémique symbolique</h2>
        </div>
      </div>
      <div id="menubar">
        <ul id="menu">
          <!-- put class="selected" in the li tag for the selected
          page - to highlight which page you're on -->
          <li><a href="../../index.html">Accueil</a></li>
          <li><a href="../../membres.html">Membres</a></li>
          <li><a href="../../manif.html">Manifestations</a></li>
          <li><a href="../../projets.html">Projets</a></li>
          <li><a href="../../contact.html">Contacts</a></li>
        </ul>
      </div>
    </div>

    <div id="content_header"></div>

    <div id="site_content">

      <!-- <div id="banner"></div> -->

      <div id="sidebar_container">
        <div class="sidebar">
          <div class="sidebar_top"></div>
          <div class="sidebar_item"></div>
          <div class="sidebar_base"></div>
        </div>

        <div class="sidebar">
          <div class="sidebar_top"></div>
          <div class="sidebar_item">
            <h3>Liens utiles</h3>
            <ul>
              <li><a href="http://www.cnrs.fr/">
		  <img src="../../img/logoCNRS.jpg" alt="CNRS"></a></li>
              <li><a href="http://www.gdr-bim.cnrs.fr/">
		  <img src="../../img/logoBIM.jpg" alt="GDR Bioinformatique 
						  moléculaire"></a></li>
              <li><a href="https://www.gdr-im.fr/">
		  <img src="../../img/logoIM.jpg" alt="GDR Informatique 
						  mathématique"></a></li>
            </ul>
          </div>
          <div class="sidebar_base"></div>
        </div>
      </div>

      <div id="content">
        <!-- insert the page content here -->
        <h1>Journées annuelles du groupe de travail (1ère édition)</h1>

	<h3>Informations générales</h3>
	<p align="justify">
	  Date : 23 novembre 2015
	</p>
	<p align="justify">
	  Lieu :
	  Amphithéâtre Turing,
	  Bâtiment Sophie Germain,
	  Université Paris Diderot,
	  Paris.
	</p>
	<p align="justify">
	  Organisateurs : <a url="http://www.pps.univ-paris-diderot.fr/~jkrivine/homepage/Home.html">Jean Krivine</a> et <a url="http://www.pps.univ-paris-diderot.fr/~ruet/">Paul Ruet</a>
	</p>
	
	<p align="justify">
	  Pour sa première année d'existence, le GT Bioss organise la 1ère
	  édition de ces journées annuelles (qui se tiendront cette année sur
	  une journée). Ces journées se veulent être la rencontre
	  annuelle des membres du groupe de travail "Biologie systémique
	  symbolique", commun au GDR Infomatique mathémétique et au GDR
	  Bio-informatique moléculaire.
	</p>	
	

	<h3>Programme</h3>
	<p align="left">
	<b>09h00 - 10h00</b> - <b>Accueil</b><br>
	<b>10h00 - 11h00</b> - <b>Conférence plénière</b> - Christine Brun -
	<i>Interactomes of multifunctional proteins</i><br>
	<b>11h00 - 11h15</b> - <b>Café</b> - <i>Mise en place Session 1
	  exposés courts</i><br>
	<b>11h15 - 11h30</b> - Célia Biane - <i>Interaction network game
	  applied to drug prediction in precision medicine</i><br>
	<b>11h30 - 11h45</b> - Sucheendra Palaniappan - <i>Approximating the
	  dynamics of the hybrid stochastic-deterministic apoptosis
	  pathway</i><br>
	<b>11h45 - 12h00</b> - Adrien Rougny - <i>Two qualitative dynamics
	semantics for SBGN process description maps</i><br>
	<b>12h00 - 12h15</b> - François Fages - <i>Synthesizing configurable
	  biochemical implementation of linear systems from their transfer
	  function specifications</i><br>
	<b>12h15 - 12h30</b> - Nathalie Théret  - <i>Microenvironment and
	  activation of TGF-β</i><br>
	<b>12h30 - 14h00</b> - <b>Déjeuner</b><br>
	<b>14h00 - 15h00</b> - <b>Conférence plénière</b> - Oded Maler -
	<i>Dynamical systems biology</i><br>
	<b>15h00 - 15h15</b> - <b>Café</b> - <i>Mise en place Session 2
	  exposés courts</i><br>
	<b>15h15 - 15h30</b> - Vincent Danos - <i>Models of growth</i><br>
	<b>15h30 - 15h45</b> - Jérôme Feret - <i>Une approche algébrique pour
	  détecter et utiliser les symmétries d'un modèle basé sur des règles
	  de récriture</i><br>
	<b>15h45 - 16h00</b> - Adrien Basso-Blandin - <i>A knowledge
	  representation meta-model for rule-based modelling of signalling
	  networks</i><br>
	<b>16h00 - 16h15</b> - Loïc Paulevé - <i>Abstractions pour la dynamique
	  des réseaux qualitatifs</i><br>
	<b>16h15 - 16h30</b> - Paul Ruet - <i>Negative local feedbacks in
	  Boolean networks</i><br>
	<b>16h30 - 16h45</b> - Adrien Richard - <i>Simple dynamics on
	  graphs</i><br>
	<b>16h45 - 17h00</b> - <b>Café</b> - <i>Mise en place Session 3 exposés
	  courts</i><br>
	<b>17h00 - 17h15</b> - Carito Guziolowski - <i>Integrating omics data
	  into large-scale biological networks</i><br>
	<b>17h15 - 17h30</b> - Gautier Stoll - <i>MaBoSS tool : modeling
	  signaling network in a Boolean framework with continuous time.
	  Principles and applications</i><br>
	<b>17h30 - 17h45</b> - Vincent Picard - <i>Analyse stationnaire des
	  réseaux de réactions : systèmes de contraintes en modélisation
	  stochastique</i><br>
	<b>17h45 - 18h00</b> - Virgile Andreani - <i>TBA</i><br>
	</p>
		
	<h3>Résumés</h3> 
	<p align="justify">
	  <b>Virgile Andreani</b> - <i>A stochastic model of
	    metabolism and growth</i><br>
	  It has been recently demonstrated that stochastic
	  fluctuations in the expression level of metabolic enzymes
	  can cause growth fluctuations, and that conversely, growth
	  fluctuations can propagate back to perturb gene expression
	  [1]. However, our quantitative understanding of these
	  observations is limited. In particular, the specific
	  contribution to the global phenotypic heterogeneity of these
	  two intertwined processes in unclear. Our objective here is
	  to propose a model that relates in a simple but quantitative
	  manner cell metabolism, gene expression and growth, together
	  with their temporal fluctuations. To do so, we will leverage
	  on and extend the model of Weiße et al. [2] representing in
	  an abstract manner the main aspects of the economy of a
	  growing cell. In this talk I will present our strategy to
	  extend the model.<br>
	  [1] Kiviet et al. Stochasticity of metabolism and growth at
	  the single-cell level, Nature, 2014, 514:376-379.<br>
	  [2] Weiße et al. Mechanistic links between cellular
	  trade-offs, gene expression, and growth, PNAS, 2015,
	  112(9):E1038-E1047.<br><br>
	  <b>Adrien Basso-Blandin</b> - <i>A knowledge representation
	  meta-model for rule-based modelling of signalling
	  networks</i><br>
	  The study of cellular signalling pathways and their
	  deregulation in disease states, such as cancer, is a large
	  and extremely complex task. Indeed, these systems involve
	  many parts and processes but are studied piecewise and their
	  literatures and data are consequently fragmented,
	  distributed and sometimes - at least apparently -
	  inconsistent. This makes it extremely difficult to build
	  significant explanatory models with the result that effects
	  in these systems that are brought about by many interacting
	  factors are poorly understood. In this context, we introduce
	  a graph-based meta-model, attuned to the representation of
	  cellular signalling networks, which aims to ease this
	  massive cognitive burden on the rule-based curation
	  process. This meta-model is a generalization of that used by
	  Kappa and BNGL which allows for the flexible representation
	  of knowledge at various levels of granularity. In
	  particular, it allows us to deal with information which has
	  either too little, or too much, detail with respect to the
	  strict rule-based meta-model. Our approach provides a basis
	  for the gradual aggregation of fragmented biological
	  knowledge extracted from the literature into an instance of
	  the meta-model from which we can define an automated
	  translation into executable Kappa programs.<br><br>
	  <b>Célia Biane</b> - <i>Interaction network game applied to
	  drug prediction in precision medicine </i><br>
	  Precision medicine aims at the use, in the clinic, of the
	  unique molecular profile of each patient to predict the
	  risks and benefits of treatments. This approach would be
	  particularly helpful in the case of complex diseases such as
	  cancer, where only a fraction of patients are responsive to
	  drugs while others can exhibit severe side-effects. The
	  field is looking forward for new computational methods
	  guiding clinical decision-making toward the best therapy for
	  the patient. In the endeavor of establishing a causal
	  relationship between molecular profiles and clinical
	  phenotypes of patients, network medicine studies the cause
	  of diseases on the molecular interaction networks of
	  patients. In these networks molecules are represented as
	  nodes and interactions between these molecules are
	  represented as edges. In this context, the prediction of
	  therapies results from a decision-making process based on
	  the dynamics of the network. We propose to study the impacts
	  of disease and treatment on the dynamics of molecular
	  networks in order to predict beneficial therapies. We
	  developed a computational model coupling two theoretical
	  frameworks: game theory to model decision-making and Boolean
	  models of dynamics to represent the evolution of the
	  patient's molecular interaction system. We applied the model
	  to best therapeutic strategy prediction in the case of
	  breast cancer.<br><br>
	  <b>Christine Brun</b> - <i>Interactomes of multifunctional
	    proteins (Keynote talk)</i><br><br>
	  <b>Vincent Danos</b> - <i>Models of growth</i><br><br>
	  <b>François Fages</b> - <i>Synthesizing configurable
	    biochemical implementation of linear systems from their
	    transfer function specifications</i><br><br>
	  <b>Jérôme Feret</b> - <i>Une approche algébrique pour
	  détecter et utiliser les symmétries d'un modèle basé sur des
	  règles de récriture</i><br>
	  Nous proposons de décrire des groupes de transformations qui
	  opèrent sur des graphes à sites, et montrons rapidement sous
	  quelles hypothèses ils induisent diverses formes de
	  bisimulations sur diverses sémantiques de Kappa.<br><br>
	  <b>Oded Maler</b> - <i>Dynamical systems biology (Keynote
	    talk)</i><br>
	  In this talk I argue that progress in Biology requires,
	  among other things, a more modern approach to modeling and
	  analysis of dynamical models. Such models should not be
	  restricted to classical dynamical systems but also involve
	  concepts and ideas from discrete-event dynamical systems
	  (automata) and hybrid (discrete-continuous) systems. I will
	  present some recent techniques for exploring the dynamics of
	  under-determined systems, that is, systems that admit
	  uncertainty in initial conditions, parameters and
	  environmental conditions. These techniques, inspired by
	  formal verification, can be used to assess the robustness of
	  proposed models and increase our confidence in their
	  plausibility.<br><br>
	  <b>Sucheendra Palaniappan</b> - <i>Approximating the
	  dynamics of the hybrid stochastic-deterministic apoptosis
	  pathway</i><br>
	  Modeling and analysis of the dynamics of biological systems
	  while accounting for single cell fluctuations is
	  important. In particular, there has been recent work on a
	  hybrid stochastic-deterministic (HSD) model of TRAIL induced
	  apoptosis that combines a deterministic signal transduction
	  modeland a stochastic model for protein turnover that can
	  explain fractional killing and predict the time dependent
	  evolution of cell resistance to TRAIL. While this model is
	  extremely useful for analyzing TRAIL induced apoptosis by
	  drawing simulations in a single cell setting, it can be
	  limiting in cases when we want to analyse the system in a
	  multi-scale setting (say modeling a spheroid of millions of
	  cells at larger time horizon for clinical trials). In such
	  cases, simulating the original model for repeated analysis
	  tasks can become extremely time consuming due to the scale
	  of the resultant system. Instead, one could directly
	  approximate the dynamics of the underlying system as an
	  intermediate level behavioral model and use this
	  approximation instead. In this talk, we will present results
	  describing a minimalist discrete appromixation (Dynamic
	  Bayesian Networks (DBNs) ) of the dynamics of the HSD
	  model. We will describe how analysis tasks on the original
	  HSD model translates to probabilistic inference tasks on the
	  DBN. We will also describe several algorithmic improvements
	  we make over existing analysis methods on DBNs in
	  general.<br><br>
	  <b>Loïc Paulevé</b> - <i>Abstractions pour la dynamique des
	    réseaux qualitatifs</i><br>
	  Un rapide aperçu de résultats et perspectives reposant sur
	  des techniques d'interprétation abstraite pour appréhender
	  la dynamique des réseaux booléens et discrets à grande
	  échelle : réduction et vérification de modèles, prédiction
	  de mutations, reprogrammation cellulaire...<br><br>
	  <b>Vincent Picard</b> - <i>Analyse stationnaire des réseaux
	    de réactions : systèmes de contraintes en modélisation
	    stochastique</i><br>
	  L'étude de la dynamique des réseaux de réactions est un
	  enjeu majeur de la biologie des systèmes. Cela peut être
	  réalisé de deux manières : soit de manière déterministe à
	  l'aide d'équations différentielles, soit de manière
	  probabiliste à l'aide de chaînes de Markov. Dans les deux
	  cas, un problème majeur est celui de la détermination des
	  lois cinétiques impliquées et l'inférence de paramètres
	  cinétiques associés. Pour cette raison, l'étude directe de
	  grands réseaux de réactions est impossible. Dans le cas de
	  la modélisation déterministe, ce problème peut-être
	  contourné à l'aide d'une analyse stationnaire du réseau. Une
	  méthode connue est celle de l'analyse des flux à l'équilibre
	  (FBA) qui permet d'obtenir des systèmes de contraintes
	  linéaires à partir d'informations sur les pentes moyennes
	  des trajectoires. Dans cet exposé je présenterai des pistes
	  pour étendre ces approches dans le contexte stochastique en
	  déduisant des contraintes non nécessairement linéaires à
	  partir d'informations sur les moments (moyennes, variances,
	  covariances) d'un ensemble de trajectoires.<br><br>
	  <b>Adrien Richard</b> - <i>Simple dynamics on graphs</i><br>
	  Biological networks, such gene or neural networks, are often
	  modeled by finite dynamical systems, that is, dynamical
	  systems where each variable evolves in a finite interval of
	  integer A. In this presentation, we address the following
	  question: does the interaction graph of a finite dynamical
	  system can force this system to have a "complex" dynamics ?
	  We provide a negative answer when |A|>2 by proving that, for
	  every signed digraph G, there exists a finite dynamical with
	  interaction graph G that converges toward a unique fixed
	  point in logarithmic time. The boolean case |A|=2 is more
	  difficult, and we provide partial answers instead. For
	  instance, given an unsigned digraph G, we prove that if G
	  contains a directed wheel (resp. is symmetric), there exists
	  a boolean system with interaction graph G that converges
	  toward a unique fixed point in linear time (resp. constant
	  time).<br><br>

	  <b>Adrien Rougny</b> - <i>Two qualitative dynamics semantics
	    for SBGN process description maps</i><br>
	  Qualitative dynamics semantics allow to model large reaction
	  networks with unknown kinetic parameters. In this work, we
	  present two qualitative dynamics semantics for reaction
	  networks formalized into the SBGN Process Description
	  language (SBGN-PD). These two semantics, namely the general
	  semantics and the stories semantics, allow to model any
	  SBGN-PD map into an automata network, that can then be
	  simulated to catch the main dynamical features of the
	  network. While the general semantics refines the standard
	  Boolean semantics of reaction networks by taking into
	  account all the main features of SBGN-PD, the stories
	  semantics allows to model several molecules of a network by
	  a unique variable, reducing in this way the size of the
	  models. We present those two semantics and compare them on a
	  large biological network example, the E2F/RB
	  pathway.<br><br>

	  <b>Paul Ruet</b> - <i>Negative local feedbacks in Boolean
	    networks</i><br>
	  
	  <b>Gautier Stoll</b> - <i>MaBoSS tool: modeling signaling
	    network in a Boolean framework with continuous
	    time. Principles and applications</i><br>  
	  MaBoSS is a C++ software, that models signaling network, in
	  a Boolean framework with continuous time. Influences between
	  nodes is given in a specific language, that mixes Boolean
	  logic and real number operators, in order to specify a rate
	  of activation and a rate of inhibition for each node. Each
	  of these rates depends on the Boolean states of the other
	  nodes of the network. MaBoSS applies a continuous time
	  Markov process to a model described in this language, and
	  produces time-dependent probabilities and estimates
	  asymptotic behavior. MaBoSS has been applied to several
	  biological situations (cell cycle, cell fate,
	  senescence/geroconversion). Quantitative modeling results
	  can be confronted to experimental data, resulting in
	  interesting interpretations. MaBoSS modeling framework can
	  be interpreted as a method between ODE and Boolean
	  modeling.<br><br>

	  <b>Nathalie Théret</b> - <i>Microenvironment and activation
	    of TGF-β</i><br>

	  Transforming growth factor TGF-β plays pivotal roles in
	  numerous biological processes including tissue homeostasis
	  and morphogenesis, and is implicated in a number of
	  pathological processes including inflammation, fibrosis and
	  cancer. Targeting the deleterious effects of TGF-β without
	  affecting its physiological role is the common goal of
	  therapeutic strategies. While several strategies based on
	  blocking TGF-β antibodies or small inhibitors of TGF-β
	  receptors have been investigated, the impact of the cellular
	  microenvironment that triggers and regulates TGF-β
	  bioavailability has not been taken into account so
	  far. Indeed, TGF-β is synthesized in large amount and exists
	  as an inactive molecule, latent TGF-β (LAP-TGF-β), which
	  needs to be activated and released from the extracellular
	  matrix network. Changes in the cellular microenvironment in
	  pathological situations are expected to play a direct and
	  important role in the alteration of TGF-β activity. As a
	  result, the complexity of microenvironment networks requires
	  modeling approaches to understand and predict how TGF-β
	  activation is regulated and ultimately identify putative
	  targets suitable for future therapy. To model the dynamic of
	  TGF-β activation out of the cell, we use a rule-based
	  modeling approach (Kappa language), which consists in
	  describing explicitly the biochemical structure of chemical
	  species as graphs of connected proteins. Rewriting rules
	  encoding complexation, decomplexation, and
	  post-translational modifications are well suited for
	  describing the extracellular matrix network that regulates
	  TGF-β activation. Literature curation (116 publications from
	  1988 to 2014) allowed us to collect information relative to
	  the regulation of TGF-β activation in the extracellular
	  matrix and to elaborate a model integrating 31 proteins and
	  96 rules. Using proteomic data to parameterize the model, we
	  investigated the sensitivity of TGF-β release to changes in
	  microenvironment. Such program will provide a significant
	  input in our understanding of the dynamics of TGF-β
	  activation regulated by microenvironment. We believe that
	  the extracellular microenvironment is a major parameter to
	  consider in future therapeutic approaches targeting TGF-β in
	  cancer.
	</p>

        <h3>Participants</h3>
	<p align="justify">
	  Patrick AMAR, LRI, Université Paris Sud<br>
	  Virgile ANDREANI, ENS<br>
	  Paolo Ballarini, MICS, École centrale Paris<br>
	  Adrien BASSO-BLANDIN, LIP, ENS-Lyon<br>
	  Éléonore BELLOT, ENS<br>
	  Célia BIANE, IBISC, Université d'Évry - Val d'Essonne<br>
	  Marc BOUFFARD, LRI, Université Paris Sud<br>
	  François BOULIER, CRISTAL, Université de Lille<br>
	  Christine BRUN, TAGC, CNRS Marseille<br>
	  Vincent DANOS, DIENS, CNRS Paris<br>
	  Victorien DELANNÉE, IRISA, Université de Rennes<br>
	  Franck DELAPLACE, IBISC, Université d'Évry - Val d'Essonne<br>
	  Cinzia DI GIUSTO, I3S, Université de Nice - Sophia Antipolis<br>
	  Mohamed ELATI, ISSB, Université d'Évry - Val d'Essonne<br>
	  François FAGES, INRIA Saclay<br>
	  Éric FANCHON, TIMC-IMAG, CNRS Grenoble<br>
	  Jérôme FERET, INRIA Paris<br>
	  Enrico FORMENTI, I3S, Université de Nice - Sophia Antipolis<br>
	  Christine FROIDEVAUX, LRI, Université Paris Sud<br>
	  Olivier GANDRILLON, LBMC, CNRS Lyon<br>
	  Carito GUZIOLOWSKI, IRCCyN, École centrale de Nantes<br>
	  Adrien HUSSON<br>
	  Jean KRIVINE, PPS, CNRS Paris<br>
	  Jonathan LAURENT, ENS<br>
	  Pascale LE GALL, MICS, École centrale Paris<br>
	  Cédric LHOUSSAINE, CRISTAL, Université de Lille<br>
	  Guillaume MADELAINE, CRISTAL, Université de Lille<br>
	  Morgan MAGNIN, IRCCyN, École centrale de Nantes<br>
	  Oded Maler, VERIMAG, CNRS Grenoble<br>
	  Tarek MELLITI, IBISC, Université d'Évry - Val d'Essonne<br>
	  Joachim NIEHREN, INRIA Lille<br>
	  Sucheendra PALANIAPPAN, IRISA, INRIA Rennes<br>
	  Loïc PAULEVÉ, LRI, CNRS Orsay<br>
	  Kévin PERROT, LIF, Université d'Aix-Marseille<br>
	  Vincent PICARD, LINA, Université de Nantes<br>
	  Damien REGNAULT, IBISC, Université d'Évry - Val d'Essonne<br>
	  Élisabeth REMY, I2M, CNRS Marseille<br>
	  Adrien RICHARD, I3S, CNRS Nice - Sophia Antipolis<br>
	  Adrien ROUGNY, LRI, Université Paris Sud<br>
	  Olivier ROUX, IRCCyN, École centrale de Nantes<br>
	  Paul RUET, PPS, CNRS Paris<br>
	  Sylvain SENÉ, LIF, Université d'Aix-Marseille<br>
	  Anne SIEGEL, IRISA, CNRS Rennes<br>
	  Pierre SIEGEL, LIF, Université d'Aix-Marseille<br>
	  Gautier STOLL, Institut Curie Paris<br>
	  Guillaume TERRADOT, DIENS, ENS<br>
	  Nathalie THÉRET, IRISA, INSERM Rennes<br>
	  Serghei VERLAN, LACL, Université Paris Est Créteil<br>
	</p>

	
	<p style="text-indent:2em" align="justify"></p>
      </div>

      <!-- <div id="banner"></div> -->

    </div>

    <div id="content_footer"></div>

    <div id="footer">
      <p><a href="../../index.html">Accueil</a>
      | <a href="../../membres.html">Membres</a>
      | <a href="../../manif.html">Manifestations</a>
      | <a href="../../projets.html">Projets</a>
      | <a href="../../contact.html">Contact</a></p>
      <p><a href="http://validator.w3.org/check?uri=referer">HTML5</a>
      | <a href="http://jigsaw.w3.org/css-validator/check/referer">CSS</a>
      | <a href="http://www.html5webtemplates.co.uk">HTML5 Web
      Templates</a></p>
    </div>
  </div>
</body>
</html>
