<!DOCTYPE HTML>
<html>

<head>
  <title>Bioss</title> 

  <meta name="description" content="Page du groupe de travail sur la
				    biologie systémique symbolique">
				    
  <meta name="keywords" content="Modélisation symbolique de systèmes
				 biologiques, Systèmes dynamiques
				 discrets, Systèmes formels,
				 Langages de modélisation,
				 Sémantique, Vérification de
				 modèles, Réduction, Prédiction
				 sous incertitude, Inférences
				 d'interactions et de règles,
				 Systèmes hybrides">

  <meta http-equiv="content-type" content="text/html;
					   charset=utf-8">

  <link rel="stylesheet" type="text/css" href="../../style/style.css">
</head>

<body>
  <div id="main">
    <div id="header">
      <div id="logo">
        <div id="logo_text">
          <!-- class="logo_colour", allows you to change the colour of
          the text -->
          <h1><a href="index.html">
	      <span class="logo_colour">Bioss</span>
	  </a></h1>
          <h2>Groupe de travail sur la biologie systémique symbolique</h2>
        </div>
      </div>
      <div id="menubar">
        <ul id="menu">
          <!-- put class="selected" in the li tag for the selected
          page - to highlight which page you're on -->
          <li><a href="../../index.html">Accueil</a></li>
          <li><a href="../../membres.html">Membres</a></li>
          <li><a href="../../manif.html">Manifestations</a></li>
          <li><a href="../../projets.html">Projets</a></li>
          <li><a href="../../contact.html">Contacts</a></li>
        </ul>
      </div>
    </div>

    <div id="content_header"></div>

    <div id="site_content">

      <!-- <div id="banner"></div> -->

      <div id="sidebar_container">
        <div class="sidebar">
          <div class="sidebar_top"></div>
          <div class="sidebar_item"></div>
          <div class="sidebar_base"></div>
        </div>

        <div class="sidebar">
          <div class="sidebar_top"></div>
          <div class="sidebar_item">
            <h3>Liens utiles</h3>
            <ul>
              <li><a href="http://www.cnrs.fr/">
		  <img src="../../img/logoCNRS.jpg" alt="CNRS"></a></li>
              <li><a href="http://www.gdr-bim.cnrs.fr/">
		  <img src="../../img/logoBIM.jpg" alt="GDR Bioinformatique 
						  moléculaire"></a></li>
              <li><a href="https://www.gdr-im.fr/">
		  <img src="../../img/logoIM.jpg" alt="GDR Informatique 
						  mathématique"></a></li>
            </ul>
          </div>
          <div class="sidebar_base"></div>
        </div>
      </div>

      <div id="content">
        <!-- insert the page content here -->
        <h1>Journées annuelles du groupe de travail (2ème édition)</h1>

	
	<h3>Informations générales</h3>
	<p align="justify">
	  Date : 1er et 2 juillet 2016
	</p>
	<p align="justify">
	  Lieu : 
	  Salle "conférence",
	  1, place de l'École (en rez de chaussée, en dessous de la Maison
	  des mathématiques et de l'informatique, en face de l'amphithéâtre
	  Charles Mérieux),
	  69007 Lyon<br>
	  Plan d'accès <a href="https://www.google.fr/maps/place/1+Place+de+l'École,+69007+Lyon-7E-Arrondissement/@45.729955,4.8257707,17z/data=!3m1!4b1!4m2!3m1!1s0x47f4ea2beb0a4f0b:0x6b747c560d0e395f">ici</a>
	</p>
	
	<p align="justify">
	  Organisateurs :
	  <a url="http://www.ens-lyon.fr/LBMC/spip/spip.php?article262">Olivier
	  Gandrillon</a>,
	  <a url="http://www.cristal.univ-lille.fr/~lhoussai/">Cédric
	  Lhoussaine</a>,
	  <a url="http://iml.univ-mrs.fr/~remy/remy.html">Élisabeth
	  Remy</a>,
	  <a url="http://pageperso.lif.univ-mrs.fr/~sylvain.sene/">Sylvain
	  Sené</a> et
	  <a url="http://www.irisa.fr/dyliss/anne.siegel">Anne
	  Siegel</a>
	</p>
	
	<p align="justify">
	  La deuxième édition des journées annuelles du GT Bioss va
	  se dérouler à la suite des Journées ouvertes de biologie,
	  informatique et mathématiques
	  (<a href="http://jobim2016.sciencesconf.org">JOBIM</a>)
	  organisées par la Société française de bio-informatique
	  (<a href="http://www.sfbi.fr">SFBI</a>). Ainsi, les 1er et 2
	  juillet 2016, les membres du GT auront le plaisir de se rencontrer
	  autour de conférences autour des thèmes suivants :<br>
	  - la modélisation stochastique en biologie ;<br>
	  - la régulation génétique ;<br>
	  - le métabolisme.
	</p>

	<h3>Inscription</h3>
	<p align="justify">
	  L'inscription, gratuite mais obligatoire, se fait en
	  remplissant le formulaire
	  accessible <a href="https://docs.google.com/forms/d/1cs20k_5aeBQK8gFkhNmgh9uSIjmDuRkKZqnWMcrYCwk/viewform?c=0&w=1&usp=mail_form_link">ici</a>.
	</p>
	
	<h3>Orateurs invités</h3>
	<p align="justify">
	  Grégory BATT, INRIA Saclay<br>
	  Marcelline KAUFMAN, Université libre de Bruxelles<br>
	  Marie-France SAGOT, INRIA Lyon<br>
	</p>
	
	<h3>Programme</h3>

	<h4>Vendredi 1er juillet</h4>

	<p align="left">
	  <b>09h00 - 09h30</b> - <b>Accueil</b><br>
	  
	  <b>09h30 - 09h45</b> - <b>Introduction des journées</b><br>

	  <b>09h45 - 10h30</b> - <b>Conférence plénière</b> - Grégory Batt -
	  <i>Predicting long-term effects of apoptosis-inducing drug
	  treatments: coupling signal transduction pathways with
	  stochastic protein turnover models </i><br>

	  <b>10h30 - 11h00</b> - Bertrand Miannay -
	  <i>Identification des voies de signalisation impliquées dans
	  le myélome multiple par programmation par contrainte</i><br>
	  <b>11h00 - 11h30</b> - Arnaud Bonnaffoux -
	  <i>Toward a dynamic multi-scale/level approach for gene regulatory
	    network inference</i><br>
	  <b>11h30 - 12h00</b> - Nicolas Schabanel -
	  <i>Folding Turing is hard but feasible</i><br>
	  
	  <b>12h00 - 13h30</b> - <b>Pause déjeuner</b><br>
	  
	  <b>13h30 - 14h15</b> - <b>Conférence plénière</b> -
	  Marie-France Sagot -
	  <i>Species interactions from a metabolism perspective</i><br>
	  
	  <b>14h15 - 14h45</b> - Nils Giordano -
	  <i>Dynamical allocation of cellular resources as an optimal
	    control problem</i><br>
	  <b>14h45 - 15h15</b> - Victorien Delannée -
	  <i>A modeling approach to evaluate the balance between
	    bioactivation and detoxification of MeIQx in human
	    hepatocytes</i><br>

	  <b>15h15 - 15h45</b> - <b>Discussion Bioss / GDR</b><br>

	  <b>15h45 - 16h15</b> - <b>Pause</b><br>
	  
	  <b>16h15 - 16h45</b> - Hugues Berry -
	  <i>Estimating the effects of spatial non-homogeneities in
	    intracellular diffusion-reactions</i><br>
	  <b>16h45 - 17h15</b> - Dan Goreac -
	  <i>Hybrid designing using stochastic backward
	    equations</i><br>
	  <b>17h15 - 17h45</b> - Guillaume Madelaine -
	  <i>Structural simplifications of reaction networks: the
	  confluence problem</i><br>
	  <b>17h45 - 18h15</b> - Ferdinanda Camporesi -
	  <i>Context-sensitive flow analyses: a hierarchy of model
	  reductions</i><br>
	</p>
	

	<h4>Samedi 2 juillet</h4>
	  
	<p align="left">
	  
	  <b>09h00 - 09h45</b> - <b>Conférence plénière</b> -
	  Marcelline Kaufman -
	  <i>On multistationarity in chemical reaction networks</i><br>

	  <b>09h45 - 10h15</b> - Kévin Perrot -
	  <i>On the flora of asynchronous locally non-monotonic
	  Boolean networks</i><br>
	  <b>10h15 - 10h45</b> - Élisabeth Remy -
	  <i>Discrete dynamics of compound regulatory circuits</i><br>

	  <b>10h45 - 11h00</b> - <b>Pause</b><br>

	  <b>11h00 - 11h30</b> - Loïc Paulevé -
	  <i>Around reachability in automata networks</i><br>

	  <b>11h30 - 12h00</b> - Emna Ben Abdallah -
	  <i>Inference of biological regulatory networks from time
	    series data</i><br>

	  <b>12h00 - 12h30</b> - Adrien Richard -
	  <i>Points fixes dans les réseaux booléens monotones</i><br>
	</p>
		
	<h3>Résumés</h3> 
	<p align="justify">
	  <b>Grégory Batt</b> - <i>Predicting long-term effects of
	    apoptosis-inducing drug treatments: coupling signal
	    transduction pathways with stochastic protein turnover
	    models</i><br>
	  TRAIL is an anti-cancer drug that induces apoptosis
	  selectively in cancer cells. Unfortunately even high doses
	  of TRAIL do not kill all cells and subsequent TRAIL
	  treatments are transiently less effective. Despite extensive
	  studies, a mechanistic understanding of these phenomena is
	  still lacking.  In this talk, I will present an extension of
	  a previously-proposed model describing TRAIL signal
	  transduction in Hela cells (Spencer et al, Nature 2011) with
	  models accounting for the turnover of the proteins involved
	  in the pathway at the cell level and the dynamics (growth
	  and death) of the cell population in monolayers or in 3D
	  spheroids.  This model is minimalistic in the sense that it
	  uses default values from the literature for all but two
	  parameters. Yet, it explains the existence of survivors
	  (fractional killing), the increased resistance of the
	  surviving population and its transient aspect. The analysis
	  of model predictions calls into question the importance of
	  survival pathways and highlights the critical role of the
	  stochastic turnover of proteins in zymogen-based pathways in
	  which activated forms are rapidly degraded.<br><br>

	  <b>Emna Ben Abdallah</b> - <i>Inference of biological
	    regulatory networks from time series data</i><br>
	  With the development of high-throughput data, there is a
	  growing need for methods that automatically generate
	  (resp. revise) admissible models. Our research aims at
	  providing a logical approach to infer Biological Regulatory
	  Networks based on given time series data. We propose a new
	  methodology for models expressed through a timed extension
	  of the Process Hitting framework (well suited for biological
	  systems). The main purpose is to have as a result the most
	  consistent network as possible with the observed data. The
	  originality of our work relies on the integration of
	  quantitative time delays directly in our learning
	  approach.<br>
	  Taking as input a background knowledge under the form of
	  influence graph and time series data, the contribution of
	  our method lies in the fact that we identify the set of
	  actions between biological components by concretizing the
	  signs (negative or positive) besides providing thresholds
	  and associating the quantitative time delays. Starting from
	  the structure of the system and its experimental time
	  series, the method addresses both inference and revision:
	  (1) If no previous dynamic model is given, we infer the
	  dynamics of the system. (2) Otherwise we take profit from
	  new time series to revise actions and delays.<br>
	  We will show the benefits of such automatic approach on
	  dynamical biological model, the circadian clock, and we
	  conduct benchmarks on the DREAM4 datasets, a popular
	  reverse-engineering challenge, in order to discuss the
	  computational performances of our algorithm.<br><br>
	  
	  <b>Hugues Berry</b> - <i>Estimating the effects of spatial
	  non-homogeneities in intracellular
	    diffusion-reactions</i><br>

	  The inner of living cells exhibits disorder, non-homogeneity
	  and obstruction. For instance, cell membranes are
	  heterogeneous collections of hierarchical spatial domains
	  with various length scales and timescales (e.g., fences,
	  lipid rafts, and caveolae) that spatially modulate the
	  diffusion of proteins. This defines a spatially
	  nonhomogeneous diffusion problem with position-dependent
	  diffusion coefficient. The impact of these deviations from
	  simple Brownian motion on the biochemical reactions that
	  take place in cells cannot be studied with the classical
	  mass-action laws of biochemical kinetics and are just
	  starting to be explored by spatially-explicit stochastic
	  simulations. In this talk, I will present an overview of the
	  recent modelling work carried out in our group on the
	  effects of receptor clustering on the dynamics of
	  ligand-binding equilibrium, and on correlations in gene
	  positions for repressilator-like gene regulation loops. Our
	  results suggest that spatial non-homogeneities are potent
	  modulators of the apparent affinity of the equilibrium
	  reaction or of the dynamical regime itself, even when the
	  elementary reaction rates are not altered.<br><br>

	  <b>Arnaud Bonnaffoux</b> - <i>Toward a dynamic
	    multi-scale/level approach for gene regulatory network
	    inference</i><br>

	  Gene regulatory networks (GRN) play an important role in
	  many biological processes, such as differentiation, and
	  their identification has raised great expectations for
	  understanding cell behavior. Many computational GRN
	  inference approaches have been described, which are based on
	  expression data but they face common issues such as data
	  scarcity, high dimensionality or population blurring (Chai
	  et al., 2014). We believe that recent high-throughput single
	  cell expression data (see e.g. Pina et al., 2012 ; Shalek et
	  al., 2014) acquired in time-series will allow to overcome
	  these issues and give access to causality, instead of
	  « simple » correlation, for gene interactions. Causality is
	  very important for mechanistic model inference and
	  biological relevance because it enables the emergence of
	  cellular decision-making. Emergent properties of a
	  mechanistic model of a GRN should then match with
	  multi-scale (molecular/cellular) and multi-level (single
	  cell/population) observations.  We will expose a GRN
	  inference framework based on these assumptions. It follows
	  three steps:<br>
	  1. Node parametric inference. We have inferred
	  the parameters from a stochastic mechanistic model of gene
	  expression, the Random Telegraph model (Kim and Marioni,
	  2013), thank's to time-series single cell expression data
	  from a population of chicken erythrocyte progenitor during
	  their differentiation process (Gandrillon et al., 1999)<br>
	  2. Model reduction. This is mostly an ongoing work, and will
	  make use of specific constraints applying to the network.<br>
	  3. The final step will consist in network inference
	  constrained by dynamic multi-scale/level
	  observations.<br><br>

	  <b>Ferdinanda Camporesi</b> - <i>Context-sensitive flow
	  analyses: a hierarchy of model reductions</i><br>
	  Rule-based modelling allows very compact descriptions of
	  protein-protein interaction networks. However, combinatorial
	  complexity increases again when one attempts to describe
	  formally the behaviour of the networks, which motivates the
	  use of abstractions to make these models more
	  coarse-grained. Context-insensitive abstractions of the
	  intrinsic flow of information among the sites of chemical
	  complexes through the rules have been proposed to infer
	  sound coarse-graining, providing an efficient way to find
	  macro-variables and the corresponding reduced models. In
	  this paper, we propose a framework to allow the tuning of
	  the context-sensitivity of the information flow analyses and
	  show how these finer analyses can be used to find fewer
	  macro-variables and smaller reduced differential
	  models.<br><br>


	  <b>Victorien Delannée</b> - <i>A modeling approach to
	    evaluate the balance between bioactivation and
	    detoxification of MeIQx in human hepatocytes</i><br>

	  Heterocyclic aromatic amines (HAA) are environmental and
	  food contaminants that are potentially carcinogen for
	  human. 2-Amino-3-methylimidazo(4,5-f)-quinoxaline (MeIQx) is
	  one of the most abundant HAA formed in cooked meat. MeIQx is
	  metabolized by cytochrome P450 1A2 in human liver into
	  detoxification and bioactivation products. Once
	  bioactivated, MeIQx metabolites can lead to DNA adduct
	  formation responsible for further genome instability.  Using
	  a computational approach, we develop a numerical model for
	  MeIQx metabolism that predicts the MeIQx biotransformation
	  into detoxification or bioactivation pathways according to
	  the concentration of MeIQx.  Our model permits to
	  investigate the balance between bioactivation (i.e. DNA
	  adduct formation pathway through Ester-O-NH-MeIQx) and
	  detoxification of MeIQx in order to predict the behaviour of
	  this environmental contaminant in human liver.<br>
	  Our results demonstrate that 1) the detoxification pathway
	  predominates, 2) predicting the bioactivation and the
	  detoxification for any initial concentration of MeIQx at any
	  time is feasible for any dataset and 3) the ratio between
	  detoxification and bioactivation pathways is not linear and
	  shows a maximum at 10µM of MeIQx in hepatocyte cell
	  model.<br><br>
	  
	  <b>Nils Giordano</b> - <i>Dynamical allocation of cellular
	    resources as an optimal control problem: novel insights
	    into microbial growth strategies</i><br>  
	  Microbial physiology exhibits growth laws that relate the
	  macromolecular composition of the cell to the growth
	  rate. Recent work has shown that these empirical
	  regularities can be derived from coarse-grained models of
	  resource allocation. While these studies focus on
	  steady-state growth, such conditions are rarely found in
	  natural habitats, where microorganisms are continually
	  challenged by environmental fluctuations. The aim of this
	  paper is to extend the study of microbial growth strategies
	  to dynamical environments, using a self-replicator model. We
	  formulate dynamical growth maximization as an optimal
	  control problem that can be solved using Pontryagin’s
	  Maximum Principle. We compare this theoretical gold standard
	  with different possible implementations of growth control in
	  bacterial cells. We find that simple control strategies
	  enabling growth-rate maximization at steady state are
	  suboptimal for transitions from one growth regime to
	  another, for example when shifting bacterial cells to a
	  medium supporting a higher growth rate. A near-optimal
	  control strategy in dynamical conditions is shown to require
	  information on several, rather than a single physiological
	  variable. Interestingly, this strategy has structural
	  analogies with the regulation of ribosomal protein synthesis
	  by ppGpp in the enterobacterium Escherichia coli. It
	  involves sensing a mismatch between precursor and ribosome
	  concentrations, as well as the adjustment of ribosome
	  synthesis in a switch-like manner. Our results show how the
	  capability of regulatory systems to integrate information
	  about several physiological variables is critical for
	  optimizing growth in a changing environment.<br><br>

	  <b>Dan Goreac</b> - <i>Hybrid designing using stochastic
	    backward equations</i><br>
	  We present some targeted-behaviour based issues in the
	  hybrid modelling of networks. The common method is derived
	  from the theory of BSDEs (backward stochastic differential
	  equations) by interpreting the reaction speeds as externally
	  controlled (thus, modifiable) parameters. In the case of
	  first-order (linear) models, we give explicit (algebraic)
	  conditions on the sets of parameters leading to
	  "controllability" (targeted behaviour). For more general
	  systems, if the time allows it, we give an intuition on how
	  parameters might be chosen by using reflected backward
	  equations and embedding in spaces of measures.<br><br>

	  <b>Marcelline Kaufman</b> - <i>On multistationarity in
	    chemical reaction networks</i><br>
	  Résumé au format
	  pdf <a href="res/res_Kaufman.pdf">ici</a>.<br><br>

	  <b>Guillaume Madelaine</b> - <i>Structural simplifications
	  of reaction networks: the confluence problem</i><br>
	  Models in system biology are often big, and need to be
	  simplified in order to be analyzed, simulated or
	  verified. We will first present a set of simplification
	  rules for reaction networks without kinetic rates. This
	  simplification preserves the non-deterministic semantics, in
	  terms of reachability of final strongly connected
	  components, called attractors.  Then we will extend the
	  reaction networks with kinetic rates. We will show that,
	  under partial steady-state assumptions, we can simplify the
	  networks by removing some linear intermediate molecular
	  species, while preserving the deterministic semantics of the
	  other species.  We will focus on the confluence of this
	  simplification, that is do we always obtain the same fully
	  simplified network, independently of the order in which the
	  simplification rules are applied. We will show that removing
	  the linear intermediate species is not confluent in
	  general. By adding another rule that simplifies some
	  "dependent reactions", we will show that we can always
	  obtain the same structure of the network and the same
	  ODEs. However, the distribution of the kinetic rates between
	  the reactions can be different. <br><br>

	  <b>Bertrand Miannay</b> - <i>Identification des voies de
	    signalisation impliquées dans le myélome multiple par
	    programmation par contrainte</i><br>
	  Résumé au format
	  pdf <a href="res/res_Miannay.pdf">ici</a>.<br><br>

	  <b>Loïc Paulevé</b> - <i>Around reachability in automata
	    networks</i><br>
	  Many elaborated questions in systems biology involve the one
	  of reachability : the existence / inevitability of a
	  sequence of events leading from a state to another. Some
	  involve the verification of reachability, many more the
	  inference of mutations for its control.  Reachability is a
	  difficult computational problem: it is PSPACE-complete for
	  Automata Networks / Petri nets with finite discrete state
	  space.  Methods relying on network topology, concurrency,
	  abstract interpretation, model reduction, aim at coping with
	  reachability in large scale networks.  In this talk, I'll
	  give an overview of a range of these methods and related
	  tools, with their applications to model-checking, dynamical
	  bifurcation identification, control target prediction, and
	  cellular differentiation.<br><br>

	  <b>Kévin Perrot</b> - <i>On the flora of asynchronous
	  locally non-monotonic Boolean networks</i><br>
	  Studies on the dynamics of Boolean networks (BNs) have
	  mainly focused on monotonic networks, where fundamental
	  questions on the links relating their static and dynamical
	  properties have been raised and addressed. In this
	  presentation, we will explore analogous questions on
	  non-monotonic networks, xor-BNs, that are BNs where all the
	  local transition functions are xor-functions. Using
	  algorithmic tools, we will present a general
	  characterisation of the asynchronous dynamics for most of
	  the cactus xor-BNs and strongly connected xor-BNs, through
	  new bisimulation equivalences specific to xor-BANs.<br><br>
	  
	  <b>Élisabeth Remy</b> - <i>Discrete dynamics of compound
	    regulatory circuits</i><br>
	  In biological regulatory networks represented in terms of
	  signed, directed graphs, topological motifs such as circuits
	  are known to play key dynamical roles. We present results on
	  the dynamical impact of the addition of a short-cut in a
	  regulatory circuit. More precisely, based on a Boolean
	  formalisation of regulatory graphs, we have unrolled
	  complete descriptions of the discrete dynamics of particular
	  motifs, under the synchronous and asynchronous updating
	  schemes. These motifs are made of a circuit of arbitrary
	  length, combining positive and negative interactions in any
	  sequence, encompassing a short circuit, and using AND, OR
	  and XOR logical rules.<br><br>
	  
	  <b>Adrien Richard</b> - <i>Points fixes dans les réseaux
	  booléens monotones</i><br>
	  Les réseaux booléens sont des systèmes dynamiques où chaque
	  variable ne peut prendre que deux états possibles: 0 ou
	  1. Depuis les travaux pionniers de Kauffman et Thomas, ce
	  sont des modèles très classiques pour les réseaux de
	  gènes. Dans ce contexte, les points fixes sont d'un intérêt
	  particulier: ils correspondent à des patterns stables
	  d'expression des gènes souvent reliés à des fonctions
	  cellulaires bien précises. Cependant, les premières
	  informations disponibles sur un réseau de gènes concernent
	  généralement le graphe d'interaction du réseau et non sa
	  dynamique.<br>
	  Une question naturelle est donc la suivante:
	  que peut-on dire sur les points fixes d'un réseau booléen en
	  fonction de son graphe d'interaction seulement ?<br>
	  Dans cette exposé, on présente une étude du plus grand
	  nombre de points fixes qu'un réseau booléen monotone peut
	  admettre en fonction de son graphe d'interaction. On donnera
	  des bornes inférieures et supérieures qui ne dépendent que
	  de la structure des cycles du graphe d'interaction. Les deux
	  paramètres centraux seront, d'une part, la taille d'un plus
	  petit ensemble de sommets intersectant tous les cycles et,
	  d'autre part, le plus grand nombre de cycles
	  disjoints. L'étude fera intervenir des théorèmes, classiques
	  en combinatoire, sur le treillis booléen et ses antichaines.<br>
	  C'est un travail réalisé en collaboration avec Julio Aracena
	  et Lilian Salinas (Université de Concepcion, Chili)
	  disponible à l'adresse suivante:
	  <a href="http://arxiv.org/abs/1602.03109">
	    http://arxiv.org/abs/1602.03109</a>.<br><br>

	  <b>Marie-France Sagot</b> - <i>Species interactions from a
	    metabolism perspective</i><br>
	  
	  The frontier between different species may be considered
	  very fuzzy as is more and more observed. Organisms are no
	  longer perceived as single genetically identical individuals
	  and are rather considered as part of communities. At its
	  extreme, one could see thus the whole of life as forming one
	  single community, or a community of communities interacting
	  sometimes closely and for long periods of evolutionary
	  time. Such interactions appear essential to understand some
	  if not all of the most fundamental evolutionary and
	  functional questions related to living organisms. They
	  however remain very little explored by computational
	  biologists, perhaps due to the difficult modelling and
	  computational issues raised. Yet, because of the complexity
	  and singularity of these communities, it is clear that
	  experimental data alone do not allow to fully understand the
	  global capacities and functions of these organisms and their
	  interactions. In this talk, I will briefly present some of
	  the models and algorithms, in the case related to
	  metabolism, that we have recently been developing with the
	  goal of better understanding some such close and often
	  persistent interactions. I will also mention a much longer
	  term objective of this work that would be to become able in
	  some cases to suggest the means of controlling for
	  equilibrium in an interacting community.<br><br>

	  <b>Nicolas Schabanel</b> - <i>Folding Turing is hard but
	    feasible</i><br>
	  We introduce and study the computational power of Oritatami,
	  a theoretical model to explore greedy molecular folding, by
	  which the molecule begins to fold before waiting the end of
	  its production. This model is inspired by our recent
	  experimental work demonstrating the construction of shapes
	  at the nanoscale by folding an RNA molecule during its
	  transcription from an engineered sequence of synthetic
	  DNA. While predicting the most likely conformation is known
	  to be NP-complete in other models, Oritatami sequences fold
	  optimally in linear time. Although our model uses only a
	  small subset of the mechanisms known to be involved in
	  molecular folding, we show that it is capable of efficient
	  universal computation, implying that any extension of this
	  model will have this property as well.<br>
	  We develop several general design techniques for programming
	  these molecules. Our main result in this direction is an
	  algorithm in time linear in the sequence length, that finds
	  a rule for folding the sequence deterministically into a
	  prescribed set of shapes depending of its environment. This
	  shows the corresponding problem is fixed-parameter tractable
	  although we proved it is NP-complete in the number of
	  possible environments. This algorithm was used effectively
	  to design several key steps of our constructions.<br>
	</p>

        <h3>Participants</h3>
	<p align="justify">
	  Jalouli ACHREF, Université de Limoges<br>
	  Vicente ACUNA, CMM, Santiago, Chili<br>
	  Émilie ALLART, CRISTAL, Université de Lille<br>
	  Adel Amar AMOURI, Dpt. de biologie, Université d'Oran<br>
	  Emna BEN ABDALLAH, IRCCyN, École centrale de Nantes<br>
	  Adrien BASSO-BLANDIN, LIP, ENS-Lyon<br>
	  Grégory BATT, Lifeware, INRIA Saclay<br>
	  Guillaume BEAUMONT, IPS2, Université Paris Sud<br>
	  Emmanuelle BECKER, IRSET, Université de Rennes<br>
	  Hugues BERRY, Beagle, INRIA Lyon<br>
	  Arnaud BONNAFFOUX, LBMC, ENS-Lyon<br>
	  Ferdinanda CAMPORESI, DIENS, ENS<br>
	  Thomas COKELAER, Biomics, Institut Pasteur<br>
	  Victorien DELANNÉE, IRISA, Université de Rennes<br>
	  Ronan DUCHESNE, LBMC, ENS-Lyon<br>
	  Maxime FOLSCHETTE, I3S, Université de Nice - Sophia Antipolis<br>
	  Enrico FORMENTI, I3S, Université de Nice - Sophia Antipolis<br>
	  Olivier GANDRILLON, LBMC, CNRS Lyon<br> 
	  Nils GIORDANO, INRIA Grenoble<br>
	  Dan GOREAC, LAMA, Université Paris-Est Marne-la-Vallée<br>
	  Carito GUZIOLOWSKI, IRCCyN, École centrale de Nantes<br>
	  Pierre GUILLON, I2M, CNRS Marseille<br>
	  Russ HARMER, LIP, ENS-Lyon<br>
	  Ulysse HERBACH, LBMC, ENS-Lyon<br>
	  Marcelline KAUFMAN, Dpt. de chimie physique et biologie théorique,
	  Université libre de Bruxelles<br>
	  Cédric LHOUSSAINE, CRISTAL, Université de Lille<br>
	  Guillaume MADELAINE, CRISTAL, Université de Lille<br>
	  Bertrand MIANNAY, IRCCyN, École centrale de Nantes<br>
	  Jean-Michel MULLER, LIP, CNRS Lyon<br>
	  Loïc PAULEVÉ, LRI, CNRS Orsay<br>
	  Kévin PERROT, LIF, Université d'Aix-Marseille<br>
	  Arnaud PORET, LBMC, ÉNS-Lyon<br>
	  Sylvain PRIGENT, Sysbio, Université de Chalmers<br>
	  Élisabeth REMY, I2M, CNRS Marseille<br>
	  Adrien RICHARD, I3S, CNRS Nice - Sophia Antipolis<br>
	  Marie-France SAGOT, ERABLE, INRIA Lyon<br>
	  Nicolas SCHABANEL, IRIF, CNRS Paris<br>
	  Sylvain SENÉ, LIF, Université d'Aix-Marseille<br>
	  Anne SIEGEL, IRISA, CNRS Rennes<br>
	  Laurent TRILLING, TIMC-IMAG, Université de Grenoble<br>
	  Jean-Yves TROSSET, BIRL, SupBioTech
	</p>

	<p style="text-indent:2em" align="justify"></p>
      </div>

      <!-- <div id="banner"></div> -->

    </div>

    <div id="content_footer"></div>

    <div id="footer">
      <p><a href="../../index.html">Accueil</a>
      | <a href="../../membres.html">Membres</a>
      | <a href="../../manif.html">Manifestations</a>
      | <a href="../../projets.html">Projets</a>
      | <a href="../../contact.html">Contact</a></p>
      <p><a href="http://validator.w3.org/check?uri=referer">HTML5</a>
      | <a href="http://jigsaw.w3.org/css-validator/check/referer">CSS</a>
      | <a href="http://www.html5webtemplates.co.uk">HTML5 Web
      Templates</a></p>
    </div>
  </div>
</body>
</html>
